
<!DOCTYPE html>

<html>
<head><style>.vega-actions a {margin-right: 12px;color: #757575;font-weight: normal;font-size: 13px;}.error {color: red;}</style><script src="https://cdn.jsdelivr.net/npm//vega@4" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-lite@2.6.0" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-embed@3" type="text/javascript"></script><meta content="text/html; charset=utf-8" http-equiv="Content-Type"/><meta content="IE=edge" http-equiv="X-UA-Compatible"/><meta content="width=device-width, initial-scale=1" name="viewport"/><link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/Font-Awesome-4.7/css/font-awesome.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap-theme.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/images/gnomic_icon.png" rel="icon" type="image/png"/><script src="https://phenviz.navigome.com/js/jquery.min.js"></script><script src="https://phenviz.navigome.com/js/bootstrap.js"></script><title>Gene profile: ENSG00000196866</title></head>
<body>
<h4>Gene profile across phenotypes - ENSG00000196866 (HIST1H2AD)</h4><div id="vis"></div><div class="col-lg-12 col-md-12 content-item content-item-1"><p>This is a visualization of GWAS-derived gene associations  across Navigome phenotypes. Only phenotypes for which the gene has at least one significant association (p-value â‰¤ 0.05/20000) in <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> <b>and/or</b> <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a> analyses are shown. The first panel is a visualization of gene-phenotype associations obtained using <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> (-log10(p-value)). Hover over any bar to highlight the phenotype in the other panels. The other panels are parallel coordinates visualizations for tissue-wise associations (z-scores)computed using <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a>  and eQTL expression data from GTEx and DGN. Significance thresholds are delineated with black lines. The parallel coordinates show which phenotypes have a significant association and in which tissues, based on gene expression data. Clicking on a specific phenotype will lead to the reference of the GWAS study.</p></div>
<script type="text/javascript">
    var spec = {"config": {"view": {"width": 400, "height": 300}}, "vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "bar", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "href": {"type": "nominal", "field": "phenotype_link"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "quantitative", "axis": {"title": "MAGMA -log10(p)"}, "field": "value"}, "y": {"type": "nominal", "axis": {"title": ""}, "field": "phenotype_reference"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-4e93ee1a91df87c61b7d73e211f0f4d1"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "x": {"type": "quantitative", "field": "ThresholdVal"}}}], "title": "MAGMA gene-wise association"}], "center": true, "data": {"name": "data-ac6c8a4c591e6f0ffdda9279d3f8dd45"}}, {"vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Nervous system", "width": 500}], "center": true, "data": {"name": "data-cecd2087023ab76d9396dc7a00d32c40"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Digestive System", "width": 500}], "center": true, "data": {"name": "data-4a02b1c0114d3f11e8796cd59335a3a3"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Blood, Arteries and Heart", "width": 500}], "center": true, "data": {"name": "data-3eb85da19fc29d59e6c999693dab67f6"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Adipose Tissue, Muscle, Skin and Cells", "width": 500}], "center": true, "data": {"name": "data-e0eee59ea6aa55ed472000e1af981f4f"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-0.0, 0.0]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Endocrine System and Others", "width": 500}], "center": true, "data": {"name": "data-226edb0acd4420e805a27d86dfd74a72"}}]}], "spacing": 40, "$schema": "https://vega.github.io/schema/vega-lite/v2.6.0.json", "datasets": {"data-ac6c8a4c591e6f0ffdda9279d3f8dd45": [{"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "magma_log10p", "value": 11.738856}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "magma_log10p", "value": 5.7096089999999995}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "magma_log10p", "value": 27.246471000000003}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "magma_log10p", "value": 12.597978999999999}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "magma_log10p", "value": 12.377216}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "magma_log10p", "value": 5.973712}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "magma_log10p", "value": 10.736009}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "magma_log10p", "value": 7.2716460000000005}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "magma_log10p", "value": 7.95253}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "magma_log10p", "value": 16.193725}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "magma_log10p", "value": 20.718285}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "magma_log10p", "value": 7.80682}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "magma_log10p", "value": 11.303923}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "magma_log10p", "value": 13.299816}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "magma_log10p", "value": 23.267349}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "magma_log10p", "value": 49.017055}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "magma_log10p", "value": 23.612041}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "magma_log10p", "value": 6.132556}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "magma_log10p", "value": 6.9434}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "magma_log10p", "value": 31.404372}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "magma_log10p", "value": 9.816275}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "magma_log10p", "value": 12.131121}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "magma_log10p", "value": 26.079788}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "magma_log10p", "value": 10.269605}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "magma_log10p", "value": 38.052733}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "magma_log10p", "value": 14.255864}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "magma_log10p", "value": 9.98527}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "magma_log10p", "value": 10.03375}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "magma_log10p", "value": 13.819501}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "magma_log10p", "value": 8.685038}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "magma_log10p", "value": 10.396077}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "magma_log10p", "value": 14.849704000000001}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "magma_log10p", "value": 13.112872}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "magma_log10p", "value": 14.683464}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "magma_log10p", "value": 16.868189}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "magma_log10p", "value": 11.613858}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "magma_log10p", "value": 25.122467}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "magma_log10p", "value": 11.871568}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "magma_log10p", "value": 16.282871}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "magma_log10p", "value": 12.068217}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "magma_log10p", "value": 38.354607}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "magma_log10p", "value": 7.5397490000000005}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "magma_log10p", "value": 11.805458}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "magma_log10p", "value": 12.115505}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "magma_log10p", "value": 8.190252000000001}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "magma_log10p", "value": 7.679231}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "magma_log10p", "value": 5.692654}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "magma_log10p", "value": 7.783200999999999}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "magma_log10p", "value": 10.924745}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "magma_log10p", "value": 6.822377}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "magma_log10p", "value": 7.892180000000001}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "magma_log10p", "value": 12.162272999999999}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "magma_log10p", "value": 9.920276}], "data-4e93ee1a91df87c61b7d73e211f0f4d1": [{"Threshold": "MAGMA -log10(p) \u2265 5.60206", "ThresholdVal": 5.60206}], "data-cecd2087023ab76d9396dc7a00d32c40": [{"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Brain_Caudate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Brain_Putamen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Brain_Cortex", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Nerve_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Nerve_Tibial", "value": 0.0}], "data-c4c9a654d45741ebc60aa567761bff19": [{"Text": "Significance Threshold", "Threshold": "PrediXcan zscore \u2265 4.70813", "ThresholdVal": 4.70813}], "data-4a3aa5bf8c5a41879455008930c25774": [{"Threshold": "PrediXcan zscore \u2264 -4.70813", "ThresholdVal": -4.70813}], "data-4a02b1c0114d3f11e8796cd59335a3a3": [{"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Colon_Transverse", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Esophagus_Mucosa", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Liver", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Stomach", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Stomach", "value": 0.0}], "data-3eb85da19fc29d59e6c999693dab67f6": [{"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "DGN_Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Whole_Blood", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Artery_Aorta", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Artery_Coronary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Artery_Tibial", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Spleen", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Spleen", "value": 0.0}], "data-e0eee59ea6aa55ed472000e1af981f4f": [{"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}], "data-226edb0acd4420e805a27d86dfd74a72": [{"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Pituitary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Pancreas", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Thyroid", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Lung", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Ovary", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Prostate", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Testis", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Uterus", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BFPC04", "phenotype": "Body fat percentage complete", "index": 0, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 11.738856, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0217, "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Ratio": "0.0389 (0.0109)", "study size": 353972, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO01", "phenotype": "platelet count", "index": 1, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 5.7096089999999995, "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO03", "phenotype": "hemoglobin concentration", "index": 2, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 27.246471000000003, "category": "g: blood", "LDSC Mean Chi-Square": 1.4634, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Ratio": "0.0199 (0.0502)", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO04", "phenotype": "reticulocyte count", "index": 3, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 12.597978999999999, "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Ratio": "0.0243 (0.0638)", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO05", "phenotype": "monocyte count", "index": 4, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 12.377216, "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Ratio": "0.0686 (0.0499)", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO06", "phenotype": "neutrophil count", "index": 5, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "association": 5.973712, "category": "g: blood", "LDSC Mean Chi-Square": 1.516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9567 (0.0204)", "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO07", "phenotype": "eosinophil count", "index": 6, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 10.736009, "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO09", "phenotype": "granulocyte count", "index": 7, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.2716460000000005, "category": "g: blood", "LDSC Mean Chi-Square": 1.5217, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO10", "phenotype": "myeloid white cell count", "index": 8, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 7.95253, "category": "g: blood", "LDSC Mean Chi-Square": 1.5375, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO11", "phenotype": "lymphocyte count", "index": 9, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 16.193725, "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO12", "phenotype": "white blood cell count", "index": 10, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 20.718285, "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO13", "phenotype": "Sum basophil neutrophil counts", "index": 11, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 7.80682, "category": "g: blood", "LDSC Mean Chi-Square": 1.5207, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.9585 (0.0207)", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "index": 12, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 11.303923, "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO20", "phenotype": "Hematocrit", "index": 13, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 13.299816, "category": "g: blood", "LDSC Mean Chi-Square": 1.47, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Ratio": "0.0221 (0.0511)", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "index": 14, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 23.267349, "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "index": 15, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 49.017055, "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Ratio": "0.012 (0.0532)", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO27", "phenotype": "Mean corpuscular volume", "index": 16, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 23.612041, "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Ratio": "0.0096 (0.0469)", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO30", "phenotype": "Sum neutrophil eosinophil counts", "index": 17, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.132556, "category": "g: blood", "LDSC Mean Chi-Square": 1.5234, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO33", "phenotype": "Plateletcrit", "index": 18, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 6.9434, "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Ratio": "0.0253 (0.0354)", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO35", "phenotype": "Red cell distribution width", "index": 19, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 31.404372, "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "index": 20, "PMID or link": "27863252", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 9.816275, "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Ratio": "0.0197 (0.0653)", "study size": 173480, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "EDUC03", "phenotype": "Educational Attainment: EduYears", "index": 21, "PMID or link": "30038396", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 12.131121, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 2.6521, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 766345, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "FATM04", "phenotype": "Fat mass", "index": 22, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.079788, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Ratio": "0.0277 (0.0105)", "study size": 353972, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG02", "phenotype": "Extreme Height", "index": 23, "PMID or link": "23563607", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 10.269605, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2268, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 16196, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG03", "phenotype": "Height 2014", "index": 24, "PMID or link": "25282103", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 38.052733, "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Ratio": "0.1178 (0.0152)", "study size": 253288, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04F", "phenotype": "Height female 2013", "index": 25, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 14.255864, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1891, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 73137, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HEIG04M", "phenotype": "Height male 2013", "index": 26, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 9.98527, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1464, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60586, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01M", "phenotype": "Hip circumference European male", "index": 27, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 10.03375, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0853, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8715 (0.009)", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57568, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC01", "phenotype": "Hip circumference European", "index": 28, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 13.819501, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145308, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02M", "phenotype": "Hip circumference All Ancestries male", "index": 29, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.685038, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0839, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8692 (0.009)", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 57598, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC02", "phenotype": "Hip circumference All Ancestries", "index": 30, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 10.396077, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 145414, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03F", "phenotype": "Hip circumference European female BMI-adjusted", "index": 31, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 14.849704000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1134, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86792, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03M", "phenotype": "Hip circumference European male BMI-adjusted", "index": 32, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.112872, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1064, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56939, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "index": 33, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 14.683464, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143579, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04F", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "index": 34, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 16.868189, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1104, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 86647, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04M", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "index": 35, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 11.613858, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1051, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 56978, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "index": 36, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 25.122467, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 143575, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "IRON03", "phenotype": "Transferrin saturation", "index": 37, "PMID or link": "25352340", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 11.871568, "category": "a: metabolic", "LDSC Mean Chi-Square": 1.0484, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "LDSC Lambda GC": 1.0436, "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Ratio": "0.6579 (0.2019)", "study size": 48972, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LEAN06", "phenotype": "Fat free mass / lean mass", "index": 38, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 16.282871, "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "ENSEMBL": "ENSG00000196866", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Ratio": "0.0513 (0.0128)", "study size": 353972, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "LUPU01", "phenotype": "Systemic Lupus Erythematosus", "index": 39, "PMID or link": "26502338", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.068217, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.2181, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Ratio": "0.4868 (0.0447)", "study size": 23210, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "PRSC01", "phenotype": "Primary sclerosing cholangitis", "index": 40, "PMID or link": "27992413", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 38.354607, "category": "e: autoimmune", "LDSC Mean Chi-Square": 1.0872, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Ratio": "0.0664 (0.1142)", "study size": 24751, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "RHEU02", "phenotype": "Rheumatoid Arthritis 2014 European", "index": 41, "PMID or link": "24390342", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "association": 7.5397490000000005, "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1164, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.9414 (0.0119)", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 58284, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "index": 42, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 11.805458, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Ratio": "0.0437 (0.0138)", "study size": 82315, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "SCHI02", "phenotype": "Schizophrenia PGC 2014 European", "index": 43, "PMID or link": "25056061", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 12.115505, "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Ratio": "0.0515 (0.0139)", "study size": 77096, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS01", "phenotype": "Waist circumference EU", "index": 44, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 8.190252000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7423 (0.0086)", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 231927, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS02", "phenotype": "Waist circumference all ancestries", "index": 45, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 7.679231, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.7406 (0.0087)", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 244261, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "index": 46, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 5.692654, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8926 (0.0084)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 91952, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03M", "phenotype": "Waist circumference EU male BMI-adjusted", "index": 47, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 7.783200999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0873, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 60393, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "index": 48, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 10.924745, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 229089, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "index": 49, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 6.822377, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8907 (0.0084)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 132715, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04M", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "index": 50, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 7.892180000000001, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.084, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 109077, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WAIS04", "phenotype": "Waist circumference all ancestries BMI-adjusted", "index": 51, "PMID or link": "25673412", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 12.162272999999999, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1420000000000001, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "LDSC Lambda GC": 1.0466, "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 153478, "variable": "Vagina", "value": 0.0}, {"gene": "HIST1H2AD", "code": "WEIG04F", "phenotype": "Weight European female", "index": 52, "PMID or link": "23754948", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 9.920276, "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0711, "ENSEMBL": "ENSG00000196866", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "LDSC Lambda GC": 0.9986, "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "study size": 67592, "variable": "Vagina", "value": 0.0}]}};
    var embed_opt = {"mode": "vega-lite"};

    function showError(el, error){
        el.innerHTML = ('<div class="error">'
                        + '<p>JavaScript Error: ' + error.message + '</p>'
                        + "<p>This usually means there's a typo in your chart specification. "
                        + "See the javascript console for the full traceback.</p>"
                        + '</div>');
        throw error;
    }
    const el = document.getElementById('vis');
    vegaEmbed("#vis", spec, embed_opt)
      .catch(error => showError(el, error));
  </script>
</body>
</html>
