
<!DOCTYPE html>

<html>
<head><style>.vega-actions a {margin-right: 12px;color: #757575;font-weight: normal;font-size: 13px;}.error {color: red;}</style><script src="https://cdn.jsdelivr.net/npm//vega@4" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-lite@2.6.0" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-embed@3" type="text/javascript"></script><meta content="text/html; charset=utf-8" http-equiv="Content-Type"/><meta content="IE=edge" http-equiv="X-UA-Compatible"/><meta content="width=device-width, initial-scale=1" name="viewport"/><link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/Font-Awesome-4.7/css/font-awesome.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap-theme.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/images/gnomic_icon.png" rel="icon" type="image/png"/><script src="https://phenviz.navigome.com/js/jquery.min.js"></script><script src="https://phenviz.navigome.com/js/bootstrap.js"></script><title>Gene profile: ENSG00000173327</title></head>
<body>
<h4>Gene profile across phenotypes - ENSG00000173327 (MAP3K11)</h4><div id="vis"></div><div class="col-lg-12 col-md-12 content-item content-item-1"><p>This is a visualization of GWAS-derived gene associations  across Navigome phenotypes. Only phenotypes for which the gene has at least one significant association (p-value â‰¤ 0.05/20000) in <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> <b>and/or</b> <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a> analyses are shown. The first panel is a visualization of gene-phenotype associations obtained using <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> (-log10(p-value)). Hover over any bar to highlight the phenotype in the other panels. The other panels are parallel coordinates visualizations for tissue-wise associations (z-scores)computed using <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a>  and eQTL expression data from GTEx and DGN. Significance thresholds are delineated with black lines. The parallel coordinates show which phenotypes have a significant association and in which tissues, based on gene expression data. Clicking on a specific phenotype will lead to the reference of the GWAS study.</p></div>
<script type="text/javascript">
    var spec = {"config": {"view": {"width": 400, "height": 300}}, "vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "bar", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "href": {"type": "nominal", "field": "phenotype_link"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "quantitative", "axis": {"title": "MAGMA -log10(p)"}, "field": "value"}, "y": {"type": "nominal", "axis": {"title": ""}, "field": "phenotype_reference"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-4e93ee1a91df87c61b7d73e211f0f4d1"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "x": {"type": "quantitative", "field": "ThresholdVal"}}}], "title": "MAGMA gene-wise association"}], "center": true, "data": {"name": "data-26d6da6606617c8493fda5efade179ad"}}, {"vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Nervous system", "width": 500}], "center": true, "data": {"name": "data-0739a4fdcd3c8b17854265aa7cacc7c4"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Digestive System", "width": 500}], "center": true, "data": {"name": "data-6b7aa6418e2d9b93a7344d65a5edbaa6"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Blood, Arteries and Heart", "width": 500}], "center": true, "data": {"name": "data-038b985599417d30754fe6b4d1ec9b84"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Adipose Tissue, Muscle, Skin and Cells", "width": 500}], "center": true, "data": {"name": "data-cd0b419a94cb49abe6b9685aace0e0d9"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.43527848142, 9.43527848142]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Endocrine System and Others", "width": 500}], "center": true, "data": {"name": "data-4a8205db45783061b94291660645b8ff"}}]}], "spacing": 40, "$schema": "https://vega.github.io/schema/vega-lite/v2.6.0.json", "datasets": {"data-26d6da6606617c8493fda5efade179ad": [{"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "magma_log10p", "value": 8.447514}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "magma_log10p", "value": 20.851428}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "magma_log10p", "value": 7.797294}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "magma_log10p", "value": 20.347096}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "magma_log10p", "value": 17.83062}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "magma_log10p", "value": 14.000226000000001}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "magma_log10p", "value": 12.464769}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "magma_log10p", "value": 16.775467000000003}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "magma_log10p", "value": 16.870342}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "magma_log10p", "value": 19.012204}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "magma_log10p", "value": 7.324781}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "magma_log10p", "value": 13.211231}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "magma_log10p", "value": 9.935617}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "magma_log10p", "value": 18.411604999999998}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "magma_log10p", "value": 12.113887}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "magma_log10p", "value": 12.270275999999999}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "magma_log10p", "value": 16.524765}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "magma_log10p", "value": 6.218216}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "magma_log10p", "value": 7.2999279999999995}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "magma_log10p", "value": 7.004430999999999}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "magma_log10p", "value": 3.4437699999999998}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "magma_log10p", "value": 5.8839239999999995}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "magma_log10p", "value": 6.661982000000001}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "magma_log10p", "value": 7.745863000000001}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "magma_log10p", "value": 8.253810000000001}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "magma_log10p", "value": 6.158615}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "magma_log10p", "value": 18.124187}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "magma_log10p", "value": 6.045251}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "magma_log10p", "value": 5.110525}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "magma_log10p", "value": 6.4675769999999995}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "magma_log10p", "value": 5.535332}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "magma_log10p", "value": 4.0124629999999994}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "magma_log10p", "value": 3.5163839999999995}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "magma_log10p", "value": 4.326371}], "data-4e93ee1a91df87c61b7d73e211f0f4d1": [{"Threshold": "MAGMA -log10(p) \u2265 5.60206", "ThresholdVal": 5.60206}], "data-0739a4fdcd3c8b17854265aa7cacc7c4": [{"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Brain_Amygdala", "value": 0.0}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Brain_Caudate", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Brain_Caudate", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Brain_Caudate", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Brain_Caudate", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Brain_Caudate", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Brain_Caudate", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Brain_Caudate", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Brain_Caudate", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Brain_Caudate", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Brain_Caudate", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Brain_Caudate", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Brain_Caudate", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Brain_Caudate", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Brain_Caudate", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Brain_Caudate", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Brain_Caudate", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Brain_Caudate", "value": 0.0}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Brain_Putamen", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Brain_Putamen", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Brain_Putamen", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Brain_Putamen", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Brain_Putamen", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Brain_Putamen", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Brain_Putamen", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Brain_Putamen", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Brain_Putamen", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Brain_Putamen", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Brain_Putamen", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Brain_Putamen", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Brain_Putamen", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Brain_Putamen", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Brain_Putamen", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Brain_Putamen", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Brain_Putamen", "value": 0.0}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Brain_Cerebellum", "value": 4.12342847575}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Brain_Cerebellum", "value": 6.1923032822}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Brain_Cerebellum", "value": -0.25521814081599997}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Brain_Cerebellum", "value": 5.30803745969}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Brain_Cerebellum", "value": 5.05682772033}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Brain_Cerebellum", "value": 5.046950926969999}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Brain_Cerebellum", "value": 4.643529223380001}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Brain_Cerebellum", "value": 0.326374568149}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Brain_Cerebellum", "value": 0.273870201521}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Brain_Cerebellum", "value": 1.30357033469}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Brain_Cerebellum", "value": 1.88863323778}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Brain_Cerebellum", "value": 1.23958739846}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Brain_Cerebellum", "value": 0.844721636963}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Brain_Cerebellum", "value": -9.11049178566}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Brain_Cerebellum", "value": -4.57132205547}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Brain_Cerebellum", "value": 2.2103719582}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Brain_Cerebellum", "value": -5.40565639664}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Brain_Cerebellum", "value": -0.33656060387599995}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Brain_Cerebellum", "value": -1.90825900444}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Brain_Cerebellum", "value": -0.658890043772}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Brain_Cerebellum", "value": 3.15578842163}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Brain_Cerebellum", "value": 3.86474728584}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Brain_Cerebellum", "value": 3.76836538315}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Brain_Cerebellum", "value": -0.299444830461}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Brain_Cerebellum", "value": -0.395365328342}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Brain_Cerebellum", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Brain_Cerebellum", "value": 5.0741609505500005}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Brain_Cerebellum", "value": -2.9244801277599994}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Brain_Cerebellum", "value": 4.9553008079500005}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Brain_Cerebellum", "value": -4.4847483634900005}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Brain_Cerebellum", "value": -3.54008388519}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Brain_Cerebellum", "value": 4.4226379394499995}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Brain_Cerebellum", "value": 4.27613687515}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Brain_Cerebellum", "value": 4.35850000381}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Brain_Cortex", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Brain_Cortex", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Brain_Cortex", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Brain_Cortex", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Brain_Cortex", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Brain_Cortex", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Brain_Cortex", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Brain_Cortex", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Brain_Cortex", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Brain_Cortex", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Brain_Cortex", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Brain_Cortex", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Brain_Cortex", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Brain_Cortex", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Brain_Cortex", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Brain_Cortex", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Brain_Cortex", "value": 0.0}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Brain_Hippocampus", "value": 0.0}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Brain_Hypothalamus", "value": 3.6137042189999997}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Brain_Hypothalamus", "value": 4.552950078319999}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Brain_Hypothalamus", "value": -0.7963694069930001}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Brain_Hypothalamus", "value": 5.5310938171099995}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Brain_Hypothalamus", "value": 5.05250469625}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Brain_Hypothalamus", "value": 5.34329681895}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Brain_Hypothalamus", "value": 4.80560160065}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Brain_Hypothalamus", "value": -0.702600650632}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Brain_Hypothalamus", "value": -0.7466881006389999}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Brain_Hypothalamus", "value": -0.026993302031400004}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Brain_Hypothalamus", "value": 2.32926304018}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Brain_Hypothalamus", "value": 0.777708046215}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Brain_Hypothalamus", "value": 0.162449489054}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Brain_Hypothalamus", "value": -7.065170579510001}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Brain_Hypothalamus", "value": -4.56570617302}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Brain_Hypothalamus", "value": 1.4520269913}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Brain_Hypothalamus", "value": -4.74653201619}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Brain_Hypothalamus", "value": -0.841073720234}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Brain_Hypothalamus", "value": -2.62244382162}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Brain_Hypothalamus", "value": -2.1365393801400003}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Brain_Hypothalamus", "value": 3.71650128866}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Brain_Hypothalamus", "value": 3.78329911435}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Brain_Hypothalamus", "value": 3.69229824922}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Brain_Hypothalamus", "value": 0.0036026994994399998}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Brain_Hypothalamus", "value": 0.139125243822}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Brain_Hypothalamus", "value": 2.5776168554599996}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Brain_Hypothalamus", "value": 4.176970324680001}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Brain_Hypothalamus", "value": -1.9877235315200001}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Brain_Hypothalamus", "value": 4.32322772666}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Brain_Hypothalamus", "value": -5.21248597824}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Brain_Hypothalamus", "value": -4.369744975490001}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Brain_Hypothalamus", "value": 4.39694038083}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Brain_Hypothalamus", "value": 4.26346079206}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Brain_Hypothalamus", "value": 4.330341414009999}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Nerve_Tibial", "value": 4.21071706761}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Nerve_Tibial", "value": 6.07788378116}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Nerve_Tibial", "value": -0.257881887524}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Nerve_Tibial", "value": 5.19130717119}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Nerve_Tibial", "value": 5.0020564723900005}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Nerve_Tibial", "value": 4.95234361016}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Nerve_Tibial", "value": 4.54109174987}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Nerve_Tibial", "value": 0.29597939249099997}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Nerve_Tibial", "value": 0.23902940755899998}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Nerve_Tibial", "value": 1.22289657729}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Nerve_Tibial", "value": 1.9582098541900004}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Nerve_Tibial", "value": 1.43946404732}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Nerve_Tibial", "value": 0.981948024352}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Nerve_Tibial", "value": -9.02784417296}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Nerve_Tibial", "value": -4.526405070369999}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Nerve_Tibial", "value": 2.14375550957}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Nerve_Tibial", "value": -5.28354813606}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Nerve_Tibial", "value": -0.334396365297}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Nerve_Tibial", "value": -1.5548862913600001}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Nerve_Tibial", "value": -0.609300650287}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Nerve_Tibial", "value": 3.38606123287}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Nerve_Tibial", "value": 4.02166952727}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Nerve_Tibial", "value": 3.9213742358499997}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Nerve_Tibial", "value": -0.140245864334}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Nerve_Tibial", "value": -0.5125694189509999}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Nerve_Tibial", "value": 2.59018131343}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Nerve_Tibial", "value": 5.260073631}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Nerve_Tibial", "value": -2.7883426762700005}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Nerve_Tibial", "value": 4.658455771040001}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Nerve_Tibial", "value": -4.936481945290001}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Nerve_Tibial", "value": -4.15480774326}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Nerve_Tibial", "value": 4.88242844649}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Nerve_Tibial", "value": 4.634121015390001}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Nerve_Tibial", "value": 4.8116225256}], "data-c4c9a654d45741ebc60aa567761bff19": [{"Text": "Significance Threshold", "Threshold": "PrediXcan zscore \u2265 4.70813", "ThresholdVal": 4.70813}], "data-4a3aa5bf8c5a41879455008930c25774": [{"Threshold": "PrediXcan zscore \u2264 -4.70813", "ThresholdVal": -4.70813}], "data-6b7aa6418e2d9b93a7344d65a5edbaa6": [{"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Colon_Sigmoid", "value": 4.18024929344}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Colon_Sigmoid", "value": 6.421366480930001}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Colon_Sigmoid", "value": -1.72345796707}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Colon_Sigmoid", "value": 5.871419737159999}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Colon_Sigmoid", "value": 5.51911760275}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Colon_Sigmoid", "value": 5.62516549518}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Colon_Sigmoid", "value": 5.12405395494}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Colon_Sigmoid", "value": -1.45630231014}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Colon_Sigmoid", "value": -1.6006238309799998}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Colon_Sigmoid", "value": -0.434115066025}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Colon_Sigmoid", "value": 2.3330317941900005}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Colon_Sigmoid", "value": 2.4776270494}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Colon_Sigmoid", "value": 2.0027124045099995}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Colon_Sigmoid", "value": -8.954151759550001}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Colon_Sigmoid", "value": -4.98378668695}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Colon_Sigmoid", "value": 2.67589157283}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Colon_Sigmoid", "value": -4.7808106209500005}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Colon_Sigmoid", "value": -1.6955277169799998}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Colon_Sigmoid", "value": -1.9955861450799999}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Colon_Sigmoid", "value": -2.20981849599}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Colon_Sigmoid", "value": 3.9597764787300003}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Colon_Sigmoid", "value": 4.85839185939}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Colon_Sigmoid", "value": 4.7266511523800006}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Colon_Sigmoid", "value": 0.120958406302}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Colon_Sigmoid", "value": 0.701616519292}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Colon_Sigmoid", "value": 1.42819440757}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Colon_Sigmoid", "value": 4.0390073309}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Colon_Sigmoid", "value": -3.69703943011}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Colon_Sigmoid", "value": 5.48850291496}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Colon_Sigmoid", "value": -5.467842789450001}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Colon_Sigmoid", "value": -4.83738868835}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Colon_Sigmoid", "value": 4.64086235044}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Colon_Sigmoid", "value": 4.3411075038}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Colon_Sigmoid", "value": 4.55800879741}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Colon_Transverse", "value": 4.04944016129}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Colon_Transverse", "value": 6.130179715090001}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Colon_Transverse", "value": -0.498504609155}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Colon_Transverse", "value": 5.16949264663}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Colon_Transverse", "value": 4.9319737906}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Colon_Transverse", "value": 4.9450657298400005}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Colon_Transverse", "value": 4.56329882411}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Colon_Transverse", "value": 0.0460433908524}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Colon_Transverse", "value": -0.000202742976271}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Colon_Transverse", "value": 1.09643312013}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Colon_Transverse", "value": 1.85787017315}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Colon_Transverse", "value": 1.2529289562600001}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Colon_Transverse", "value": 0.830700172688}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Colon_Transverse", "value": -9.0698702878}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Colon_Transverse", "value": -4.53735585431}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Colon_Transverse", "value": 2.1335196098900004}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Colon_Transverse", "value": -5.3992192238}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Colon_Transverse", "value": -0.601958591548}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Colon_Transverse", "value": -1.83680814371}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Colon_Transverse", "value": -0.9920619007170001}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Colon_Transverse", "value": 3.47827082802}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Colon_Transverse", "value": 4.222059736309999}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Colon_Transverse", "value": 4.07428025703}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Colon_Transverse", "value": -0.0867238806092}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Colon_Transverse", "value": -0.284239013302}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Colon_Transverse", "value": 2.4149043010900004}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Colon_Transverse", "value": 5.21445926151}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Colon_Transverse", "value": -2.96086181555}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Colon_Transverse", "value": 5.0953388010500005}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Colon_Transverse", "value": -4.84476247847}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Colon_Transverse", "value": -4.0401103999400005}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Colon_Transverse", "value": 4.6380627391400004}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Colon_Transverse", "value": 4.42823528439}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Colon_Transverse", "value": 4.5748295851}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.200353964880001}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Esophagus_Gastroesophageal_Junction", "value": 6.210500765140001}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.5776607090420001}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Esophagus_Gastroesophageal_Junction", "value": 6.00190037806}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Esophagus_Gastroesophageal_Junction", "value": 5.759279530030001}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Esophagus_Gastroesophageal_Junction", "value": 5.51653773419}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Esophagus_Gastroesophageal_Junction", "value": 5.08989040252}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.058683761913199996}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0044087835079799995}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.13394118614}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.97018582183}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.44338656753}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.963420777756}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Esophagus_Gastroesophageal_Junction", "value": -9.2541836437}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Esophagus_Gastroesophageal_Junction", "value": -5.02221905044}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.1683244366}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Esophagus_Gastroesophageal_Junction", "value": -5.36412620627}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.639125429272}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Esophagus_Gastroesophageal_Junction", "value": -1.5170815488499998}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.8875942925920001}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Esophagus_Gastroesophageal_Junction", "value": 3.9305708204900003}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.09053728427}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Esophagus_Gastroesophageal_Junction", "value": 3.98589849189}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.324409258365}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.422279507557}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.59437415683}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Esophagus_Gastroesophageal_Junction", "value": 5.18169574341}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Esophagus_Gastroesophageal_Junction", "value": -2.64514839958}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Esophagus_Gastroesophageal_Junction", "value": 6.5215710287199995}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Esophagus_Gastroesophageal_Junction", "value": -5.29047145822}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Esophagus_Gastroesophageal_Junction", "value": -4.421040655990001}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.83668278264}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.73853140465}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.766540301509999}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Esophagus_Mucosa", "value": 4.4539707676199995}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Esophagus_Mucosa", "value": 6.44087858896}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Esophagus_Mucosa", "value": -0.693127243109}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Esophagus_Mucosa", "value": 6.174257018}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Esophagus_Mucosa", "value": 6.17195060263}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Esophagus_Mucosa", "value": 5.23599487549}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Esophagus_Mucosa", "value": 4.72018586436}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Esophagus_Mucosa", "value": -0.322102189908}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Esophagus_Mucosa", "value": -0.406597763816}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Esophagus_Mucosa", "value": 0.246384877351}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Esophagus_Mucosa", "value": 1.87831097968}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Esophagus_Mucosa", "value": 1.5941244809200001}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Esophagus_Mucosa", "value": 1.14694646499}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Esophagus_Mucosa", "value": -8.7780151384}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Esophagus_Mucosa", "value": -4.7073502211100005}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Esophagus_Mucosa", "value": 2.83666138069}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Esophagus_Mucosa", "value": -4.05064789811}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Esophagus_Mucosa", "value": -0.739868539674}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Esophagus_Mucosa", "value": -1.1545774356799998}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Esophagus_Mucosa", "value": -1.9023116717400002}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Esophagus_Mucosa", "value": 3.7682596552699996}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Esophagus_Mucosa", "value": 4.36892072526}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Esophagus_Mucosa", "value": 4.23402307682}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Esophagus_Mucosa", "value": 0.137991066007}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Esophagus_Mucosa", "value": -0.199717744226}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Esophagus_Mucosa", "value": 1.90328276143}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Esophagus_Mucosa", "value": 3.67093261446}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Esophagus_Mucosa", "value": -2.9728847482}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Esophagus_Mucosa", "value": 5.298735592330001}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Esophagus_Mucosa", "value": -5.5644862649}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Esophagus_Mucosa", "value": -4.74472225786}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Esophagus_Mucosa", "value": 3.0758042319}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Esophagus_Mucosa", "value": 3.19113715947}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Esophagus_Mucosa", "value": 2.9954350394900002}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Esophagus_Muscularis", "value": 4.1516837108099995}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Esophagus_Muscularis", "value": 6.23135046345}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Esophagus_Muscularis", "value": -0.547641972692}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Esophagus_Muscularis", "value": 6.0341126460300005}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Esophagus_Muscularis", "value": 5.83754683589}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Esophagus_Muscularis", "value": 5.6119629259099995}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Esophagus_Muscularis", "value": 5.25343084524}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Esophagus_Muscularis", "value": -0.202470217826}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Esophagus_Muscularis", "value": -0.228376586873}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Esophagus_Muscularis", "value": 0.768572636427}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Esophagus_Muscularis", "value": 1.6539578838100002}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Esophagus_Muscularis", "value": 1.0601977430299998}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Esophagus_Muscularis", "value": 0.73909570826}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Esophagus_Muscularis", "value": -9.43527848142}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Esophagus_Muscularis", "value": -5.23400008203}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Esophagus_Muscularis", "value": 2.0407506839}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Esophagus_Muscularis", "value": -5.25806102515}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Esophagus_Muscularis", "value": -0.659182333341}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Esophagus_Muscularis", "value": -2.25951110978}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Esophagus_Muscularis", "value": -1.11179692014}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Esophagus_Muscularis", "value": 3.4284010888}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Esophagus_Muscularis", "value": 3.7840565760000002}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Esophagus_Muscularis", "value": 3.68968702492}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Esophagus_Muscularis", "value": -0.24296685548}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Esophagus_Muscularis", "value": -0.010696372731899998}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Esophagus_Muscularis", "value": 1.9231799893700001}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Esophagus_Muscularis", "value": 4.82152353844}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Esophagus_Muscularis", "value": -3.2982576255500002}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Esophagus_Muscularis", "value": 5.539650913380001}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Esophagus_Muscularis", "value": -5.02325556746}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Esophagus_Muscularis", "value": -4.007923895899999}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Esophagus_Muscularis", "value": 4.4819507748500005}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Esophagus_Muscularis", "value": 4.23619267752}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Esophagus_Muscularis", "value": 4.41695270387}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Liver", "value": 1.0992658960600001}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Liver", "value": 0.341162737501}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Liver", "value": 2.6875962320399998}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Liver", "value": 1.2513781141}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Liver", "value": 1.18820204365}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Liver", "value": 1.0919274527200002}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Liver", "value": 0.959708801889}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Liver", "value": 3.33173308314}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Liver", "value": 3.4227477561}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Liver", "value": 2.7028179406200006}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Liver", "value": 0.787516995018}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Liver", "value": -0.656154492401}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Liver", "value": -1.08677980066}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Liver", "value": -2.90258976936}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Liver", "value": -0.897953992097}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Liver", "value": -0.8778106090180001}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Liver", "value": -2.9947551489}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Liver", "value": 2.57504405912}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Liver", "value": 2.1764012835599997}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Liver", "value": 1.5580735767899998}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Liver", "value": 2.0833805095}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Liver", "value": 0.8188029545469999}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Liver", "value": 0.7661372054100001}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Liver", "value": 1.99785791052}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Liver", "value": -1.2757067844399999}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Liver", "value": 0.720701236065}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Liver", "value": 3.02876691695}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Liver", "value": 0.103047200976}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Liver", "value": -0.39042153345700004}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Liver", "value": -2.43037139753}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Liver", "value": -2.6280205263}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Liver", "value": 2.69456267625}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Liver", "value": 2.01737458433}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Liver", "value": 2.63826607853}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Small_Intestine_Terminal_Ileum", "value": 5.11849966383}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Small_Intestine_Terminal_Ileum", "value": 5.405659898840001}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Small_Intestine_Terminal_Ileum", "value": -0.5586987303059999}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Small_Intestine_Terminal_Ileum", "value": 6.023017216919999}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Small_Intestine_Terminal_Ileum", "value": 5.87794198888}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Small_Intestine_Terminal_Ileum", "value": 5.624967848300001}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Small_Intestine_Terminal_Ileum", "value": 5.10402398774}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.286559229441}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.20930730649999998}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Small_Intestine_Terminal_Ileum", "value": 1.53901119347}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Small_Intestine_Terminal_Ileum", "value": 2.40761799971}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Small_Intestine_Terminal_Ileum", "value": 1.57024803935}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.955321784623}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Small_Intestine_Terminal_Ileum", "value": -8.72763354745}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Small_Intestine_Terminal_Ileum", "value": -5.097694580980001}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Small_Intestine_Terminal_Ileum", "value": 1.56504592081}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Small_Intestine_Terminal_Ileum", "value": -5.17660343872}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Small_Intestine_Terminal_Ileum", "value": -0.559825631716}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Small_Intestine_Terminal_Ileum", "value": -1.12055206475}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Small_Intestine_Terminal_Ileum", "value": -0.763768823233}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Small_Intestine_Terminal_Ileum", "value": 4.52494465816}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Small_Intestine_Terminal_Ileum", "value": 3.7049050969599997}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Small_Intestine_Terminal_Ileum", "value": 3.61250956507}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.519209527082}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Small_Intestine_Terminal_Ileum", "value": -0.748425884857}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Small_Intestine_Terminal_Ileum", "value": 2.9434286082}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Small_Intestine_Terminal_Ileum", "value": 5.9055772414900005}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Small_Intestine_Terminal_Ileum", "value": -2.22755446461}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Small_Intestine_Terminal_Ileum", "value": 6.34255714073}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Small_Intestine_Terminal_Ileum", "value": -5.52030078262}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Small_Intestine_Terminal_Ileum", "value": -4.72456374118}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Small_Intestine_Terminal_Ileum", "value": 5.06756684871}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Small_Intestine_Terminal_Ileum", "value": 4.78362044173}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Small_Intestine_Terminal_Ileum", "value": 4.99407594946}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Stomach", "value": 4.02238714922}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Stomach", "value": 5.9904072705699996}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Stomach", "value": -1.54690405026}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Stomach", "value": 7.53541743104}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Stomach", "value": 7.046062577610001}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Stomach", "value": 6.622372835669999}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Stomach", "value": 6.15523676997}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Stomach", "value": -0.914684461269}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Stomach", "value": -1.2007742004700002}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Stomach", "value": 0.09722901116310001}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Stomach", "value": 1.7665731031999998}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Stomach", "value": 2.40848408265}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Stomach", "value": 2.10300677572}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Stomach", "value": -7.585404864299999}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Stomach", "value": -5.8042391788800005}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Stomach", "value": 2.81838367366}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Stomach", "value": -5.52501429242}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Stomach", "value": -1.29575561649}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Stomach", "value": -3.3357279157999997}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Stomach", "value": -2.8375784986699997}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Stomach", "value": 4.98878742444}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Stomach", "value": 4.8023174097000005}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Stomach", "value": 4.68255384361}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Stomach", "value": -1.11761967528}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Stomach", "value": 1.0078047805299999}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Stomach", "value": 1.87639319621}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Stomach", "value": 4.40468417908}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Stomach", "value": -3.58266338275}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Stomach", "value": 3.5476377224199998}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Stomach", "value": -4.890828083590001}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Stomach", "value": -4.63528963187}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Stomach", "value": 4.50832938314}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Stomach", "value": 4.7090322698}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Stomach", "value": 4.442948632869999}], "data-038b985599417d30754fe6b4d1ec9b84": [{"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "DGN_Whole_Blood", "value": 4.274541083100628}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "DGN_Whole_Blood", "value": 6.107076433405672}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "DGN_Whole_Blood", "value": -0.22954771486097875}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "DGN_Whole_Blood", "value": 5.2130998616904005}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "DGN_Whole_Blood", "value": 4.950086071157597}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "DGN_Whole_Blood", "value": 4.938905285303877}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "DGN_Whole_Blood", "value": 4.554842820448618}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "DGN_Whole_Blood", "value": 0.5020928224766783}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "DGN_Whole_Blood", "value": 0.40963719756007616}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "DGN_Whole_Blood", "value": 1.5375510314104786}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "DGN_Whole_Blood", "value": 1.9357739632608808}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "DGN_Whole_Blood", "value": 1.3161263880725451}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "DGN_Whole_Blood", "value": 0.8901503078626017}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "DGN_Whole_Blood", "value": -9.134708726280783}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "DGN_Whole_Blood", "value": -4.4831713059765885}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "DGN_Whole_Blood", "value": 2.093408685784756}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "DGN_Whole_Blood", "value": -5.628402191903441}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "DGN_Whole_Blood", "value": -0.24955166607707954}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "DGN_Whole_Blood", "value": -2.1311645457182844}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "DGN_Whole_Blood", "value": -0.6470041727843596}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "DGN_Whole_Blood", "value": 3.702450627770839}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "DGN_Whole_Blood", "value": 4.1120647490897415}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "DGN_Whole_Blood", "value": 4.0093278404666695}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "DGN_Whole_Blood", "value": -0.1388438205444636}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "DGN_Whole_Blood", "value": -0.5403413606455414}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "DGN_Whole_Blood", "value": 1.0592222765239114}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "DGN_Whole_Blood", "value": 5.193652702375197}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "DGN_Whole_Blood", "value": -3.0929750667801708}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "DGN_Whole_Blood", "value": 5.103773573667258}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "DGN_Whole_Blood", "value": -4.76800715687854}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "DGN_Whole_Blood", "value": -3.8360722514490297}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "DGN_Whole_Blood", "value": 4.504812050986568}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "DGN_Whole_Blood", "value": 4.38433389612785}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "DGN_Whole_Blood", "value": 4.439942177704596}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Whole_Blood", "value": 4.44986055191}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Whole_Blood", "value": 5.86295792181}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Whole_Blood", "value": 0.0973481645246}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Whole_Blood", "value": 5.4135856561}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Whole_Blood", "value": 5.16188989257}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Whole_Blood", "value": 5.105495524659999}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Whole_Blood", "value": 4.7099955928699995}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Whole_Blood", "value": 0.839198158332}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Whole_Blood", "value": 0.7755241900290001}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Whole_Blood", "value": 1.83233247465}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Whole_Blood", "value": 1.912210039}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Whole_Blood", "value": 1.39382025706}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Whole_Blood", "value": 0.950905596768}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Whole_Blood", "value": -9.009125664179999}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Whole_Blood", "value": -4.633863676630001}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Whole_Blood", "value": 1.9139453363999999}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Whole_Blood", "value": -5.38552307676}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Whole_Blood", "value": 0.0270374735763}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Whole_Blood", "value": -1.66765100076}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Whole_Blood", "value": -0.39175875790700004}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Whole_Blood", "value": 2.30313471656}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Whole_Blood", "value": 3.1860579732}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Whole_Blood", "value": 3.1066017602}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Whole_Blood", "value": -0.51626245812}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Whole_Blood", "value": -0.47554224517899996}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Whole_Blood", "value": 1.22589697651}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Whole_Blood", "value": 5.28096778831}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Whole_Blood", "value": -2.76785368875}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Whole_Blood", "value": 5.34039149702}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Whole_Blood", "value": -4.7540199112}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Whole_Blood", "value": -3.8355152459599995}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Whole_Blood", "value": 4.13737826879}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Whole_Blood", "value": 3.94474312247}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Whole_Blood", "value": 4.07737714859}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Artery_Aorta", "value": 4.61988230187}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Artery_Aorta", "value": 6.05676548998}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Artery_Aorta", "value": -0.308458309204}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Artery_Aorta", "value": 7.60863418753}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Artery_Aorta", "value": 7.3415841972}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Artery_Aorta", "value": 6.892861669760001}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Artery_Aorta", "value": 6.4946748340100005}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Artery_Aorta", "value": 0.38731609532}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Artery_Aorta", "value": 0.261720523137}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Artery_Aorta", "value": 1.3618534724899998}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Artery_Aorta", "value": 2.0231738778}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Artery_Aorta", "value": 1.00496633957}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Artery_Aorta", "value": 0.623043649756}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Artery_Aorta", "value": -8.60486240365}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Artery_Aorta", "value": -6.29251197451}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Artery_Aorta", "value": 2.17489830943}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Artery_Aorta", "value": -5.491935327209999}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Artery_Aorta", "value": -0.261685974257}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Artery_Aorta", "value": -3.35892590481}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Artery_Aorta", "value": -1.01765028192}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Artery_Aorta", "value": 3.75499637626}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Artery_Aorta", "value": 3.2569487083599995}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Artery_Aorta", "value": 3.17607198243}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Artery_Aorta", "value": -0.950713147326}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Artery_Aorta", "value": 0.112381785254}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Artery_Aorta", "value": 2.4683299286400002}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Artery_Aorta", "value": 4.36375172939}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Artery_Aorta", "value": -2.56593624429}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Artery_Aorta", "value": 5.09693135446}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Artery_Aorta", "value": -5.55018791381}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Artery_Aorta", "value": -4.7047719284500005}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Artery_Aorta", "value": 4.7126653388}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Artery_Aorta", "value": 4.53363626419}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Artery_Aorta", "value": 4.64064209309}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Artery_Coronary", "value": 2.59942467905}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Artery_Coronary", "value": 5.642076316540001}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Artery_Coronary", "value": -0.47515936420500005}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Artery_Coronary", "value": 3.4681950265099997}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Artery_Coronary", "value": 3.42133479032}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Artery_Coronary", "value": 3.1205656893900002}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Artery_Coronary", "value": 2.8424866283499997}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Artery_Coronary", "value": -0.285955878358}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Artery_Coronary", "value": -0.207718166383}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Artery_Coronary", "value": 0.463287248598}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Artery_Coronary", "value": 1.5634067444400002}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Artery_Coronary", "value": 1.23633877886}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Artery_Coronary", "value": 0.672139732424}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Artery_Coronary", "value": -7.628130069860001}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Artery_Coronary", "value": -2.8436329624299996}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Artery_Coronary", "value": 2.50047858942}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Artery_Coronary", "value": -4.00915404443}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Artery_Coronary", "value": -0.719901974989}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Artery_Coronary", "value": -1.9212922441599998}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Artery_Coronary", "value": -0.513258795032}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Artery_Coronary", "value": 1.9047534046}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Artery_Coronary", "value": 3.51999634578}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Artery_Coronary", "value": 3.38357459764}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Artery_Coronary", "value": -0.19087436849300002}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Artery_Coronary", "value": -0.17379618677499997}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Artery_Coronary", "value": 0.625406337361}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Artery_Coronary", "value": 2.37399993418}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Artery_Coronary", "value": -3.0939503660400005}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Artery_Coronary", "value": 6.9430540913}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Artery_Coronary", "value": -3.5418205291500002}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Artery_Coronary", "value": -2.8359141507200003}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Artery_Coronary", "value": 2.44372260806}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Artery_Coronary", "value": 2.35074672501}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Artery_Coronary", "value": 2.23726933618}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Artery_Tibial", "value": 4.73625133584}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Artery_Tibial", "value": 5.95942925618}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Artery_Tibial", "value": -0.460486245348}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Artery_Tibial", "value": 5.96254342725}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Artery_Tibial", "value": 5.71317386116}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Artery_Tibial", "value": 5.61965832725}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Artery_Tibial", "value": 5.1198502133499995}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Artery_Tibial", "value": 0.304108613393}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Artery_Tibial", "value": 0.250246941923}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Artery_Tibial", "value": 1.5361986463100001}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Artery_Tibial", "value": 2.04939632032}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Artery_Tibial", "value": 1.4644796785699998}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Artery_Tibial", "value": 0.985354167464}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Artery_Tibial", "value": -9.01211815354}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Artery_Tibial", "value": -5.03897109107}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Artery_Tibial", "value": 2.02348517226}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Artery_Tibial", "value": -5.180751719}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Artery_Tibial", "value": -0.536054118356}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Artery_Tibial", "value": -1.37691194324}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Artery_Tibial", "value": -0.9297365315080001}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Artery_Tibial", "value": 3.3047897363200005}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Artery_Tibial", "value": 4.03768107245}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Artery_Tibial", "value": 3.93698648356}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Artery_Tibial", "value": 0.0274672252679}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Artery_Tibial", "value": -0.748209381167}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Artery_Tibial", "value": 1.76355813905}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Artery_Tibial", "value": 5.472927436930001}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Artery_Tibial", "value": -2.6599382687}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Artery_Tibial", "value": 4.8924985536}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Artery_Tibial", "value": -4.85571440878}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Artery_Tibial", "value": -4.0392341508300005}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Artery_Tibial", "value": 4.52474031591}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Artery_Tibial", "value": 4.31075505102}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Artery_Tibial", "value": 4.45912165525}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Heart_Atrial_Appendage", "value": 3.7909872250400003}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Heart_Atrial_Appendage", "value": 6.45570950786}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Heart_Atrial_Appendage", "value": -1.73038044716}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Heart_Atrial_Appendage", "value": 5.85786132891}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Heart_Atrial_Appendage", "value": 5.677796848940001}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Heart_Atrial_Appendage", "value": 5.550682016980001}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Heart_Atrial_Appendage", "value": 5.07775166265}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Heart_Atrial_Appendage", "value": -1.7521557128299998}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Heart_Atrial_Appendage", "value": -1.8322665645400003}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Heart_Atrial_Appendage", "value": -0.795406596938}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Heart_Atrial_Appendage", "value": 1.82447178778}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Heart_Atrial_Appendage", "value": 2.4023147693099998}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Heart_Atrial_Appendage", "value": 1.9244352717400002}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Heart_Atrial_Appendage", "value": -8.69464089615}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Heart_Atrial_Appendage", "value": -5.09686732307}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Heart_Atrial_Appendage", "value": 2.7713392752900003}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Heart_Atrial_Appendage", "value": -4.55311639386}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Heart_Atrial_Appendage", "value": -1.8111956974599999}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Heart_Atrial_Appendage", "value": -0.856534222123}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Heart_Atrial_Appendage", "value": -2.36792229435}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Heart_Atrial_Appendage", "value": 3.8521346419199998}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Heart_Atrial_Appendage", "value": 4.1826861652}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Heart_Atrial_Appendage", "value": 4.07837545138}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Heart_Atrial_Appendage", "value": 0.48582911157799996}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Heart_Atrial_Appendage", "value": 0.371969336196}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Heart_Atrial_Appendage", "value": 1.35450816464}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Heart_Atrial_Appendage", "value": 4.60469557318}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Heart_Atrial_Appendage", "value": -2.91397451128}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Heart_Atrial_Appendage", "value": 4.4773640202300005}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Heart_Atrial_Appendage", "value": -5.07581721459}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Heart_Atrial_Appendage", "value": -4.65869026937}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Heart_Atrial_Appendage", "value": 4.45968068356}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Heart_Atrial_Appendage", "value": 4.10994405883}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Heart_Atrial_Appendage", "value": 4.3950054853400005}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Heart_Left_Ventricle", "value": 2.84190856319}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Heart_Left_Ventricle", "value": 5.3547008602999995}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Heart_Left_Ventricle", "value": -0.33711537056100005}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Heart_Left_Ventricle", "value": 4.03631524013}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Heart_Left_Ventricle", "value": 3.9258666439199996}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Heart_Left_Ventricle", "value": 3.47337341764}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Heart_Left_Ventricle", "value": 3.28315870868}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Heart_Left_Ventricle", "value": 0.174425699093}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Heart_Left_Ventricle", "value": 0.18005133424199998}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Heart_Left_Ventricle", "value": 0.9845811783670001}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Heart_Left_Ventricle", "value": 1.18686565419}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Heart_Left_Ventricle", "value": 0.981087411611}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Heart_Left_Ventricle", "value": 0.73383784352}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Heart_Left_Ventricle", "value": -8.24176179266}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Heart_Left_Ventricle", "value": -3.2787924615799997}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Heart_Left_Ventricle", "value": 1.72892796852}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Heart_Left_Ventricle", "value": -5.035115896640001}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Heart_Left_Ventricle", "value": -0.46440570288800004}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Heart_Left_Ventricle", "value": -1.2541213448000001}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Heart_Left_Ventricle", "value": -0.796170210573}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Heart_Left_Ventricle", "value": 4.49177238979}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Heart_Left_Ventricle", "value": 4.32219817711}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Heart_Left_Ventricle", "value": 4.21440810204}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Heart_Left_Ventricle", "value": 0.96595229937}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Heart_Left_Ventricle", "value": -1.34035035405}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Heart_Left_Ventricle", "value": 2.43772419996}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Heart_Left_Ventricle", "value": 3.7287634016699998}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Heart_Left_Ventricle", "value": -2.29328016786}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Heart_Left_Ventricle", "value": 6.57733613855}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Heart_Left_Ventricle", "value": -4.23983683764}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Heart_Left_Ventricle", "value": -3.4980322349599997}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Heart_Left_Ventricle", "value": 4.31805234148}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Heart_Left_Ventricle", "value": 4.02758271643}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Heart_Left_Ventricle", "value": 4.25543103608}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Spleen", "value": 4.00526107838}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Spleen", "value": 5.94194882472}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Spleen", "value": -0.857022483256}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Spleen", "value": 4.93338234244}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Spleen", "value": 4.77626567747}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Spleen", "value": 4.7176035847}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Spleen", "value": 4.33909359174}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Spleen", "value": -0.328240352342}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Spleen", "value": -0.347485301259}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Spleen", "value": 0.9582413816959999}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Spleen", "value": 1.68249789375}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Spleen", "value": 1.00290526576}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Spleen", "value": 0.623041543495}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Spleen", "value": -8.768585149789999}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Spleen", "value": -4.36576577195}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Spleen", "value": 2.03608935962}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Spleen", "value": -5.188351906369999}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Spleen", "value": -1.0033936550700002}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Spleen", "value": -1.96955755482}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Spleen", "value": -0.657360763072}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Spleen", "value": 3.52039146855}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Spleen", "value": 3.7703280922800007}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Spleen", "value": 3.65464491811}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Spleen", "value": 0.41827654821500004}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Spleen", "value": -0.35403065822499996}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Spleen", "value": 2.0450744542500003}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Spleen", "value": 5.01008914082}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Spleen", "value": -2.70321197059}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Spleen", "value": 4.418970997830001}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Spleen", "value": -4.91352304708}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Spleen", "value": -4.09839813441}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Spleen", "value": 4.27530527236}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Spleen", "value": 3.96001956828}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Spleen", "value": 4.191948454519999}], "data-cd0b419a94cb49abe6b9685aace0e0d9": [{"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Adipose_Subcutaneous", "value": 4.33685214035}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Adipose_Subcutaneous", "value": 5.8580188176600005}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Adipose_Subcutaneous", "value": -0.11941824876599999}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Adipose_Subcutaneous", "value": 5.260957192019999}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Adipose_Subcutaneous", "value": 5.0062758576699995}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Adipose_Subcutaneous", "value": 5.03992615444}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Adipose_Subcutaneous", "value": 4.64908773087}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Adipose_Subcutaneous", "value": 0.701780148981}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Adipose_Subcutaneous", "value": 0.668177824553}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Adipose_Subcutaneous", "value": 1.8228875224799999}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Adipose_Subcutaneous", "value": 1.97626205757}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Adipose_Subcutaneous", "value": 1.0298712872600002}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Adipose_Subcutaneous", "value": 0.562157381969}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Adipose_Subcutaneous", "value": -8.899947588}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Adipose_Subcutaneous", "value": -4.59463074856}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Adipose_Subcutaneous", "value": 1.9809926529599997}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Adipose_Subcutaneous", "value": -5.578324169580001}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Adipose_Subcutaneous", "value": -0.21055918085099998}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Adipose_Subcutaneous", "value": -1.6138485328}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Adipose_Subcutaneous", "value": -0.613181693299}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Adipose_Subcutaneous", "value": 3.43045182867}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Adipose_Subcutaneous", "value": 3.85952259811}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Adipose_Subcutaneous", "value": 3.73032824982}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Adipose_Subcutaneous", "value": 0.018312309575}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Adipose_Subcutaneous", "value": -0.706444711912}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Adipose_Subcutaneous", "value": 1.0938247010700002}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Adipose_Subcutaneous", "value": 5.59100937678}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Adipose_Subcutaneous", "value": -2.75224840921}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Adipose_Subcutaneous", "value": 5.3064145935500004}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Adipose_Subcutaneous", "value": -4.40179955252}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Adipose_Subcutaneous", "value": -3.51540498445}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Adipose_Subcutaneous", "value": 4.5064150605}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Adipose_Subcutaneous", "value": 4.44800760336}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Adipose_Subcutaneous", "value": 4.44094677358}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Adipose_Visceral_Omentum", "value": 4.37356233785}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Adipose_Visceral_Omentum", "value": 5.867349079169999}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Adipose_Visceral_Omentum", "value": -0.303409875935}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Adipose_Visceral_Omentum", "value": 5.89951076138}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Adipose_Visceral_Omentum", "value": 5.6376147332599995}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Adipose_Visceral_Omentum", "value": 5.45799289191}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Adipose_Visceral_Omentum", "value": 5.02159091589}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Adipose_Visceral_Omentum", "value": 0.239745451908}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Adipose_Visceral_Omentum", "value": 0.187717895742}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Adipose_Visceral_Omentum", "value": 1.1019244459200002}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Adipose_Visceral_Omentum", "value": 1.7591419998900002}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Adipose_Visceral_Omentum", "value": 1.26965437041}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Adipose_Visceral_Omentum", "value": 0.865745679433}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Adipose_Visceral_Omentum", "value": -8.550108581860002}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Adipose_Visceral_Omentum", "value": -4.92584433475}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Adipose_Visceral_Omentum", "value": 2.02290938999}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Adipose_Visceral_Omentum", "value": -5.02914626965}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Adipose_Visceral_Omentum", "value": -0.38405330841799995}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Adipose_Visceral_Omentum", "value": -2.25504827856}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Adipose_Visceral_Omentum", "value": -1.75066096405}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Adipose_Visceral_Omentum", "value": 3.7615727659300005}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Adipose_Visceral_Omentum", "value": 4.07534655113}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Adipose_Visceral_Omentum", "value": 3.9282985102599994}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Adipose_Visceral_Omentum", "value": -0.5703909763189999}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Adipose_Visceral_Omentum", "value": -0.170963601136}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Adipose_Visceral_Omentum", "value": 2.28407873979}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Adipose_Visceral_Omentum", "value": 5.523861073709999}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Adipose_Visceral_Omentum", "value": -2.7700955653799997}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Adipose_Visceral_Omentum", "value": 5.7014361487}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Adipose_Visceral_Omentum", "value": -5.34543075679}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Adipose_Visceral_Omentum", "value": -4.553528325019999}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Adipose_Visceral_Omentum", "value": 4.98657526921}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Adipose_Visceral_Omentum", "value": 4.6649344056099995}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Adipose_Visceral_Omentum", "value": 4.86749653182}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Breast_Mammary_Tissue", "value": 4.32577668602}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Breast_Mammary_Tissue", "value": 5.933655358589999}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Breast_Mammary_Tissue", "value": -0.0226678957235}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Breast_Mammary_Tissue", "value": 5.711320690599999}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Breast_Mammary_Tissue", "value": 5.4304282542}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Breast_Mammary_Tissue", "value": 5.40767401979}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Breast_Mammary_Tissue", "value": 4.96826833256}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Breast_Mammary_Tissue", "value": 0.614800310426}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Breast_Mammary_Tissue", "value": 0.600437822319}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Breast_Mammary_Tissue", "value": 1.59118257062}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Breast_Mammary_Tissue", "value": 2.09659983346}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Breast_Mammary_Tissue", "value": 1.38255183324}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Breast_Mammary_Tissue", "value": 0.8475321394390001}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Breast_Mammary_Tissue", "value": -9.13384618552}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Breast_Mammary_Tissue", "value": -4.873945173469999}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Breast_Mammary_Tissue", "value": 1.9655200294400001}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Breast_Mammary_Tissue", "value": -5.4302788474599994}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Breast_Mammary_Tissue", "value": -0.137305553196}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Breast_Mammary_Tissue", "value": -1.43847220997}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Breast_Mammary_Tissue", "value": -0.580503455707}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Breast_Mammary_Tissue", "value": 3.12488505653}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Breast_Mammary_Tissue", "value": 3.87904545621}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Breast_Mammary_Tissue", "value": 3.7950191597800003}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Breast_Mammary_Tissue", "value": -0.181884912699}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Breast_Mammary_Tissue", "value": -0.462003613539}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Breast_Mammary_Tissue", "value": 2.7303296837700004}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Breast_Mammary_Tissue", "value": 5.38046510105}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Breast_Mammary_Tissue", "value": -2.81389218425}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Breast_Mammary_Tissue", "value": 5.8254821499}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Breast_Mammary_Tissue", "value": -4.98389130574}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Breast_Mammary_Tissue", "value": -4.05652892354}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Breast_Mammary_Tissue", "value": 4.45479800794}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Breast_Mammary_Tissue", "value": 4.359300785299999}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Breast_Mammary_Tissue", "value": 4.39640318278}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Muscle_Skeletal", "value": 4.9119714038}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Muscle_Skeletal", "value": 6.03279755431}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Muscle_Skeletal", "value": -0.339419533429}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Muscle_Skeletal", "value": 6.253882540759999}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Muscle_Skeletal", "value": 5.991975364169999}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Muscle_Skeletal", "value": 5.84728579477}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Muscle_Skeletal", "value": 5.382959241819999}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Muscle_Skeletal", "value": 0.36907161581}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Muscle_Skeletal", "value": 0.328730644642}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Muscle_Skeletal", "value": 1.4996759974899998}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Muscle_Skeletal", "value": 1.9544515488499998}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Muscle_Skeletal", "value": 1.2839839442899998}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Muscle_Skeletal", "value": 0.82042628427}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Muscle_Skeletal", "value": -9.31275220702}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Muscle_Skeletal", "value": -5.301220113219999}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Muscle_Skeletal", "value": 1.90282159034}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Muscle_Skeletal", "value": -5.404787531819999}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Muscle_Skeletal", "value": -0.435935065366}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Muscle_Skeletal", "value": -1.81125087584}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Muscle_Skeletal", "value": -0.920742699848}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Muscle_Skeletal", "value": 3.8226862746099997}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Muscle_Skeletal", "value": 3.9471449210400005}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Muscle_Skeletal", "value": 3.8487081594999997}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Muscle_Skeletal", "value": -0.169724158787}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Muscle_Skeletal", "value": -0.474698137852}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Muscle_Skeletal", "value": 2.8747176441000004}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Muscle_Skeletal", "value": 5.652433301009999}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Muscle_Skeletal", "value": -2.7267704570899998}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Muscle_Skeletal", "value": 6.06504687922}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Muscle_Skeletal", "value": -5.2511559406}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Muscle_Skeletal", "value": -4.37678307945}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Muscle_Skeletal", "value": 4.9461907913}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Muscle_Skeletal", "value": 4.7246049733}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Muscle_Skeletal", "value": 4.8744601018900005}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 4.29049783469}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 6.10359649417}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -0.406407787913}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 5.57290472855}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 5.30358606798}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 5.24095326741}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 4.8054284069}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.238976200971}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.16942876598499998}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 1.2845780734799999}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 2.0695967786500002}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 1.40872468192}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.962045936745}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -8.98958630154}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -4.70603098562}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 2.2043667151700004}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -5.510063984279999}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -0.460364726441}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -1.99829481305}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -0.785415223072}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 3.23446350926}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 3.9460671888}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 3.8476572923800005}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -0.37123337141400004}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -0.418443038547}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 2.08542792566}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 5.06925334402}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -2.8900658551599996}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 5.22480543979}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -4.7002584462}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -3.76618926708}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 4.47129837144}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 4.33910543478}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 4.40645473822}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 4.615818375380001}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 6.0323387985500005}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Skin_Sun_Exposed_Lower_leg", "value": -0.615256177353}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 5.27020977459}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 5.079648011640001}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 4.95111012378}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 4.5199626694}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.266132601445}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.259494945084}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.83497886775}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.7682690509599999}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.764703631965}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.33856393990500006}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Skin_Sun_Exposed_Lower_leg", "value": -8.94479163832}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Skin_Sun_Exposed_Lower_leg", "value": -4.50418909419}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.1263685801700003}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Skin_Sun_Exposed_Lower_leg", "value": -5.18857683076}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Skin_Sun_Exposed_Lower_leg", "value": -0.75880700599}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Skin_Sun_Exposed_Lower_leg", "value": -1.56680907145}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Skin_Sun_Exposed_Lower_leg", "value": -0.111191799949}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.1685461060400004}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.83772270351}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.76695350728}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.627290351594}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Skin_Sun_Exposed_Lower_leg", "value": -0.855773268332}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.334791724452}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 5.1966067886800005}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.20308132319}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.33274656739}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Skin_Sun_Exposed_Lower_leg", "value": -4.326234164750001}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Skin_Sun_Exposed_Lower_leg", "value": -3.55748083825}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.6922913180300005}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.3209068836199997}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.6387450235400003}], "data-4a8205db45783061b94291660645b8ff": [{"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Adrenal_Gland", "value": 0.0}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Pituitary", "value": 4.5536169843300005}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Pituitary", "value": 4.20893179762}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Pituitary", "value": 0.78929656404}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Pituitary", "value": 6.08307609327}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Pituitary", "value": 5.7083359364}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Pituitary", "value": 5.75597678325}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Pituitary", "value": 5.4121456940199995}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Pituitary", "value": 2.1045717833200004}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Pituitary", "value": 2.18071085855}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Pituitary", "value": 3.5059000252699994}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Pituitary", "value": 1.4554269713499999}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Pituitary", "value": -0.814649466535}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Pituitary", "value": -1.2377506632100002}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Pituitary", "value": -7.4701924888499995}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Pituitary", "value": -5.23210016862}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Pituitary", "value": 0.645960746474}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Pituitary", "value": -5.651951706619999}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Pituitary", "value": 0.590902482879}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Pituitary", "value": -2.52866718771}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Pituitary", "value": 0.050745590569400006}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Pituitary", "value": 2.30001907188}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Pituitary", "value": 2.3184986950999997}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Pituitary", "value": 2.25166851222}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Pituitary", "value": -1.1538690245299998}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Pituitary", "value": -1.8347960529700003}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Pituitary", "value": 2.7651805515099994}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Pituitary", "value": 6.47121848997}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Pituitary", "value": -1.8212693191799998}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Pituitary", "value": 4.872360226590001}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Pituitary", "value": -3.4886645735800004}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Pituitary", "value": -2.51271757021}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Pituitary", "value": 4.2550444987}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Pituitary", "value": 3.9734716022800005}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Pituitary", "value": 4.21546161542}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Pancreas", "value": 2.53674162324}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Pancreas", "value": 4.81507334666}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Pancreas", "value": -1.7256494897099999}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Pancreas", "value": 2.64736404338}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Pancreas", "value": 2.61776134243}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Pancreas", "value": 3.01906427231}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Pancreas", "value": 3.0633822627700003}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Pancreas", "value": -1.3089909018600001}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Pancreas", "value": -1.55022560461}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Pancreas", "value": -0.444540632063}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Pancreas", "value": 1.5260545245200001}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Pancreas", "value": 1.9693947269299998}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Pancreas", "value": 1.61288728564}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Pancreas", "value": -5.799760150240001}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Pancreas", "value": -3.16292636565}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Pancreas", "value": 2.6576142031}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Pancreas", "value": -4.3575393907099995}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Pancreas", "value": -1.55838802168}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Pancreas", "value": -2.9057850616900005}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Pancreas", "value": -0.0754088763846}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Pancreas", "value": 2.1792806559}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Pancreas", "value": 2.87016801018}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Pancreas", "value": 2.7606436282}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Pancreas", "value": 0.26226825619399996}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Pancreas", "value": -0.119308002479}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Pancreas", "value": -1.0566471208900001}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Pancreas", "value": 2.6182107585599996}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Pancreas", "value": -2.9786827991400004}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Pancreas", "value": 1.6085774019299999}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Pancreas", "value": -3.4873416974099998}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Pancreas", "value": -2.98227801333}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Pancreas", "value": 2.78690646258}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Pancreas", "value": 2.6559526620800002}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Pancreas", "value": 2.73586723369}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Thyroid", "value": 4.11283165922}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Thyroid", "value": 5.84872607891}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Thyroid", "value": -0.270316732166}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Thyroid", "value": 5.3331015259}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Thyroid", "value": 5.051648699119999}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Thyroid", "value": 5.08311484547}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Thyroid", "value": 4.726820103640001}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Thyroid", "value": 0.399812836257}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Thyroid", "value": 0.384998675092}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Thyroid", "value": 1.52938809204}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Thyroid", "value": 1.69806811544}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Thyroid", "value": 1.0476774034899998}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Thyroid", "value": 0.675371008044}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Thyroid", "value": -9.029724694119999}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Thyroid", "value": -4.6402788916699995}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Thyroid", "value": 1.8158233471599998}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Thyroid", "value": -5.50574165089}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Thyroid", "value": -0.407476470755}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Thyroid", "value": -1.8783225384}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Thyroid", "value": -0.9560688417380001}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Thyroid", "value": 3.43243610272}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Thyroid", "value": 3.8421028248300004}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Thyroid", "value": 3.7435626396900004}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Thyroid", "value": -0.114650391506}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Thyroid", "value": -0.534148974957}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Thyroid", "value": 3.0706913146}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Thyroid", "value": 5.37748586842}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Thyroid", "value": -2.92339589833}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Thyroid", "value": 5.48099696549}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Thyroid", "value": -4.94449089022}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Thyroid", "value": -4.156966658430001}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Thyroid", "value": 4.67887126456}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Thyroid", "value": 4.3133582089099995}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Thyroid", "value": 4.56705517951}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Lung", "value": 3.61362991017}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Lung", "value": 5.514939944040001}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Lung", "value": -0.144706408122}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Lung", "value": 5.204403623459999}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Lung", "value": 4.79382990513}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Lung", "value": 5.02953171543}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Lung", "value": 4.52630268079}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Lung", "value": 0.492252001196}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Lung", "value": 0.563719708213}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Lung", "value": 1.5453645841499999}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Lung", "value": 2.0260683135}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Lung", "value": 0.516664853477}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Lung", "value": -0.101001964089}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Lung", "value": -8.71621264861}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Lung", "value": -4.37593739133}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Lung", "value": 1.7541424904400003}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Lung", "value": -4.8268631825199995}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Lung", "value": -0.36355224528}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Lung", "value": -0.988962293162}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Lung", "value": -0.7869639901559999}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Lung", "value": 4.5271930405}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Lung", "value": 4.77043816407}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Lung", "value": 4.66565580829}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Lung", "value": 1.17211066536}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Lung", "value": -0.611432592749}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Lung", "value": 1.4559568467200001}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Lung", "value": 5.11170939689}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Lung", "value": -2.73540477485}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Lung", "value": 5.5965063413400005}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Lung", "value": -4.70029243346}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Lung", "value": -3.9273594354300005}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Lung", "value": 4.24595157876}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Lung", "value": 4.56865351959}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Lung", "value": 4.17434370553}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Ovary", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Ovary", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Ovary", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Ovary", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Ovary", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Ovary", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Ovary", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Ovary", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Ovary", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Ovary", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Ovary", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Ovary", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Ovary", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Ovary", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Ovary", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Ovary", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Ovary", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Ovary", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Ovary", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Ovary", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Ovary", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Ovary", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Ovary", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Ovary", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Ovary", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Ovary", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Ovary", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Ovary", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Ovary", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Ovary", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Ovary", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Ovary", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Ovary", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Ovary", "value": 0.0}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Prostate", "value": 4.15339010811}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Prostate", "value": 6.305150341440001}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Prostate", "value": -0.963730510833}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Prostate", "value": 5.802812267819999}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Prostate", "value": 5.49244748877}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Prostate", "value": 5.474804998860001}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Prostate", "value": 4.99557510125}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Prostate", "value": -0.239523119046}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Prostate", "value": -0.32796725638}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Prostate", "value": 1.04140448802}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Prostate", "value": 2.16114072466}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Prostate", "value": 1.31862093206}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Prostate", "value": 0.861737236741}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Prostate", "value": -9.042238052250001}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Prostate", "value": -4.88679256384}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Prostate", "value": 2.3999221114}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Prostate", "value": -5.94285863921}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Prostate", "value": -0.981341118617}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Prostate", "value": -2.3762204582599997}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Prostate", "value": -1.22303296426}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Prostate", "value": 3.8279300733699997}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Prostate", "value": 4.19465500531}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Prostate", "value": 4.090045705450001}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Prostate", "value": -0.45779057944799995}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Prostate", "value": -0.333179590538}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Prostate", "value": 2.19578925668}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Prostate", "value": 5.20119371836}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Prostate", "value": -3.01504922148}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Prostate", "value": 5.07038841152}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Prostate", "value": -4.84293003664}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Prostate", "value": -3.9848229479}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Prostate", "value": 4.62790156866}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Prostate", "value": 4.54303272643}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Prostate", "value": 4.56078684368}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Testis", "value": 0.910242231388}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Testis", "value": 5.43254922005}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Testis", "value": -1.89787716159}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Testis", "value": 1.2361854974}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Testis", "value": 1.19051702745}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Testis", "value": 1.15392094946}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Testis", "value": 0.809469149559}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Testis", "value": -2.6844472416900005}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Testis", "value": -2.7723040972299997}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Testis", "value": -2.8155021109500002}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Testis", "value": 1.03757860355}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Testis", "value": 1.50411381347}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Testis", "value": 1.61564666593}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Testis", "value": -5.8711060245099995}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Testis", "value": -0.9278250865809999}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Testis", "value": 2.9872248921499995}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Testis", "value": -2.99859150698}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Testis", "value": -1.9837486183700002}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Testis", "value": -1.9466299313799997}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Testis", "value": -1.76164144362}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Testis", "value": 4.040556459549999}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Testis", "value": 3.7559160159800005}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Testis", "value": 3.62085181301}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Testis", "value": 1.6918646615900002}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Testis", "value": 0.972001546171}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Testis", "value": -0.316510331711}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Testis", "value": -1.17115024741}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Testis", "value": -1.38260647667}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Testis", "value": 2.20391332405}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Testis", "value": -1.72012089087}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Testis", "value": -1.6164883119799998}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Testis", "value": 1.94054265716}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Testis", "value": 1.43482627443}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Testis", "value": 1.8573640107299998}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Uterus", "value": 0.7340286828810001}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Uterus", "value": 7.43171251235}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Uterus", "value": -3.17204271667}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Uterus", "value": 2.62034883895}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Uterus", "value": 2.96480651018}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Uterus", "value": 2.50786171516}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Uterus", "value": 2.1771249342}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Uterus", "value": -4.54696556867}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Uterus", "value": -4.79294282216}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Uterus", "value": -4.63488304044}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Uterus", "value": 2.5946601081}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Uterus", "value": 4.08918198689}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Uterus", "value": 3.41665730043}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Uterus", "value": -7.684483831889999}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Uterus", "value": -2.7326342154}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Uterus", "value": 4.59212072397}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Uterus", "value": -3.61563916616}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Uterus", "value": -3.00272113698}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Uterus", "value": 0.047072650226099993}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Uterus", "value": -2.39645879623}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Uterus", "value": 4.66456503551}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Uterus", "value": 1.0101651572600001}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Uterus", "value": 0.9849729358179999}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Uterus", "value": 1.96832467908}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Uterus", "value": 1.67019307348}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Uterus", "value": -0.281745121632}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Uterus", "value": 0.8557775591210001}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Uterus", "value": -2.67020607131}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Uterus", "value": -1.00643655893}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Uterus", "value": -0.7910074473280001}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Uterus", "value": -0.6033712897}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Uterus", "value": 2.10924922937}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Uterus", "value": 1.7487213912900001}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Uterus", "value": 2.07866041905}, {"code": "ATOP01", "phenotype": "Atopic dermatitis", "category": "j: other", "LDSC Mean Chi-Square": 1.0799, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "PMID or link": "26482879", "LDSC Intercept": "1.0234 (0.007)", "ENSEMBL": "ENSG00000173327", "study size": 116863, "LDSC Total Observed Scale h2": "0.0723 (0.0167)", "phenotype_reference": "Atopic dermatitis (ATOP01) ", "index": 0, "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "gene": "MAP3K11", "association": 8.447514, "variable": "Vagina", "value": 0.0}, {"code": "BLOO01", "phenotype": "platelet count", "category": "g: blood", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "phenotype_reference": "Platelet count (BLOO01) ", "index": 1, "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.851428, "variable": "Vagina", "value": 0.0}, {"code": "BLOO04", "phenotype": "reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 2, "LDSC Lambda GC": 1.207, "LDSC Ratio": "0.0243 (0.0638)", "gene": "MAP3K11", "association": 7.797294, "variable": "Vagina", "value": 0.0}, {"code": "BLOO07", "phenotype": "eosinophil count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 3, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 20.347096, "variable": "Vagina", "value": 0.0}, {"code": "BLOO16", "phenotype": "Sum eosinophil basophil counts", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 4, "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 17.83062, "variable": "Vagina", "value": 0.0}, {"code": "BLOO17", "phenotype": "Eosinophil percentage of white cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9613 (0.0261)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1752 (0.02)", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 5, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 14.000226000000001, "variable": "Vagina", "value": 0.0}, {"code": "BLOO18", "phenotype": "Eosinophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9588 (0.0243)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1696 (0.019)", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 6, "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.464769, "variable": "Vagina", "value": 0.0}, {"code": "BLOO21", "phenotype": "High light scatter percentage of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 7, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0417 (0.0682)", "gene": "MAP3K11", "association": 16.775467000000003, "variable": "Vagina", "value": 0.0}, {"code": "BLOO22", "phenotype": "High light scatter reticulocyte count", "category": "g: blood", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 8, "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0416 (0.0672)", "gene": "MAP3K11", "association": 16.870342, "variable": "Vagina", "value": 0.0}, {"code": "BLOO23", "phenotype": "Immature fraction of reticulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0284 (0.0402)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "index": 9, "LDSC Lambda GC": 1.1555, "LDSC Ratio": "0.0626 (0.0887)", "gene": "MAP3K11", "association": 19.012204, "variable": "Vagina", "value": 0.0}, {"code": "BLOO25", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "category": "g: blood", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 10, "LDSC Lambda GC": 1.0988, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.324781, "variable": "Vagina", "value": 0.0}, {"code": "BLOO26", "phenotype": "Mean corpuscular hemoglobin MCH", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 11, "LDSC Lambda GC": 1.194, "LDSC Ratio": "0.012 (0.0532)", "gene": "MAP3K11", "association": 13.211231, "variable": "Vagina", "value": 0.0}, {"code": "BLOO27", "phenotype": "Mean corpuscular volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 12, "LDSC Lambda GC": 1.2136, "LDSC Ratio": "0.0096 (0.0469)", "gene": "MAP3K11", "association": 9.935617, "variable": "Vagina", "value": 0.0}, {"code": "BLOO29", "phenotype": "Mean platelet volume", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0051 (0.0458)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 13, "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0057 (0.0508)", "gene": "MAP3K11", "association": 18.411604999999998, "variable": "Vagina", "value": 0.0}, {"code": "BLOO31", "phenotype": "Neutrophil percentage of granulocytes", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9571 (0.0217)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 14, "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 12.113887, "variable": "Vagina", "value": 0.0}, {"code": "BLOO33", "phenotype": "Plateletcrit", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 15, "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "gene": "MAP3K11", "association": 12.270275999999999, "variable": "Vagina", "value": 0.0}, {"code": "BLOO35", "phenotype": "Red cell distribution width", "category": "g: blood", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 16, "LDSC Lambda GC": 1.1651, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 16.524765, "variable": "Vagina", "value": 0.0}, {"code": "BLOO36", "phenotype": "Reticulocyte fraction of red cells", "category": "g: blood", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "ENSEMBL": "ENSG00000173327", "study size": 173480, "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 17, "LDSC Lambda GC": 1.2038, "LDSC Ratio": "0.0197 (0.0653)", "gene": "MAP3K11", "association": 6.218216, "variable": "Vagina", "value": 0.0}, {"code": "BODY08", "phenotype": "BMI", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0353 (0.0162)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.23 (0.0081)", "phenotype_reference": "BMI (BODY08) ", "index": 18, "LDSC Lambda GC": 2.0471, "LDSC Ratio": "0.0217 (0.01)", "gene": "MAP3K11", "association": 7.2999279999999995, "variable": "Vagina", "value": 0.0}, {"code": "DIAB05", "phenotype": "Type 2 Diabetes 2017", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2307, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28566273", "PMID or link": "28566273", "LDSC Intercept": "0.9978 (0.0095)", "ENSEMBL": "ENSG00000173327", "study size": 159208, "LDSC Total Observed Scale h2": "0.0782 (0.0057)", "phenotype_reference": "Type 2 Diabetes 2017 (DIAB05) ", "index": 19, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 7.004430999999999, "variable": "Vagina", "value": 0.0}, {"code": "HEIG03", "phenotype": "Height 2014", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "ENSEMBL": "ENSG00000173327", "study size": 253288, "LDSC Total Observed Scale h2": "0.341 (0.0174)", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 20, "LDSC Lambda GC": 2.0007, "LDSC Ratio": "0.1178 (0.0152)", "gene": "MAP3K11", "association": 3.4437699999999998, "variable": "Vagina", "value": 0.0}, {"code": "HIPC03", "phenotype": "Hip circumference European BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "ENSEMBL": "ENSG00000173327", "study size": 143579, "LDSC Total Observed Scale h2": "0.1459 (0.009)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "index": 21, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.8839239999999995, "variable": "Vagina", "value": 0.0}, {"code": "HIPC04", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "ENSEMBL": "ENSG00000173327", "study size": 143575, "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "index": 22, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.661982000000001, "variable": "Vagina", "value": 0.0}, {"code": "LEAN06", "phenotype": "Fat free mass / lean mass", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "ENSEMBL": "ENSG00000173327", "study size": 353972, "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 23, "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "gene": "MAP3K11", "association": 7.745863000000001, "variable": "Vagina", "value": 0.0}, {"code": "MENA02F", "phenotype": "Age at Menarche (marker of puberty in females)", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "PMID or link": "28436984", "LDSC Intercept": "1.0084 (0.0147)", "ENSEMBL": "ENSG00000173327", "study size": 329345, "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "index": 24, "LDSC Lambda GC": 1.6108, "LDSC Ratio": "0.0081 (0.0141)", "gene": "MAP3K11", "association": 8.253810000000001, "variable": "Vagina", "value": 0.0}, {"code": "OSTE03", "phenotype": "Osteoporosis: lumbar spine bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1412, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26367794", "PMID or link": "26367794", "LDSC Intercept": "0.9839 (0.0136)", "ENSEMBL": "ENSG00000173327", "study size": 28498, "LDSC Total Observed Scale h2": "0.2772 (0.0376)", "phenotype_reference": "Osteoporosis: lumbar spine bone mineral density (OSTE03) ", "index": 25, "LDSC Lambda GC": 1.0864, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.158615, "variable": "Vagina", "value": 0.0}, {"code": "OSTE04", "phenotype": "Osteoporosis: heel bone mineral density", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 2.1384, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28869591", "PMID or link": "28869591", "LDSC Intercept": "1.0787 (0.0296)", "ENSEMBL": "ENSG00000173327", "study size": 142487, "LDSC Total Observed Scale h2": "0.3752 (0.0403)", "phenotype_reference": "Osteoporosis: heel bone mineral density (OSTE04) ", "index": 26, "LDSC Lambda GC": 1.4926, "LDSC Ratio": "0.0692 (0.026)", "gene": "MAP3K11", "association": 18.124187, "variable": "Vagina", "value": 0.0}, {"code": "SCHI01", "phenotype": "Schizophrenia PGC 2014", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "ENSEMBL": "ENSG00000173327", "study size": 82315, "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 27, "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "gene": "MAP3K11", "association": 6.045251, "variable": "Vagina", "value": 0.0}, {"code": "URAT01", "phenotype": "Urate", "category": "a: metabolic", "LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "PMID or link": "23263486", "LDSC Intercept": "0.8546 (0.045)", "ENSEMBL": "ENSG00000173327", "study size": 110347, "LDSC Total Observed Scale h2": "0.1522 (0.053)", "phenotype_reference": "Urate (URAT01) ", "index": 28, "LDSC Lambda GC": 1.0315, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.110525, "variable": "Vagina", "value": 0.0}, {"code": "URIN07", "phenotype": "Glomerular filtration rate (serum creatinine)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.2721, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9752 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 133814, "LDSC Total Observed Scale h2": "0.1073 (0.0093)", "phenotype_reference": "Glomerular filtration rate (serum creatinine) (URIN07) ", "index": 29, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 6.4675769999999995, "variable": "Vagina", "value": 0.0}, {"code": "URIN09", "phenotype": "Glomerular filtration rate (serum creatinine non diabetes mellitus)", "category": "f: genitourinary", "LDSC Mean Chi-Square": 1.26, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26831199", "PMID or link": "26831199", "LDSC Intercept": "0.9786 (0.0131)", "ENSEMBL": "ENSG00000173327", "study size": 118460, "LDSC Total Observed Scale h2": "0.1146 (0.0104)", "phenotype_reference": "Glomerular filtration rate (serum creatinine non diabetes mellitus) (URIN09) ", "index": 30, "LDSC Lambda GC": 1.1459, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 5.535332, "variable": "Vagina", "value": 0.0}, {"code": "WAIS03F", "phenotype": "Waist circumference EU female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 91952, "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "index": 31, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.0124629999999994, "variable": "Vagina", "value": 0.0}, {"code": "WAIS03", "phenotype": "Waist circumference EU BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 229089, "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "index": 32, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 3.5163839999999995, "variable": "Vagina", "value": 0.0}, {"code": "WAIS04F", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "ENSEMBL": "ENSG00000173327", "study size": 132715, "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "index": 33, "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "gene": "MAP3K11", "association": 4.326371, "variable": "Vagina", "value": 0.0}]}};
    var embed_opt = {"mode": "vega-lite"};

    function showError(el, error){
        el.innerHTML = ('<div class="error">'
                        + '<p>JavaScript Error: ' + error.message + '</p>'
                        + "<p>This usually means there's a typo in your chart specification. "
                        + "See the javascript console for the full traceback.</p>"
                        + '</div>');
        throw error;
    }
    const el = document.getElementById('vis');
    vegaEmbed("#vis", spec, embed_opt)
      .catch(error => showError(el, error));
  </script>
</body>
</html>
