
<!DOCTYPE html>

<html>
<head><style>.vega-actions a {margin-right: 12px;color: #757575;font-weight: normal;font-size: 13px;}.error {color: red;}</style><script src="https://cdn.jsdelivr.net/npm//vega@4" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-lite@2.6.0" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-embed@3" type="text/javascript"></script><meta content="text/html; charset=utf-8" http-equiv="Content-Type"/><meta content="IE=edge" http-equiv="X-UA-Compatible"/><meta content="width=device-width, initial-scale=1" name="viewport"/><link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/Font-Awesome-4.7/css/font-awesome.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap-theme.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/images/gnomic_icon.png" rel="icon" type="image/png"/><script src="https://phenviz.navigome.com/js/jquery.min.js"></script><script src="https://phenviz.navigome.com/js/bootstrap.js"></script><title>Gene profile: ENSG00000124575</title></head>
<body>
<h4>Gene profile across phenotypes - ENSG00000124575 (HIST1H1D)</h4><div id="vis"></div><div class="col-lg-12 col-md-12 content-item content-item-1"><p>This is a visualization of GWAS-derived gene associations  across Navigome phenotypes. Only phenotypes for which the gene has at least one significant association (p-value â‰¤ 0.05/20000) in <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> <b>and/or</b> <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a> analyses are shown. The first panel is a visualization of gene-phenotype associations obtained using <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> (-log10(p-value)). Hover over any bar to highlight the phenotype in the other panels. The other panels are parallel coordinates visualizations for tissue-wise associations (z-scores)computed using <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a>  and eQTL expression data from GTEx and DGN. Significance thresholds are delineated with black lines. The parallel coordinates show which phenotypes have a significant association and in which tissues, based on gene expression data. Clicking on a specific phenotype will lead to the reference of the GWAS study.</p></div>
<script type="text/javascript">
    var spec = {"config": {"view": {"width": 400, "height": 300}}, "vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "bar", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "href": {"type": "nominal", "field": "phenotype_link"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "quantitative", "axis": {"title": "MAGMA -log10(p)"}, "field": "value"}, "y": {"type": "nominal", "axis": {"title": ""}, "field": "phenotype_reference"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-4e93ee1a91df87c61b7d73e211f0f4d1"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "x": {"type": "quantitative", "field": "ThresholdVal"}}}], "title": "MAGMA gene-wise association"}], "center": true, "data": {"name": "data-0c6ac0520a5919810915fd925c5eff01"}}, {"vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Nervous system", "width": 500}], "center": true, "data": {"name": "data-b18c768ad5ad36f928d4cf84c557f320"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Digestive System", "width": 500}], "center": true, "data": {"name": "data-acc785951044d5b0df58df985be9ca6a"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Blood, Arteries and Heart", "width": 500}], "center": true, "data": {"name": "data-9653fa4494fa010c4ba5be9ff6c1d59a"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Adipose Tissue, Muscle, Skin and Cells", "width": 500}], "center": true, "data": {"name": "data-a34e7b216b695c31da963e2ff5265bcb"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-16.8761894645, 16.8761894645]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Endocrine System and Others", "width": 500}], "center": true, "data": {"name": "data-46696b3ae9ca277d7032413742f41de3"}}]}], "spacing": 40, "$schema": "https://vega.github.io/schema/vega-lite/v2.6.0.json", "datasets": {"data-0c6ac0520a5919810915fd925c5eff01": [{"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 17.02326}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 8.802692}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 32.093024}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 14.674361}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 10.45193}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 11.724735}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 7.914924000000001}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 10.140916}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 23.81953}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 22.670826}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 6.116066}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 12.269824}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 22.223924}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 10.840313}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 10.6094}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 48.617245000000004}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 78.28195500000001}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 49.249762}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 7.474359}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 6.647528}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 7.470428}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 49.668067}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 29.309343}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 8.32661}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 15.354076999999998}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 26.594022}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 9.336243}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 22.449136}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 17.124343}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 10.014955}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 8.999392}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 11.109741}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 8.393339}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 12.425286}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 16.366854999999997}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 13.1015}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 13.832063}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 17.774794}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 12.718376}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 26.804405}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 13.509578}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 24.27005}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 23.225154999999997}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 12.518728999999999}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 37.405287}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 6.47881}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 12.724480999999999}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 11.460134}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 6.336176999999999}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 6.5981}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 6.3720300000000005}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 9.298337}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 14.206112}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 7.861603}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 10.216547}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 14.155728}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "magma_log10p", "value": 10.157110000000001}], "data-4e93ee1a91df87c61b7d73e211f0f4d1": [{"Threshold": "MAGMA -log10(p) \u2265 5.60206", "ThresholdVal": 5.60206}], "data-b18c768ad5ad36f928d4cf84c557f320": [{"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.45440298471}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -4.27190986737}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 10.8647066924}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 5.20827238826}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.585045848636}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.122176488293}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.811328586738}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.824938379763}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -1.33808348455}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -1.18746445382}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.859847415607}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0341452304435}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 7.188298485830001}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 3.24813057205}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 2.99969447256}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 11.1786917517}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 16.8761894645}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 13.7516046655}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.7561525881919999}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.688041942965}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -4.54081304511}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -13.1359643656}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 5.653618530519999}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.6424039660809999}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 2.9211765724400003}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.543472762781}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.795946506718}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.921502067939}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.90123786365}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.12841539294600002}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.8198354448599998}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.9264063235599997}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.8439124316099997}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.99888475765}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.22596320468}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.35324949479}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.63361363122}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.63257419118}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.41396511229}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.83001533757}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -1.5945479148}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 5.08226643019}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.623842413019}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.576679947083}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -1.51890544319}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.963764407552}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.8451506938529999}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.7676969630249999}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.36925838131}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.46627957291}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.42173780389}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.013338015039600002}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.9939932218299999}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.73920994492}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.022084658994800003}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.09615400249}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.04239155229}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Nerve_Tibial", "value": 0.0}], "data-c4c9a654d45741ebc60aa567761bff19": [{"Text": "Significance Threshold", "Threshold": "PrediXcan zscore \u2265 4.70813", "ThresholdVal": 4.70813}], "data-4a3aa5bf8c5a41879455008930c25774": [{"Threshold": "PrediXcan zscore \u2264 -4.70813", "ThresholdVal": -4.70813}], "data-acc785951044d5b0df58df985be9ca6a": [{"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Sigmoid", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.458050355477}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.574134975461}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 10.3541961751}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.51992593908}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.1450645092}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.39192148539300004}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.6231629284}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.9289877120599996}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.5077185209600001}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.04757329873}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.16154010889}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.22653613660799998}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 5.8885342843199995}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.9749096428099997}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.61036833082}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 11.2695196649}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 15.893881943199998}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 12.7990773403}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.58866352143}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.2305445301599995}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.38577399999399997}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -10.0375427479}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.87086171224}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.57299087114}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.0306894660400001}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.320335337889}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -1.14324826361}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.805231580353}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.782034059201}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -1.1116199199799999}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.295258382287}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.13387088066199998}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.267284835368}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.051608341322099995}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.32495977947500004}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -1.25394200909}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.590773326462}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.2854152616}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -1.1759125822499998}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.68135433306}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -6.8635090965400005}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 11.330851844100001}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -1.08889758503}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.603823413257}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -1.49175381371}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.23341036244000002}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.08779661293}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.1453646050700002}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.175440238063}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.220933259968}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.44231838387199995}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.6333622576519999}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.399895903298}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.673416681418}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.61568184495}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.24855790858399998}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.20751189486299998}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Liver", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 0.479981963969}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -5.4949150612999995}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 2.35227381578}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -2.01500751199}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -0.642559639122}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -0.34996837434699996}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -0.804288444385}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -0.920849450479}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -1.2299490074600001}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -1.0681429738700001}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -0.338080007535}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 0.292210353396}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 1.68745343208}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -3.38473782426}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -3.5270150745199995}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 1.7005152331600002}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 4.54001107241}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 4.57167257513}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 4.93577171278}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -0.184443293457}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -3.6267774418900003}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -3.08131572912}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -2.01441823582}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 0.33397154357}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 1.5730052149899998}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 1.54565749312}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 0.466039730673}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 1.24428825982}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 0.803074887243}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 0.595072639541}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 1.7960868484}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 1.6383376568099999}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 1.78828337012}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 1.5770413840700002}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 1.8660926659400001}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -0.17378617876400002}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 1.25061303261}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 1.8586976073900001}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -0.19954936813299998}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 1.18835806771}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 0.7678266870530001}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 2.89492494976}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 3.28395622707}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -0.5178493287479999}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -2.2421607932799996}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -1.21266369015}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -2.58809327982}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -2.27920535841}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 1.2162295954200002}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 1.16747732578}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 0.8061175413860001}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -0.44540851446099994}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 0.35257562786700003}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 0.816971401946}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": -0.44275708585}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 0.329989377802}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Stomach", "value": 0.35601391373499996}], "data-9653fa4494fa010c4ba5be9ff6c1d59a": [{"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "DGN_Whole_Blood", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Whole_Blood", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Artery_Tibial", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 4.18994865076}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -7.458248569310001}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -1.028214228}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -0.25331225968000004}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -1.03978665585}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -0.9898529404620001}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -2.1147955851599995}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -2.02319931248}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -3.01591320071}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -2.79392877972}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -1.36989737655}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -0.08908927405860001}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -1.31629972997}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -0.335133711279}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -0.8964038245489999}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 0.882925723775}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 3.61887171455}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 3.76167265851}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 6.02885383785}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -1.2913160376200001}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -5.34061698148}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 0.0197114412106}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 0.00164973435925}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -0.524034188751}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -0.043172420141}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 5.12810451254}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 0.9956871963639999}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 3.25487295325}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 2.53902855398}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 0.666384876322}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 0.841340283026}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 1.44122760241}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 0.831381332616}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 1.3597918472100001}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 2.6864698731200005}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -0.205719462375}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 1.7519679683200002}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 2.7485641239}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -0.10524783519}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 1.8490573450499999}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -0.252401575803}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 1.65464350758}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 4.04150922629}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 4.68128573608}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 4.40146048154}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 2.12310976146}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -4.795557302380001}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -4.71408892979}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 1.1257216645}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 0.9788031230769999}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 1.59219638796}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -0.436764228074}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 0.8490949832729999}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 1.58591469477}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": -0.40108335509099996}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 0.865217364515}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Atrial_Appendage", "value": 1.50530273893}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -1.66929180088}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -5.34151463541}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 5.595294858740001}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 1.0612866022}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 1.16837637744}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 1.77865241912}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 0.9194827304719999}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 0.869118020199}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 0.312714127676}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 1.0482192775799999}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 0.5861661238710001}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 2.25284675346}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 3.43724657051}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -1.37762804505}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -1.35625134266}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 3.85979767315}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 5.99015486874}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 4.79047348649}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 2.60037934132}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 0.9311222001389999}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -4.716775671530001}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -4.24120503018}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 0.260711135325}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 3.2218737025300004}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 6.150346965680001}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -1.71176200227}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -1.11369921827}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -2.71003582704}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 0.42698677357799997}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -1.85980741262}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 0.435203039864}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -1.09381420623}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 0.520036646677}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -1.0692055563299998}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -0.368094622149}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -0.7058773352609999}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -0.936102122245}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -0.239704758026}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -0.704319301089}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -0.7987567721600001}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -1.57289099061}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 3.66602256393}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -1.4123960616100002}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 0.603144967777}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -2.71459818576}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -0.422217100053}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -0.65797333423}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -0.426497042365}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -0.7408597501649999}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -0.751484346434}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 0.27578625250300004}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -0.0355289648634}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 0.035594223814800006}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 0.289030057501}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -0.202205333515}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": 0.07945725187210001}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Heart_Left_Ventricle", "value": -0.8769058875010001}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Spleen", "value": 0.0}], "data-a34e7b216b695c31da963e2ff5265bcb": [{"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 0.717170011311}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": -4.68438915678}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 6.989641923410001}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 4.0955372175699996}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 0.993913188462}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": -0.584262116365}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": -0.39435714208899997}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": -0.37287403569399996}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 0.795244033625}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 0.046410655948}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": -0.213067912135}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": -0.663022448362}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 4.58339342682}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 2.11031061905}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 1.8865591833900002}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 7.555672615339999}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 11.3112996584}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 9.441420272030001}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 1.98981250695}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": -0.34555644462800006}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": -4.43732423538}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": -7.83720053541}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 4.42603931075}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 1.68521374907}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 1.9845114045200003}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 2.00829332532}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": -0.149598325783}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 2.46924212027}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 2.45392697205}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 0.539312836624}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 2.2337086037000002}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 2.43839490599}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 2.21152979336}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 2.36690596921}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 3.31299808332}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 0.937072611677}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 2.97387776376}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 3.27903168647}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 0.838346960179}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 2.82389461001}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": -1.94707877937}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 5.12687044715}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 3.53980745047}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": -0.526773268715}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": -2.14026344782}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 0.22686265864900002}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": -0.35648547901100003}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": -0.502522845908}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 2.50083878743}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 2.44401825031}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 2.86321272304}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 1.21977598036}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 2.7909382869800003}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 2.85652152774}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 1.19223852428}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 2.73724132749}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Subcutaneous", "value": 1.8840911929799997}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Muscle_Skeletal", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -0.137607273967}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -4.07858772326}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -1.7896128951400003}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -3.12225455462}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -1.5034914857100001}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 2.5875919689}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -1.06800848192}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -1.24463488944}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -0.7439331955130001}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -0.879781687842}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -1.59531680857}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 2.42452293519}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -0.527499857846}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -3.39313770144}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -3.1508252843999998}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -2.0918704776}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -3.83784868266}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -3.20557364635}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 2.2658640063799997}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -1.29795713432}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -3.96641180861}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 1.9916582093000001}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -3.50549863674}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 1.93794471564}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 2.2805458184200003}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 1.02017892164}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -1.1659084959200001}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 0.150306385406}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 1.8949329430400002}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -1.07119520877}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 0.465327415428}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -0.286197723753}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 0.715103472863}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -0.204429200752}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -0.14628759768400001}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 0.727695260564}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 0.30473330328299997}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 0.006862907906899999}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 0.8890127418070001}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 0.433113825242}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 6.3740120645700005}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -5.10921373874}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 3.37608289319}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -0.558365403627}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -1.5719484608399998}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 0.516549171774}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -1.3795880875400002}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -1.13478792885}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -0.314252700959}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -0.352577638697}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 0.546886135564}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -0.222697505214}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 0.46778205031099995}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 0.551030207614}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": -0.113972481456}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 0.446755463505}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Cells_Transformed_fibroblasts", "value": 0.8713113101909999}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}], "data-46696b3ae9ca277d7032413742f41de3": [{"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Adrenal_Gland", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Pituitary", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Pancreas", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Thyroid", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Lung", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Ovary", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Prostate", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Testis", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Uterus", "value": 0.0}, {"phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Lambda GC": 2.0217, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 0, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "association": 17.02326, "code": "BFPC04", "LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "platelet count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Lambda GC": 1.2531, "category": "g: blood", "gene": "HIST1H1D", "index": 1, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "association": 8.802692, "code": "BLOO01", "LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "hemoglobin concentration", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 2, "LDSC Ratio": "0.0199 (0.0502)", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "association": 32.093024, "code": "BLOO03", "LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Lambda GC": 1.207, "category": "g: blood", "gene": "HIST1H1D", "index": 3, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "association": 14.674361, "code": "BLOO04", "LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "monocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 4, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "association": 10.45193, "code": "BLOO05", "LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "eosinophil count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Lambda GC": 1.1876, "category": "g: blood", "gene": "HIST1H1D", "index": 5, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "association": 11.724735, "code": "BLOO07", "LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "granulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 6, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "association": 7.914924000000001, "code": "BLOO09", "LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "myeloid white cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Lambda GC": 1.2168, "category": "g: blood", "gene": "HIST1H1D", "index": 7, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9623 (0.02)", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "association": 10.140916, "code": "BLOO10", "LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "lymphocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 8, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "association": 23.81953, "code": "BLOO11", "LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "white blood cell count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Lambda GC": 1.2431, "category": "g: blood", "gene": "HIST1H1D", "index": 9, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "association": 22.670826, "code": "BLOO12", "LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 10, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "association": 6.116066, "code": "BLOO13", "LDSC Mean Chi-Square": 1.5207, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Lambda GC": 1.1908, "category": "g: blood", "gene": "HIST1H1D", "index": 11, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "association": 12.269824, "code": "BLOO16", "LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Hematocrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Lambda GC": 1.2103, "category": "g: blood", "gene": "HIST1H1D", "index": 12, "LDSC Ratio": "0.0221 (0.0511)", "LDSC Intercept": "1.0104 (0.024)", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "association": 22.223924, "code": "BLOO20", "LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 13, "LDSC Ratio": "0.0417 (0.0682)", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "association": 10.840313, "code": "BLOO21", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Lambda GC": 1.2201, "category": "g: blood", "gene": "HIST1H1D", "index": 14, "LDSC Ratio": "0.0416 (0.0672)", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "association": 10.6094, "code": "BLOO22", "LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Lambda GC": 1.0988, "category": "g: blood", "gene": "HIST1H1D", "index": 15, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "association": 48.617245000000004, "code": "BLOO25", "LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Lambda GC": 1.194, "category": "g: blood", "gene": "HIST1H1D", "index": 16, "LDSC Ratio": "0.012 (0.0532)", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "association": 78.28195500000001, "code": "BLOO26", "LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 17, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "association": 49.249762, "code": "BLOO27", "LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Mean platelet volume", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Lambda GC": 1.2005, "category": "g: blood", "gene": "HIST1H1D", "index": 18, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "association": 7.474359, "code": "BLOO29", "LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Lambda GC": 1.2136, "category": "g: blood", "gene": "HIST1H1D", "index": 19, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.961 (0.0204)", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "association": 6.647528, "code": "BLOO30", "LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Plateletcrit", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Lambda GC": 1.2498, "category": "g: blood", "gene": "HIST1H1D", "index": 20, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "association": 7.470428, "code": "BLOO33", "LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Red cell distribution width", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Lambda GC": 1.1651, "category": "g: blood", "gene": "HIST1H1D", "index": 21, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "association": 49.668067, "code": "BLOO35", "LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "study size": 173480, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Lambda GC": 1.2038, "category": "g: blood", "gene": "HIST1H1D", "index": 22, "LDSC Ratio": "0.0197 (0.0653)", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "association": 29.309343, "code": "BLOO36", "LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Attainment of a college or a university degree", "PMID or link": "27046643", "study size": 111114, "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "LDSC Lambda GC": 1.3068, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 23, "LDSC Ratio": "0.0875 (0.0232)", "LDSC Intercept": "1.0322 (0.0085)", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "association": 8.32661, "code": "EDUC02", "LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "study size": 766345, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Lambda GC": 2.094, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 24, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.979 (0.016)", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "association": 15.354076999999998, "code": "EDUC03", "LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Lambda GC": 2.0175, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 25, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Intercept": "1.043 (0.0163)", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "association": 26.594022, "code": "FATM04", "LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Extreme Height", "PMID or link": "23563607", "study size": 16196, "phenotype_reference": "Extreme Height (HEIG02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 26, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8886 (0.0149)", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "association": 9.336243, "code": "HEIG02", "LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Height 2014", "PMID or link": "25282103", "study size": 253288, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Lambda GC": 2.0007, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 27, "LDSC Ratio": "0.1178 (0.0152)", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "association": 22.449136, "code": "HEIG03", "LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Height female 2013", "PMID or link": "23754948", "study size": 73137, "phenotype_reference": "Height female 2013 (HEIG04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 28, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8259 (0.0107)", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "association": 17.124343, "code": "HEIG04F", "LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Height male 2013", "PMID or link": "23754948", "study size": 60586, "phenotype_reference": "Height male 2013 (HEIG04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 29, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8343 (0.0098)", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "association": 10.014955, "code": "HEIG04M", "LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Hip circumference European male", "PMID or link": "25673412", "study size": 57568, "phenotype_reference": "Hip circumference European male (HIPC01M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 30, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8715 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "association": 8.999392, "code": "HIPC01M", "LDSC Mean Chi-Square": 1.0853, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Hip circumference European", "PMID or link": "25673412", "study size": 145308, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 31, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "association": 11.109741, "code": "HIPC01", "LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male", "PMID or link": "25673412", "study size": 57598, "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 32, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8692 (0.009)", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "association": 8.393339, "code": "HIPC02M", "LDSC Mean Chi-Square": 1.0839, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "study size": 145414, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 33, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "association": 12.425286, "code": "HIPC02", "LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "study size": 86792, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 34, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "association": 16.366854999999997, "code": "HIPC03F", "LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "study size": 56939, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 35, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Total Observed Scale h2": "0.204 (0.016)", "association": 13.1015, "code": "HIPC03M", "LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "study size": 143579, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 36, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "association": 13.832063, "code": "HIPC03", "LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 86647, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 37, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "association": 17.774794, "code": "HIPC04F", "LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 56978, "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 38, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.866 (0.0095)", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "association": 12.718376, "code": "HIPC04M", "LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 143575, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 39, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8066 (0.009)", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "association": 26.804405, "code": "HIPC04", "LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Transferrin", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Lambda GC": 1.0864, "category": "a: metabolic", "gene": "HIST1H1D", "index": 40, "LDSC Ratio": "0.6135 (0.1678)", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "association": 13.509578, "code": "IRON02", "LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Transferrin saturation", "PMID or link": "25352340", "study size": 48972, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Lambda GC": 1.0436, "category": "a: metabolic", "gene": "HIST1H1D", "index": 41, "LDSC Ratio": "0.6579 (0.2019)", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "association": 24.27005, "code": "IRON03", "LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "study size": 353972, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Lambda GC": 2.1851, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 42, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Intercept": "1.11 (0.0274)", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "association": 23.225154999999997, "code": "LEAN06", "LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "study size": 23210, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Lambda GC": 1.1715, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 43, "LDSC Ratio": "0.4868 (0.0447)", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "association": 12.518728999999999, "code": "LUPU01", "LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "study size": 24751, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Lambda GC": 1.0557, "category": "e: autoimmune", "gene": "HIST1H1D", "index": 44, "LDSC Ratio": "0.0664 (0.1142)", "LDSC Intercept": "1.0058 (0.01)", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "association": 37.405287, "code": "PRSC01", "LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Rheumatoid Arthritis 2014", "PMID or link": "24390342", "study size": 79799, "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Lambda GC": 1.0466, "category": "h: musculoskeletal", "gene": "HIST1H1D", "index": 45, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.9435 (0.0122)", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "association": 6.47881, "code": "RHEU01", "LDSC Mean Chi-Square": 1.1179999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "study size": 82315, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Lambda GC": 1.5696, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 46, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Intercept": "1.033 (0.0104)", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "association": 12.724480999999999, "code": "SCHI01", "LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "study size": 77096, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Lambda GC": 1.5622, "category": "c: mental and behavioural", "gene": "HIST1H1D", "index": 47, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "association": 11.460134, "code": "SCHI02", "LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Waist circumference EU", "PMID or link": "25673412", "study size": 231927, "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 48, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7423 (0.0086)", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "association": 6.336176999999999, "code": "WAIS01", "LDSC Mean Chi-Square": 1.1487, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Waist circumference all ancestries", "PMID or link": "25673412", "study size": 244261, "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 49, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.7406 (0.0087)", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "association": 6.5981, "code": "WAIS02", "LDSC Mean Chi-Square": 1.1516, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Waist circumference EU female BMI-adjusted", "PMID or link": "25673412", "study size": 91952, "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 50, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8926 (0.0084)", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "association": 6.3720300000000005, "code": "WAIS03F", "LDSC Mean Chi-Square": 1.0848, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Waist circumference EU male BMI-adjusted", "PMID or link": "25673412", "study size": 60393, "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 51, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8744 (0.008)", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "association": 9.298337, "code": "WAIS03M", "LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "study size": 229089, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 52, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "association": 14.206112, "code": "WAIS03", "LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Waist circumference all ancestries female BMI-adjusted", "PMID or link": "25673412", "study size": 132715, "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 53, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8907 (0.0084)", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "association": 7.861603, "code": "WAIS04F", "LDSC Mean Chi-Square": 1.0851, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Waist circumference all ancestries male BMI-adjusted", "PMID or link": "25673412", "study size": 109077, "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 54, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8729 (0.0079)", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "association": 10.216547, "code": "WAIS04M", "LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Waist circumference all ancestries BMI-adjusted", "PMID or link": "25673412", "study size": 153478, "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "LDSC Lambda GC": 1.0466, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 55, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8232 (0.0084)", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "association": 14.155728, "code": "WAIS04", "LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}, {"phenotype": "Weight European female", "PMID or link": "23754948", "study size": 67592, "phenotype_reference": "Weight European female (WEIG04F) ", "LDSC Lambda GC": 0.9986, "category": "b: anthropometric", "gene": "HIST1H1D", "index": 56, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Intercept": "0.8241 (0.0071)", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "association": 10.157110000000001, "code": "WEIG04F", "LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "ENSEMBL": "ENSG00000124575", "variable": "Vagina", "value": 0.0}]}};
    var embed_opt = {"mode": "vega-lite"};

    function showError(el, error){
        el.innerHTML = ('<div class="error">'
                        + '<p>JavaScript Error: ' + error.message + '</p>'
                        + "<p>This usually means there's a typo in your chart specification. "
                        + "See the javascript console for the full traceback.</p>"
                        + '</div>');
        throw error;
    }
    const el = document.getElementById('vis');
    vegaEmbed("#vis", spec, embed_opt)
      .catch(error => showError(el, error));
  </script>
</body>
</html>
