
<!DOCTYPE html>

<html>
<head><style>.vega-actions a {margin-right: 12px;color: #757575;font-weight: normal;font-size: 13px;}.error {color: red;}</style><script src="https://cdn.jsdelivr.net/npm//vega@4" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-lite@2.6.0" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-embed@3" type="text/javascript"></script><meta content="text/html; charset=utf-8" http-equiv="Content-Type"/><meta content="IE=edge" http-equiv="X-UA-Compatible"/><meta content="width=device-width, initial-scale=1" name="viewport"/><link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/Font-Awesome-4.7/css/font-awesome.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap-theme.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/images/gnomic_icon.png" rel="icon" type="image/png"/><script src="https://phenviz.navigome.com/js/jquery.min.js"></script><script src="https://phenviz.navigome.com/js/bootstrap.js"></script><title>Gene profile: ENSG00000187145</title></head>
<body>
<h4>Gene profile across phenotypes - ENSG00000187145 (MRPS21)</h4><div id="vis"></div><div class="col-lg-12 col-md-12 content-item content-item-1"><p>This is a visualization of GWAS-derived gene associations  across Navigome phenotypes. Only phenotypes for which the gene has at least one significant association (p-value â‰¤ 0.05/20000) in <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> <b>and/or</b> <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a> analyses are shown. The first panel is a visualization of gene-phenotype associations obtained using <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> (-log10(p-value)). Hover over any bar to highlight the phenotype in the other panels. The other panels are parallel coordinates visualizations for tissue-wise associations (z-scores)computed using <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a>  and eQTL expression data from GTEx and DGN. Significance thresholds are delineated with black lines. The parallel coordinates show which phenotypes have a significant association and in which tissues, based on gene expression data. Clicking on a specific phenotype will lead to the reference of the GWAS study.</p></div>
<script type="text/javascript">
    var spec = {"config": {"view": {"width": 400, "height": 300}}, "vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "bar", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "href": {"type": "nominal", "field": "phenotype_link"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "quantitative", "axis": {"title": "MAGMA -log10(p)"}, "field": "value"}, "y": {"type": "nominal", "axis": {"title": ""}, "field": "phenotype_reference"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-4e93ee1a91df87c61b7d73e211f0f4d1"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "x": {"type": "quantitative", "field": "ThresholdVal"}}}], "title": "MAGMA gene-wise association"}], "center": true, "data": {"name": "data-f327008b4271ba69131b90121738836c"}}, {"vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Nervous system", "width": 500}], "center": true, "data": {"name": "data-436d496cf78b7348dbf4688d2cc94bc7"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Digestive System", "width": 500}], "center": true, "data": {"name": "data-b68a55ea046c327fac9299f36709f94e"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Blood, Arteries and Heart", "width": 500}], "center": true, "data": {"name": "data-79552c639cb54ebdfb55648a5ff48e3a"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Adipose Tissue, Muscle, Skin and Cells", "width": 500}], "center": true, "data": {"name": "data-62fc743d7e6f1676cda1eecad93ea81b"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-9.64353257898, 9.64353257898]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Endocrine System and Others", "width": 500}], "center": true, "data": {"name": "data-118f57edfa0a8a9b44ef432d1909fe1b"}}]}], "spacing": 40, "$schema": "https://vega.github.io/schema/vega-lite/v2.6.0.json", "datasets": {"data-f327008b4271ba69131b90121738836c": [{"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "magma_log10p", "value": 9.027075}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "magma_log10p", "value": 8.075039}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "magma_log10p", "value": 8.092578}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "magma_log10p", "value": 13.866557}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "magma_log10p", "value": 4.4924}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "magma_log10p", "value": 6.593477}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "magma_log10p", "value": 3.06077}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "magma_log10p", "value": 2.777726}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "magma_log10p", "value": 8.74213}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "magma_log10p", "value": 2.716902}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "magma_log10p", "value": 12.926832000000001}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "magma_log10p", "value": 3.453507}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "magma_log10p", "value": 9.157534}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "magma_log10p", "value": 3.811156}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "magma_log10p", "value": 4.027117}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "magma_log10p", "value": 12.158196}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "magma_log10p", "value": 4.976501}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "magma_log10p", "value": 3.445559}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "magma_log10p", "value": 3.460924}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "magma_log10p", "value": 3.55544}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "magma_log10p", "value": 3.675121}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "magma_log10p", "value": 2.4355}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "magma_log10p", "value": 3.2362130000000002}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "magma_log10p", "value": 2.534736}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "magma_log10p", "value": 2.946883}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "magma_log10p", "value": 5.65606}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "magma_log10p", "value": 6.782911}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "magma_log10p", "value": 6.266979}], "data-4e93ee1a91df87c61b7d73e211f0f4d1": [{"Threshold": "MAGMA -log10(p) \u2265 5.60206", "ThresholdVal": 5.60206}], "data-436d496cf78b7348dbf4688d2cc94bc7": [{"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.49514839231299995}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.3838981587}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.385464095518}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.272350697695}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.02203102822}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.22195025651}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.417707336575}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.82394612209}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 2.37178975114}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": -2.12585726042}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": -2.1029091609}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.5421784873100001}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": -0.363689632619}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.28421064931}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.3131370047700002}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.58904667717}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.421709299273}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.7917835514329999}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.34770165031}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.771920483338}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.3108104372}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.69610827555}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.6398652449600001}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cerebellar_Hemisphere", "value": 1.6809637596}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.63285643372}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Brain_Cerebellar_Hemisphere", "value": -1.72870482938}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0415627831729}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.049801538735}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Brain_Cerebellum", "value": -4.3853463369300005}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Brain_Cerebellum", "value": 1.9092395751700002}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellum", "value": 3.59358780056}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellum", "value": 3.0347351229400004}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellum", "value": 2.3760778511700003}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellum", "value": 2.8111575740099997}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellum", "value": 1.32668811219}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellum", "value": 0.8324189088330001}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellum", "value": 0.43406557311300004}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellum", "value": -1.54056467285}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellum", "value": 0.0501830631401}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellum", "value": -1.10760445478}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cerebellum", "value": 2.8488227078799997}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Cerebellum", "value": 2.87292742842}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Cerebellum", "value": 3.03095555958}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Brain_Cerebellum", "value": 5.841533192259999}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Brain_Cerebellum", "value": 1.8963364102900002}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cerebellum", "value": 1.73059420778}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cerebellum", "value": 1.90307286384}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cerebellum", "value": 1.68710094336}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cerebellum", "value": 1.8509658340700001}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cerebellum", "value": 1.59665533155}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cerebellum", "value": 1.19706763027}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cerebellum", "value": 1.57021521468}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cerebellum", "value": 1.16721437715}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Brain_Cerebellum", "value": 2.0391398631}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Cerebellum", "value": -0.03734006404180001}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Cerebellum", "value": 0.10027891283500001}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Brain_Hypothalamus", "value": -1.92956105654}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Brain_Hypothalamus", "value": 3.11061521074}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hypothalamus", "value": 2.94797689503}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hypothalamus", "value": 8.06173206065}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hypothalamus", "value": 4.25391924314}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hypothalamus", "value": 4.1673920326800005}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hypothalamus", "value": 3.6165669344699998}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hypothalamus", "value": 3.5710580411699997}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hypothalamus", "value": -4.04327181384}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hypothalamus", "value": -0.04974096370619999}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hypothalamus", "value": 4.4852331304}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hypothalamus", "value": -3.4117537474699997}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Hypothalamus", "value": 1.0715181952}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Hypothalamus", "value": 2.06880736879}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Hypothalamus", "value": 2.1855010829}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Brain_Hypothalamus", "value": 3.3127860521200003}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Brain_Hypothalamus", "value": 2.9582298938599996}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Hypothalamus", "value": 2.54890500548}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Hypothalamus", "value": 1.1836677092299999}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Hypothalamus", "value": 2.51363995827}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Hypothalamus", "value": 1.0780740956700001}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Hypothalamus", "value": 1.25702205856}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Hypothalamus", "value": -0.17701636243500002}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Hypothalamus", "value": 1.31892945548}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Hypothalamus", "value": -0.19334077208}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Brain_Hypothalamus", "value": 1.70531172943}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Hypothalamus", "value": 0.43050338292499996}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Hypothalamus", "value": 0.154160915911}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Nerve_Tibial", "value": -3.25757681797}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Nerve_Tibial", "value": 3.0020640023299996}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Nerve_Tibial", "value": 1.17970056114}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Nerve_Tibial", "value": 6.88251717345}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Nerve_Tibial", "value": 5.2556280169}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Nerve_Tibial", "value": 5.23906102822}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Nerve_Tibial", "value": 3.32569563489}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Nerve_Tibial", "value": 2.5909653489099997}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Nerve_Tibial", "value": -1.8660753698900003}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Nerve_Tibial", "value": -2.75860457447}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Nerve_Tibial", "value": 3.04983558382}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Nerve_Tibial", "value": -2.8959997323900004}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Nerve_Tibial", "value": 0.640228429052}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Nerve_Tibial", "value": 4.27880567727}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Nerve_Tibial", "value": 4.516172066269999}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Nerve_Tibial", "value": 4.95714652926}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Nerve_Tibial", "value": 3.23777351267}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Nerve_Tibial", "value": 3.2789316677099998}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Nerve_Tibial", "value": 3.3742167114}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Nerve_Tibial", "value": 3.19175388089}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Nerve_Tibial", "value": 3.2535809051999998}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Nerve_Tibial", "value": 2.0748320354}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Nerve_Tibial", "value": 2.04205322622}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Nerve_Tibial", "value": 2.0641810232599997}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Nerve_Tibial", "value": 1.9853375905500001}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Nerve_Tibial", "value": 3.08703504025}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Nerve_Tibial", "value": -3.0048145817000003}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Nerve_Tibial", "value": -2.89128472241}], "data-c4c9a654d45741ebc60aa567761bff19": [{"Text": "Significance Threshold", "Threshold": "PrediXcan zscore \u2265 4.70813", "ThresholdVal": 4.70813}], "data-4a3aa5bf8c5a41879455008930c25774": [{"Threshold": "PrediXcan zscore \u2264 -4.70813", "ThresholdVal": -4.70813}], "data-b68a55ea046c327fac9299f36709f94e": [{"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Colon_Sigmoid", "value": -1.30372918675}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Colon_Sigmoid", "value": 4.33593860004}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Sigmoid", "value": 0.889671621457}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Sigmoid", "value": 1.28102718435}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Sigmoid", "value": 3.63203386491}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Sigmoid", "value": 3.73852552834}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Sigmoid", "value": 3.3218888148500003}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Sigmoid", "value": 2.75871248247}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Sigmoid", "value": 0.278857093469}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Sigmoid", "value": -1.3177681113500002}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Sigmoid", "value": 0.326694815876}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Sigmoid", "value": -2.38348681697}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Sigmoid", "value": -0.5557945512420001}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Colon_Sigmoid", "value": 3.5642027799300005}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Colon_Sigmoid", "value": 3.9224527352}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Colon_Sigmoid", "value": 2.44914503685}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Colon_Sigmoid", "value": 5.21229366721}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Colon_Sigmoid", "value": 5.01624250555}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Colon_Sigmoid", "value": 5.1892559507}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Colon_Sigmoid", "value": 4.84528160699}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Colon_Sigmoid", "value": 4.853881643469999}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Colon_Sigmoid", "value": 4.59478747296}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Colon_Sigmoid", "value": 4.50011399016}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Colon_Sigmoid", "value": 4.5127634610900005}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Colon_Sigmoid", "value": 4.38865083339}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Colon_Sigmoid", "value": 5.116307920280001}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Colon_Sigmoid", "value": -3.2951151786199997}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Colon_Sigmoid", "value": -3.2050519525700003}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.871816329682}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Esophagus_Gastroesophageal_Junction", "value": 5.44090548584}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -2.1601196091099997}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.9031528773800002}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 3.3557386308200003}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 3.46936062066}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.3009670032700003}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.31483306177}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.5438775448}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -3.93843760584}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.443297301138}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -1.30189992914}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -2.55259069343}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.05754970006}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.22601832305}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.68949651173}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.90498123216}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.22464179784}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.24813135631}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.11210188754}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.10744501798}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.76911264955}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Gastroesophageal_Junction", "value": 3.03447114331}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.70396171481}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.93667335982}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.9987741730900002}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Esophagus_Gastroesophageal_Junction", "value": -3.96350620801}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Esophagus_Gastroesophageal_Junction", "value": -3.87654631255}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Esophagus_Mucosa", "value": 0.613990136705}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Esophagus_Mucosa", "value": 3.5768534623900003}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Mucosa", "value": -2.9501310401}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Mucosa", "value": 1.35265481524}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Mucosa", "value": 1.43426094713}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Mucosa", "value": 3.38208637929}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Mucosa", "value": 2.58532106861}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Mucosa", "value": 1.9212805897299998}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Mucosa", "value": -0.43576823099399997}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Mucosa", "value": -2.20014127091}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Mucosa", "value": 1.24110031285}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Mucosa", "value": -1.9811353009900001}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Mucosa", "value": -2.02787836097}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Esophagus_Mucosa", "value": 3.9294436065900005}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Esophagus_Mucosa", "value": 4.20947804607}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Esophagus_Mucosa", "value": 1.47483462718}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Esophagus_Mucosa", "value": 4.13094366253}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Mucosa", "value": 5.206245012159999}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Mucosa", "value": 5.04179216612}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Mucosa", "value": 5.04503786313}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Mucosa", "value": 4.80980571556}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Mucosa", "value": 3.9252351964599996}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Mucosa", "value": 4.09013530475}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Mucosa", "value": 3.85863885648}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Mucosa", "value": 3.9786123524900003}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Esophagus_Mucosa", "value": 3.4412124573500003}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Esophagus_Mucosa", "value": -2.22984987722}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Esophagus_Mucosa", "value": -2.47512339735}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Esophagus_Muscularis", "value": -3.33790388149}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Esophagus_Muscularis", "value": 3.80858802989}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Muscularis", "value": 0.314753451841}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Muscularis", "value": 3.3266661859799997}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Muscularis", "value": 4.02400272996}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Muscularis", "value": 4.51440330226}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Muscularis", "value": 2.87495172423}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Muscularis", "value": 2.00405066754}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Muscularis", "value": 0.158150349668}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Muscularis", "value": -2.9772201432}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Muscularis", "value": 0.849458701832}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Muscularis", "value": -2.29949784159}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Esophagus_Muscularis", "value": 0.254129043541}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Esophagus_Muscularis", "value": 4.367252296559999}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Esophagus_Muscularis", "value": 4.6116233381}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Esophagus_Muscularis", "value": 4.7903215298}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Esophagus_Muscularis", "value": 3.9186524131300002}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Muscularis", "value": 3.8020026600800003}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Muscularis", "value": 3.98556675056}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Muscularis", "value": 3.7064508070199995}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Muscularis", "value": 3.86875948939}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Muscularis", "value": 2.8161231376599996}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Muscularis", "value": 2.93044117801}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Muscularis", "value": 2.76970322108}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Esophagus_Muscularis", "value": 2.85916284979}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Esophagus_Muscularis", "value": 3.38759166839}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Esophagus_Muscularis", "value": -2.6797322862}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Esophagus_Muscularis", "value": -2.51836749922}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Liver", "value": -3.23704012385}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Liver", "value": 0.265337287894}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Liver", "value": -1.26037085073}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Liver", "value": 1.9646796519}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Liver", "value": 1.6516165161399998}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Liver", "value": 3.3279680019}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Liver", "value": 2.15313526361}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Liver", "value": 1.74885800118}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Liver", "value": -3.08872431787}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Liver", "value": -1.6376164033}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Liver", "value": 3.99902782921}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Liver", "value": -2.11614190625}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Liver", "value": 2.06843638647}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Liver", "value": 3.6006918426300003}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Liver", "value": 3.8180662802199996}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Liver", "value": 4.4508900431}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Liver", "value": 0.8785622034039999}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Liver", "value": 2.89312328233}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Liver", "value": 2.68571874301}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Liver", "value": 2.8204134081700003}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Liver", "value": 2.61218248577}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Liver", "value": 1.54994443208}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Liver", "value": 1.5474134672600002}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Liver", "value": 1.5242779298}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Liver", "value": 1.50882296557}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Liver", "value": 2.48610519687}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Liver", "value": -0.785452626656}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Liver", "value": -0.522320658755}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Stomach", "value": -4.58214629773}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Stomach", "value": 1.7313846996400002}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Stomach", "value": 2.1999148222}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Stomach", "value": 5.452385093459999}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Stomach", "value": 3.9447794351699996}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Stomach", "value": 4.34524831754}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Stomach", "value": 2.68342251866}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Stomach", "value": 1.88161140878}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Stomach", "value": -0.100942811696}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Stomach", "value": -2.58069876477}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Stomach", "value": 1.0239596666700002}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Stomach", "value": -2.2532986084599997}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Stomach", "value": 0.862692000773}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Stomach", "value": 4.27526001869}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Stomach", "value": 4.5480763280800005}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Stomach", "value": 5.08519264095}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Stomach", "value": 2.0195829278599997}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Stomach", "value": 3.6464553426699995}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Stomach", "value": 3.94158189486}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Stomach", "value": 3.5548127503300004}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Stomach", "value": 3.83365968132}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Stomach", "value": 2.6133026406}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Stomach", "value": 2.9157509463499998}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Stomach", "value": 2.57002727168}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Stomach", "value": 2.84303579833}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Stomach", "value": 2.85312671741}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Stomach", "value": -2.0315542467900003}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Stomach", "value": -1.88258174461}], "data-79552c639cb54ebdfb55648a5ff48e3a": [{"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "DGN_Whole_Blood", "value": -2.1576995580442984}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "DGN_Whole_Blood", "value": 2.8261148638125015}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "DGN_Whole_Blood", "value": -1.1956208288191101}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "DGN_Whole_Blood", "value": 9.048561941582562}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "DGN_Whole_Blood", "value": 6.754103605417179}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "DGN_Whole_Blood", "value": 5.653427749960872}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "DGN_Whole_Blood", "value": 4.023935909842812}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "DGN_Whole_Blood", "value": 3.4364013207266355}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "DGN_Whole_Blood", "value": -3.505607301420216}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "DGN_Whole_Blood", "value": -2.584565175997411}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "DGN_Whole_Blood", "value": 4.829095433474278}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "DGN_Whole_Blood", "value": -3.7753547558103837}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "DGN_Whole_Blood", "value": -0.5334325898562953}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "DGN_Whole_Blood", "value": 3.7768733289958734}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "DGN_Whole_Blood", "value": 3.902238723720647}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "DGN_Whole_Blood", "value": 4.164788445462769}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "DGN_Whole_Blood", "value": 2.6430743764759828}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "DGN_Whole_Blood", "value": 2.565501179337711}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "DGN_Whole_Blood", "value": 2.4705830101216484}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "DGN_Whole_Blood", "value": 2.4990201428698597}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "DGN_Whole_Blood", "value": 2.397597036532342}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "DGN_Whole_Blood", "value": 0.9577881610124692}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "DGN_Whole_Blood", "value": 0.9188300893522934}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "DGN_Whole_Blood", "value": 0.9414458539075397}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "DGN_Whole_Blood", "value": 0.8937421831313992}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "DGN_Whole_Blood", "value": 1.7900220238708693}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "DGN_Whole_Blood", "value": -4.207284649813109}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "DGN_Whole_Blood", "value": -4.214196422816968}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Whole_Blood", "value": -0.993250852369}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Whole_Blood", "value": 4.41780368461}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Whole_Blood", "value": -2.57634514068}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Whole_Blood", "value": 2.71436449399}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Whole_Blood", "value": 2.32148036701}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Whole_Blood", "value": 3.0624701553500002}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Whole_Blood", "value": 1.4783378323199998}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Whole_Blood", "value": 0.502146618056}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Whole_Blood", "value": 0.24354016041499998}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Whole_Blood", "value": -4.88097340624}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Whole_Blood", "value": 1.4510763577399999}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Whole_Blood", "value": -0.7293162181110001}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Whole_Blood", "value": -1.40667473007}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Whole_Blood", "value": 4.33381053181}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Whole_Blood", "value": 4.50369987071}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Whole_Blood", "value": 3.82012737635}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Whole_Blood", "value": 4.57829305862}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Whole_Blood", "value": 3.72998797325}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Whole_Blood", "value": 4.0127450521300005}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Whole_Blood", "value": 3.6388860225300004}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Whole_Blood", "value": 3.91309472938}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Whole_Blood", "value": 2.50439057779}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Whole_Blood", "value": 2.7374961822799997}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Whole_Blood", "value": 2.46365983429}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Whole_Blood", "value": 2.66922669672}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Whole_Blood", "value": 3.1763653602599997}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Whole_Blood", "value": -4.3359931956699995}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Whole_Blood", "value": -4.19023182129}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Artery_Aorta", "value": -3.75655543171}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Artery_Aorta", "value": 2.12799362621}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Aorta", "value": 1.5835701720700002}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Aorta", "value": 4.861143342169999}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Aorta", "value": 4.487521625209999}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Aorta", "value": 5.33280016579}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Aorta", "value": 3.68538430435}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Aorta", "value": 3.11031654285}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Aorta", "value": -1.24767598844}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Aorta", "value": -1.5308810667700001}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Aorta", "value": 2.0045869500999998}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Aorta", "value": -3.4365920754800006}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Aorta", "value": 0.352399261839}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Artery_Aorta", "value": 3.4602736867400004}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Artery_Aorta", "value": 3.6676427240199994}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Artery_Aorta", "value": 5.24848740235}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Artery_Aorta", "value": 2.2536810895599997}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Aorta", "value": 2.7894611091900003}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Aorta", "value": 2.92785735697}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Aorta", "value": 2.6757642445400003}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Aorta", "value": 2.81108816051}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Aorta", "value": 2.1464473481599997}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Aorta", "value": 2.41924608938}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Aorta", "value": 2.11343065858}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Aorta", "value": 2.3597712202900003}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Artery_Aorta", "value": 2.3090565726900003}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Artery_Aorta", "value": -2.31138672775}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Artery_Aorta", "value": -2.12996161225}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Artery_Tibial", "value": -1.3045644063200001}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Artery_Tibial", "value": 5.07834548818}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Tibial", "value": -2.39920490154}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Tibial", "value": 0.660867648144}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Tibial", "value": 2.5314522094400003}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Tibial", "value": 3.26523790277}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Tibial", "value": 1.9018235551400002}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Tibial", "value": 0.8066913620749999}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Tibial", "value": 1.30782462595}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Tibial", "value": -4.76268303964}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Tibial", "value": 0.201371719933}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Tibial", "value": -0.838143010947}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Artery_Tibial", "value": -1.69733748361}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Artery_Tibial", "value": 4.79812074469}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Artery_Tibial", "value": 5.12590695701}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Artery_Tibial", "value": 2.79003393402}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Artery_Tibial", "value": 4.84473747288}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Tibial", "value": 4.31175624033}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Tibial", "value": 4.75917311075}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Tibial", "value": 4.21550884468}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Tibial", "value": 4.50984323611}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Tibial", "value": 3.18451751335}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Tibial", "value": 3.4515938398800006}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Tibial", "value": 3.1543595218299996}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Artery_Tibial", "value": 3.3869959776200003}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Artery_Tibial", "value": 2.4661842132}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Artery_Tibial", "value": -2.83975182514}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Artery_Tibial", "value": -2.85490145985}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Heart_Atrial_Appendage", "value": -1.32059578847}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Heart_Atrial_Appendage", "value": 4.0420376957599995}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Atrial_Appendage", "value": -1.69116671481}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Atrial_Appendage", "value": 3.68977029132}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Atrial_Appendage", "value": 2.8515120978099997}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Atrial_Appendage", "value": 3.37093355371}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Atrial_Appendage", "value": 2.03178442625}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Atrial_Appendage", "value": 1.12170499072}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Atrial_Appendage", "value": 0.689552971025}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Atrial_Appendage", "value": -3.8869134641900005}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Atrial_Appendage", "value": 0.549659177117}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Atrial_Appendage", "value": -1.1971853479999999}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Atrial_Appendage", "value": -1.22078530233}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Heart_Atrial_Appendage", "value": 3.88534339013}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Heart_Atrial_Appendage", "value": 4.03848222469}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Heart_Atrial_Appendage", "value": 3.38668004148}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Heart_Atrial_Appendage", "value": 3.65214981035}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Heart_Atrial_Appendage", "value": 3.89588127111}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Heart_Atrial_Appendage", "value": 3.83208338265}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Heart_Atrial_Appendage", "value": 3.7926131435800006}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Heart_Atrial_Appendage", "value": 3.7032875824300002}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Heart_Atrial_Appendage", "value": 2.90962737922}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Heart_Atrial_Appendage", "value": 2.97942148807}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Heart_Atrial_Appendage", "value": 2.85625516923}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Heart_Atrial_Appendage", "value": 2.8854727263499997}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Heart_Atrial_Appendage", "value": 1.7183487769999999}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Heart_Atrial_Appendage", "value": -3.58632803655}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Heart_Atrial_Appendage", "value": -3.4480419145599996}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Heart_Left_Ventricle", "value": -0.53514227561}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Heart_Left_Ventricle", "value": 3.4487252222099998}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Left_Ventricle", "value": -1.1284962676899999}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Left_Ventricle", "value": 5.414648647490001}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Left_Ventricle", "value": 4.46634491873}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Left_Ventricle", "value": 4.22064088005}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Left_Ventricle", "value": 3.10595677458}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Left_Ventricle", "value": 2.41399499931}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Left_Ventricle", "value": -1.36243743041}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Left_Ventricle", "value": -3.22907524069}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Left_Ventricle", "value": 2.71539988843}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Left_Ventricle", "value": -2.46453687237}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Heart_Left_Ventricle", "value": -1.85804974262}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Heart_Left_Ventricle", "value": 3.9185305313199996}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Heart_Left_Ventricle", "value": 4.1185889401399995}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Heart_Left_Ventricle", "value": 3.58527022006}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Heart_Left_Ventricle", "value": 3.87519342075}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Heart_Left_Ventricle", "value": 3.6543283664499997}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Heart_Left_Ventricle", "value": 3.6357060881199996}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Heart_Left_Ventricle", "value": 3.5430214374199998}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Heart_Left_Ventricle", "value": 3.5115505009599994}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Heart_Left_Ventricle", "value": 1.99644277661}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Heart_Left_Ventricle", "value": 2.0212878611900003}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Heart_Left_Ventricle", "value": 1.95424481062}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Heart_Left_Ventricle", "value": 1.97692242324}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Heart_Left_Ventricle", "value": 3.1057809632200004}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Heart_Left_Ventricle", "value": -4.3545224850599995}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Heart_Left_Ventricle", "value": -4.454172111459999}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Spleen", "value": -2.0300518312799998}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Spleen", "value": 4.31516601156}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Spleen", "value": -0.9314698319560001}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Spleen", "value": 4.1902066939300004}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Spleen", "value": 3.2722924202}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Spleen", "value": 4.161294101559999}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Spleen", "value": 2.54570859397}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Spleen", "value": 1.65593552068}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Spleen", "value": -0.444717979416}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Spleen", "value": -3.66962704631}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Spleen", "value": 1.75605555042}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Spleen", "value": -1.9148308045700002}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Spleen", "value": -0.957137456371}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Spleen", "value": 4.1530406522400005}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Spleen", "value": 4.3399762948000005}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Spleen", "value": 3.70697053181}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Spleen", "value": 3.9769639835}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Spleen", "value": 3.5887540815400003}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Spleen", "value": 3.69381200144}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Spleen", "value": 3.4985615861699997}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Spleen", "value": 3.59267378604}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Spleen", "value": 2.41733785194}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Spleen", "value": 2.66002270087}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Spleen", "value": 2.3773075607}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Spleen", "value": 2.5936852091900002}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Spleen", "value": 2.2526317906599997}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Spleen", "value": -3.84387300306}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Spleen", "value": -3.7338951831699996}], "data-62fc743d7e6f1676cda1eecad93ea81b": [{"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Adipose_Subcutaneous", "value": -3.5530402017300005}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Adipose_Subcutaneous", "value": 2.67901711527}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Subcutaneous", "value": 0.931620760661}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Subcutaneous", "value": 3.38227532246}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Subcutaneous", "value": 3.8585261472800005}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Subcutaneous", "value": 4.12507072696}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Subcutaneous", "value": 2.3095159982799998}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Subcutaneous", "value": 1.6392356397}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Subcutaneous", "value": -0.18342079576900003}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Subcutaneous", "value": -2.3385388767400004}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Subcutaneous", "value": 1.03626987034}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Subcutaneous", "value": -1.7729668319499998}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Subcutaneous", "value": 2.20665434028}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Adipose_Subcutaneous", "value": 4.31329928423}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Adipose_Subcutaneous", "value": 4.60764699873}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Adipose_Subcutaneous", "value": 4.5854791614999995}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Adipose_Subcutaneous", "value": 2.62887433518}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adipose_Subcutaneous", "value": 3.4689031317900003}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adipose_Subcutaneous", "value": 3.4770626623300003}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adipose_Subcutaneous", "value": 3.32480292236}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adipose_Subcutaneous", "value": 3.2619775575800003}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adipose_Subcutaneous", "value": 2.63694311496}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adipose_Subcutaneous", "value": 2.23485554073}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adipose_Subcutaneous", "value": 2.4606665653}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adipose_Subcutaneous", "value": 2.02624789184}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Adipose_Subcutaneous", "value": 2.05411453507}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Adipose_Subcutaneous", "value": -1.1018968139}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Adipose_Subcutaneous", "value": -1.04766432891}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Adipose_Visceral_Omentum", "value": -2.40451417018}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Adipose_Visceral_Omentum", "value": 3.59564790061}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Visceral_Omentum", "value": 1.45799957435}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Visceral_Omentum", "value": 4.387207323409999}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Visceral_Omentum", "value": 3.6240566345699996}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Visceral_Omentum", "value": 4.3068365042000005}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Visceral_Omentum", "value": 3.4649644724699997}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Visceral_Omentum", "value": 2.9479146579299997}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Visceral_Omentum", "value": -0.590541428566}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Visceral_Omentum", "value": -1.34349636931}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Visceral_Omentum", "value": 1.2421738722}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Visceral_Omentum", "value": -2.87794873796}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.387439914855}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Adipose_Visceral_Omentum", "value": 4.70843673906}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Adipose_Visceral_Omentum", "value": 4.97900846259}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Adipose_Visceral_Omentum", "value": 3.8163119601199997}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Adipose_Visceral_Omentum", "value": 3.76759111215}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adipose_Visceral_Omentum", "value": 3.9370861242400004}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adipose_Visceral_Omentum", "value": 4.03373926775}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adipose_Visceral_Omentum", "value": 3.8248525567300002}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adipose_Visceral_Omentum", "value": 3.92035698893}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adipose_Visceral_Omentum", "value": 2.48108027093}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adipose_Visceral_Omentum", "value": 2.1048453133400002}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adipose_Visceral_Omentum", "value": 2.4528107982}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adipose_Visceral_Omentum", "value": 2.08614742086}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Adipose_Visceral_Omentum", "value": 3.3430641012800004}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Adipose_Visceral_Omentum", "value": -1.83953495751}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Adipose_Visceral_Omentum", "value": -1.7468828197}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Breast_Mammary_Tissue", "value": -3.42579134965}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Breast_Mammary_Tissue", "value": 2.8837944057800002}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Breast_Mammary_Tissue", "value": 1.4103109946899999}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Breast_Mammary_Tissue", "value": 4.70182546551}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Breast_Mammary_Tissue", "value": 3.7987064480800004}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Breast_Mammary_Tissue", "value": 4.43392244198}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Breast_Mammary_Tissue", "value": 2.64524095126}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Breast_Mammary_Tissue", "value": 2.02189320211}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Breast_Mammary_Tissue", "value": -0.965376233809}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Breast_Mammary_Tissue", "value": -2.12380974963}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Breast_Mammary_Tissue", "value": 1.80092089978}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Breast_Mammary_Tissue", "value": -2.37199542342}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Breast_Mammary_Tissue", "value": 1.2594872215200001}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Breast_Mammary_Tissue", "value": 4.14467803172}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Breast_Mammary_Tissue", "value": 4.38202163843}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Breast_Mammary_Tissue", "value": 5.071803836030001}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Breast_Mammary_Tissue", "value": 2.86181224458}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Breast_Mammary_Tissue", "value": 2.79632794021}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Breast_Mammary_Tissue", "value": 3.03229373668}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Breast_Mammary_Tissue", "value": 2.7260506626099996}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Breast_Mammary_Tissue", "value": 2.93553174255}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Breast_Mammary_Tissue", "value": 1.93988142802}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Breast_Mammary_Tissue", "value": 1.7599681802900002}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Breast_Mammary_Tissue", "value": 1.91093415363}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Breast_Mammary_Tissue", "value": 1.72000388935}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Breast_Mammary_Tissue", "value": 2.57282084695}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Breast_Mammary_Tissue", "value": -2.2421390861000003}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Breast_Mammary_Tissue", "value": -2.0092918429299997}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Muscle_Skeletal", "value": -4.04491238346}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Muscle_Skeletal", "value": 3.5993903459}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Muscle_Skeletal", "value": 0.485415082262}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Muscle_Skeletal", "value": 3.42863860238}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Muscle_Skeletal", "value": 4.03582947812}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Muscle_Skeletal", "value": 4.76723107911}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Muscle_Skeletal", "value": 2.97579810681}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Muscle_Skeletal", "value": 2.23956256805}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Muscle_Skeletal", "value": -0.734602355633}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Muscle_Skeletal", "value": -2.50616857574}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Muscle_Skeletal", "value": 1.715340729}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Muscle_Skeletal", "value": -2.61332740798}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Muscle_Skeletal", "value": 1.35388867607}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Muscle_Skeletal", "value": 4.4108500696699995}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Muscle_Skeletal", "value": 4.6016662114599995}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Muscle_Skeletal", "value": 5.3497716124300005}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Muscle_Skeletal", "value": 3.6880309859}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Muscle_Skeletal", "value": 3.32917492262}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Muscle_Skeletal", "value": 3.45973291743}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Muscle_Skeletal", "value": 3.24550620921}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Muscle_Skeletal", "value": 3.35927481385}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Muscle_Skeletal", "value": 2.28824797543}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Muscle_Skeletal", "value": 2.12188059567}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Muscle_Skeletal", "value": 2.24867701839}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Muscle_Skeletal", "value": 2.06984648357}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Muscle_Skeletal", "value": 3.31499160057}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Muscle_Skeletal", "value": -2.04692341522}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Muscle_Skeletal", "value": -1.9263335045400003}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Cells_Transformed_fibroblasts", "value": 0.10365980091}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Cells_Transformed_fibroblasts", "value": 5.34801866052}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_Transformed_fibroblasts", "value": -2.67500768244}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_Transformed_fibroblasts", "value": -0.411434720616}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_Transformed_fibroblasts", "value": 1.53070743899}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_Transformed_fibroblasts", "value": 2.09995508064}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_Transformed_fibroblasts", "value": 1.81522498074}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_Transformed_fibroblasts", "value": 1.15262965654}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_Transformed_fibroblasts", "value": 1.77042922801}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_Transformed_fibroblasts", "value": -2.39696174014}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_Transformed_fibroblasts", "value": -1.1927335755600001}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_Transformed_fibroblasts", "value": -1.03014811409}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_Transformed_fibroblasts", "value": -2.4026827227500003}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Cells_Transformed_fibroblasts", "value": 3.89540694064}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Cells_Transformed_fibroblasts", "value": 4.114219481619999}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Cells_Transformed_fibroblasts", "value": 1.34999742822}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Cells_Transformed_fibroblasts", "value": 5.48629614064}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Cells_Transformed_fibroblasts", "value": 5.07398588989}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Cells_Transformed_fibroblasts", "value": 5.044353860119999}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Cells_Transformed_fibroblasts", "value": 4.8894706266400005}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Cells_Transformed_fibroblasts", "value": 4.84705562015}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Cells_Transformed_fibroblasts", "value": 4.02065371614}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Cells_Transformed_fibroblasts", "value": 4.31238182193}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Cells_Transformed_fibroblasts", "value": 3.9556621872199993}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Cells_Transformed_fibroblasts", "value": 4.22027511219}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Cells_Transformed_fibroblasts", "value": 3.72972190624}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Cells_Transformed_fibroblasts", "value": -1.57790531843}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Cells_Transformed_fibroblasts", "value": -1.3815439221}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Cells_EBV_transformed_lymphocytes", "value": -0.0286515803911}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Cells_EBV_transformed_lymphocytes", "value": 2.9664219604400004}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": -2.2446213391900005}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": -0.8017568154379999}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 2.7332285380199997}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 2.2369745713499998}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 2.6383558779599996}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 2.2230660528799997}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 1.32327100301}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": -0.6632018272779999}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": -1.20622676452}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": -1.96331038002}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": -1.9057501871200002}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Cells_EBV_transformed_lymphocytes", "value": 3.45774379495}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Cells_EBV_transformed_lymphocytes", "value": 3.9338943057699995}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.474116912485}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Cells_EBV_transformed_lymphocytes", "value": 3.28883534432}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Cells_EBV_transformed_lymphocytes", "value": 5.45755309995}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Cells_EBV_transformed_lymphocytes", "value": 5.46589026085}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Cells_EBV_transformed_lymphocytes", "value": 5.1770084363199995}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Cells_EBV_transformed_lymphocytes", "value": 5.01922716375}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Cells_EBV_transformed_lymphocytes", "value": 5.01779149244}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Cells_EBV_transformed_lymphocytes", "value": 5.5418287799099994}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Cells_EBV_transformed_lymphocytes", "value": 5.000333811630001}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Cells_EBV_transformed_lymphocytes", "value": 5.4558035379}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Cells_EBV_transformed_lymphocytes", "value": 2.50829511972}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Cells_EBV_transformed_lymphocytes", "value": -0.337146572834}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Cells_EBV_transformed_lymphocytes", "value": -0.335573494274}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -1.7612190544400002}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 4.71689142177}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -0.391838030309}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 3.88281782379}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 4.18943443489}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 4.243219810319999}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 3.23018518909}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 2.66357896909}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.028474031703400002}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -2.12437089683}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.723226596965}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -2.4670735493400002}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -1.0867398332}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 3.5832988004300006}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 3.7483340045099998}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 3.97990605601}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 3.4161125390700002}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 3.4868827027}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 2.7728192281800004}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 3.34688370382}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 2.59141620116}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 2.07646273947}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 1.6757574098399999}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 1.87733660732}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 1.44472514169}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -0.110692366892}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -3.71919283465}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -3.58278949949}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.6517233403900002}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.39581725551}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.4834720540600002}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.86944660241}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.4862533717}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 4.06957479203}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.6195679838099997}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.85777286575}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -0.0564152074622}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.8527085441900004}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.0516747664200001}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -1.94325472151}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.05893985041}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 4.25805645313}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 4.4585141381}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 4.45573717417}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.8797038890599995}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.37530450658}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.3954540428400004}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.2651190135299997}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.28077640611}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.39557485428}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.04969149985}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.31889130289}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.97023258161}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.29051164307}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.90496371064}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.6659073633}], "data-118f57edfa0a8a9b44ef432d1909fe1b": [{"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Adrenal_Gland", "value": -2.18225369433}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Adrenal_Gland", "value": 3.4918055564099997}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adrenal_Gland", "value": 0.838548037947}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adrenal_Gland", "value": 5.05707748179}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adrenal_Gland", "value": 3.9830808891000005}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adrenal_Gland", "value": 4.42998711124}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adrenal_Gland", "value": 2.79095481482}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adrenal_Gland", "value": 2.00302787991}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adrenal_Gland", "value": -0.845126197196}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adrenal_Gland", "value": -3.19872162611}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adrenal_Gland", "value": 2.07239958209}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adrenal_Gland", "value": -2.2933749350599997}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Adrenal_Gland", "value": -0.156113427404}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Adrenal_Gland", "value": 4.2093496436}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Adrenal_Gland", "value": 4.41475681709}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Adrenal_Gland", "value": 4.21763306604}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Adrenal_Gland", "value": 3.40998894819}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adrenal_Gland", "value": 3.12083711027}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adrenal_Gland", "value": 3.31838085694}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adrenal_Gland", "value": 3.04359119744}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adrenal_Gland", "value": 3.2296588756299998}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adrenal_Gland", "value": 1.77354316349}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adrenal_Gland", "value": 1.85929516715}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adrenal_Gland", "value": 1.74470226363}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Adrenal_Gland", "value": 1.8150038945200002}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Adrenal_Gland", "value": 2.3857923795500002}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Adrenal_Gland", "value": -4.2114156627199995}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Adrenal_Gland", "value": -4.0345088233399995}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Pituitary", "value": -3.3452905901}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Pituitary", "value": 3.6857995447000005}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pituitary", "value": -0.289271722778}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pituitary", "value": 3.01927819643}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pituitary", "value": 3.53613569633}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pituitary", "value": 4.1894552581}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pituitary", "value": 2.4545515754}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pituitary", "value": 1.68834317283}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pituitary", "value": -0.29091361784200004}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pituitary", "value": -3.03687584403}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pituitary", "value": 1.3863944052300001}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pituitary", "value": -1.9857968889099997}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pituitary", "value": 0.705744000141}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Pituitary", "value": 4.2237561981}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Pituitary", "value": 4.42173972608}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Pituitary", "value": 5.02227593615}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Pituitary", "value": 3.36883113205}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Pituitary", "value": 3.04859875481}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Pituitary", "value": 3.2300574285000003}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Pituitary", "value": 2.9586158366400004}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Pituitary", "value": 3.10996908762}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Pituitary", "value": 1.9910497845900001}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Pituitary", "value": 1.8488965848199999}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Pituitary", "value": 1.9117609191299998}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Pituitary", "value": 1.7503682774200002}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Pituitary", "value": 2.0913520191}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Pituitary", "value": -2.58228840152}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Pituitary", "value": -2.4947464505}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Pancreas", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Thyroid", "value": -4.16186542299}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Thyroid", "value": 2.35888915587}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Thyroid", "value": 2.01000250116}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Thyroid", "value": 5.93065408365}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Thyroid", "value": 3.90856282373}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Thyroid", "value": 4.70901924459}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Thyroid", "value": 2.69380780035}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Thyroid", "value": 2.0706362667700002}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Thyroid", "value": -1.67610674599}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Thyroid", "value": -2.22849692663}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Thyroid", "value": 2.68494783546}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Thyroid", "value": -2.5637093974900003}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Thyroid", "value": 1.9326577948599997}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Thyroid", "value": 4.05696050084}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Thyroid", "value": 4.28427105621}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Thyroid", "value": 5.8644178391}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Thyroid", "value": 2.4282981817}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Thyroid", "value": 2.1778360801299996}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Thyroid", "value": 2.45811158974}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Thyroid", "value": 2.12310276337}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Thyroid", "value": 2.39080734641}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Thyroid", "value": 1.25748005495}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Thyroid", "value": 1.0474508333299999}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Thyroid", "value": 1.23665659452}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Thyroid", "value": 1.02132877494}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Thyroid", "value": 2.6796698082}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Thyroid", "value": -2.2443436653599997}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Thyroid", "value": -2.01100606599}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Lung", "value": -2.2742609466599997}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Lung", "value": 3.8867692224800003}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Lung", "value": 0.78502155227}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Lung", "value": 4.63876432704}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Lung", "value": 3.5791266533599995}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Lung", "value": 4.441240179659999}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Lung", "value": 2.7201533758700003}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Lung", "value": 1.83779064311}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Lung", "value": -0.164795720185}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Lung", "value": -3.9221279365199995}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Lung", "value": 1.5167809434200001}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Lung", "value": -2.00416010064}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Lung", "value": -0.314506793813}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Lung", "value": 4.0967610185800005}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Lung", "value": 4.31170776814}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Lung", "value": 4.564931844369999}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Lung", "value": 3.47727503674}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Lung", "value": 3.2743387534099995}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Lung", "value": 3.46453016394}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Lung", "value": 3.18766730738}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Lung", "value": 3.37030246702}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Lung", "value": 2.07304708947}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Lung", "value": 1.83052729917}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Lung", "value": 2.0317782032599996}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Lung", "value": 1.7819688414300001}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Lung", "value": 1.63435183449}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Lung", "value": -3.49931723556}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Lung", "value": -3.2734078895599996}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Ovary", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Prostate", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Testis", "value": -0.511551155073}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Testis", "value": 3.87278531416}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Testis", "value": -2.18763540465}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Testis", "value": 1.3873066002}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Testis", "value": 2.37871599124}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Testis", "value": 3.96380925573}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Testis", "value": 2.69251977755}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Testis", "value": 1.6716427469100001}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Testis", "value": 0.518469810971}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Testis", "value": -3.86011183136}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Testis", "value": 0.80871870453}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Testis", "value": -1.84630765199}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Testis", "value": -1.60862386931}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Testis", "value": 3.98543010685}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Testis", "value": 4.262496982769999}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Testis", "value": 3.10681264112}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Testis", "value": 3.8252358609199995}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Testis", "value": 3.6692403846399997}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Testis", "value": 3.7473889508199996}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Testis", "value": 3.55960254256}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Testis", "value": 3.6167060455400004}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Testis", "value": 2.47866804491}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Testis", "value": 2.72141341026}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Testis", "value": 2.44319997727}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Testis", "value": 2.65440228721}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Testis", "value": 2.1221279047099997}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Testis", "value": -2.65235636873}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Testis", "value": -2.64474903344}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Uterus", "value": 0.0}, {"LDSC Total Observed Scale h2": "0.0723 (0.0167)", "PMID or link": "26482879", "association": 9.027075, "phenotype_reference": "Atopic dermatitis (ATOP01) ", "LDSC Intercept": "1.0234 (0.007)", "LDSC Mean Chi-Square": 1.0799, "gene": "MRPS21", "index": 0, "category": "j: other", "LDSC Lambda GC": 1.0527, "LDSC Ratio": "0.2934 (0.0873)", "ENSEMBL": "ENSG00000187145", "study size": 116863, "phenotype": "Atopic dermatitis", "code": "ATOP01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26482879", "variable": "Vagina", "value": -0.577621369133}, {"LDSC Total Observed Scale h2": "0.2143 (0.0066)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 8.075039, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Mean Chi-Square": 2.5273, "gene": "MRPS21", "index": 1, "category": "b: anthropometric", "LDSC Lambda GC": 2.0217, "LDSC Ratio": "0.0389 (0.0109)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Body fat percentage complete", "code": "BFPC04", "phenotype_link": "nopmid.html", "variable": "Vagina", "value": 2.20152888192}, {"LDSC Total Observed Scale h2": "0.2268 (0.0185)", "PMID or link": "27863252", "association": 8.092578, "phenotype_reference": "Platelet count (BLOO01) ", "LDSC Intercept": "0.9967 (0.0271)", "LDSC Mean Chi-Square": 1.768, "gene": "MRPS21", "index": 2, "category": "g: blood", "LDSC Lambda GC": 1.2531, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "platelet count", "code": "BLOO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Vagina", "value": 1.40490211602}, {"LDSC Total Observed Scale h2": "0.1863 (0.0217)", "PMID or link": "27863252", "association": 13.866557, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Mean Chi-Square": 1.6776, "gene": "MRPS21", "index": 3, "category": "g: blood", "LDSC Lambda GC": 1.2201, "LDSC Ratio": "0.0686 (0.0499)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "monocyte count", "code": "BLOO05", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Vagina", "value": 9.64353257898}, {"LDSC Total Observed Scale h2": "0.1808 (0.0221)", "PMID or link": "27863252", "association": 4.4924, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Mean Chi-Square": 1.6196, "gene": "MRPS21", "index": 4, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "eosinophil count", "code": "BLOO07", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Vagina", "value": 6.078415177819999}, {"LDSC Total Observed Scale h2": "0.1712 (0.0201)", "PMID or link": "27863252", "association": 6.593477, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Mean Chi-Square": 1.5912, "gene": "MRPS21", "index": 5, "category": "g: blood", "LDSC Lambda GC": 1.1908, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Sum eosinophil basophil counts", "code": "BLOO16", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Vagina", "value": 5.95555476112}, {"LDSC Total Observed Scale h2": "0.1752 (0.02)", "PMID or link": "27863252", "association": 3.06077, "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Intercept": "0.9613 (0.0261)", "LDSC Mean Chi-Square": 1.5997, "gene": "MRPS21", "index": 6, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of white cells", "code": "BLOO17", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Vagina", "value": 5.15234778856}, {"LDSC Total Observed Scale h2": "0.1696 (0.019)", "PMID or link": "27863252", "association": 2.777726, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Mean Chi-Square": 1.5779, "gene": "MRPS21", "index": 7, "category": "g: blood", "LDSC Lambda GC": 1.1876, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Eosinophil percentage of granulocytes", "code": "BLOO18", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Vagina", "value": 4.719295674890001}, {"LDSC Total Observed Scale h2": "0.1561 (0.018)", "PMID or link": "27863252", "association": 8.74213, "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "LDSC Intercept": "1.0168 (0.03)", "LDSC Mean Chi-Square": 1.5446, "gene": "MRPS21", "index": 8, "category": "g: blood", "LDSC Lambda GC": 1.1683, "LDSC Ratio": "0.0309 (0.0551)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Granulocyte percentage of myeloid white cells", "code": "BLOO19", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Vagina", "value": -5.32285655891}, {"LDSC Total Observed Scale h2": "0.1474 (0.0115)", "PMID or link": "27863252", "association": 2.716902, "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Intercept": "0.9664 (0.0153)", "LDSC Mean Chi-Square": 1.4799, "gene": "MRPS21", "index": 9, "category": "g: blood", "LDSC Lambda GC": 1.2038, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Lymphocyte percentage of white cells", "code": "BLOO24", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Vagina", "value": -1.61560181734}, {"LDSC Total Observed Scale h2": "0.1686 (0.0207)", "PMID or link": "27863252", "association": 12.926832000000001, "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "LDSC Intercept": "1.0362 (0.035)", "LDSC Mean Chi-Square": 1.6022, "gene": "MRPS21", "index": 10, "category": "g: blood", "LDSC Lambda GC": 1.1715, "LDSC Ratio": "0.0601 (0.0581)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Monocyte percentage of white cells", "code": "BLOO28", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Vagina", "value": 6.5364204377199995}, {"LDSC Total Observed Scale h2": "0.1586 (0.0172)", "PMID or link": "27863252", "association": 3.453507, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Mean Chi-Square": 1.5319, "gene": "MRPS21", "index": 11, "category": "g: blood", "LDSC Lambda GC": 1.1843, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Neutrophil percentage of granulocytes", "code": "BLOO31", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Vagina", "value": -4.85852190449}, {"LDSC Total Observed Scale h2": "0.2019 (0.0171)", "PMID or link": "27863252", "association": 9.157534, "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Intercept": "1.0176 (0.0246)", "LDSC Mean Chi-Square": 1.6946, "gene": "MRPS21", "index": 12, "category": "g: blood", "LDSC Lambda GC": 1.2498, "LDSC Ratio": "0.0253 (0.0354)", "ENSEMBL": "ENSG00000187145", "study size": 173480, "phenotype": "Plateletcrit", "code": "BLOO33", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "variable": "Vagina", "value": -1.16866551209}, {"LDSC Total Observed Scale h2": "0.1433 (0.0079)", "PMID or link": "25673413", "association": 3.811156, "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Intercept": "0.6766 (0.01)", "LDSC Mean Chi-Square": 1.321, "gene": "MRPS21", "index": 13, "category": "b: anthropometric", "LDSC Lambda GC": 1.1301, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 339224, "phenotype": "BMI (Mixed ancestry Locke)", "code": "BODY03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Vagina", "value": 3.4177607353400004}, {"LDSC Total Observed Scale h2": "0.1374 (0.0077)", "PMID or link": "25673413", "association": 4.027117, "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Intercept": "0.6475 (0.0096)", "LDSC Mean Chi-Square": 1.2586, "gene": "MRPS21", "index": 14, "category": "b: anthropometric", "LDSC Lambda GC": 1.0741, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 322154, "phenotype": "BMI (European Locke)", "code": "BODY04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "variable": "Vagina", "value": 3.6911945890300006}, {"LDSC Total Observed Scale h2": "0.1015 (0.0059)", "PMID or link": "27494321", "association": 12.158196, "phenotype_reference": "Chronotype (CHRO01) ", "LDSC Intercept": "1.0138 (0.0076)", "LDSC Mean Chi-Square": 1.2651, "gene": "MRPS21", "index": 15, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.2005, "LDSC Ratio": "0.0519 (0.0285)", "ENSEMBL": "ENSG00000187145", "study size": 128266, "phenotype": "Chronotype", "code": "CHRO01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27494321", "variable": "Vagina", "value": 2.6461583806}, {"LDSC Total Observed Scale h2": "0.2218 (0.0073)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 4.976501, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Mean Chi-Square": 2.5512, "gene": "MRPS21", "index": 16, "category": "b: anthropometric", "LDSC Lambda GC": 2.0175, "LDSC Ratio": "0.0277 (0.0105)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat mass", "code": "FATM04", "phenotype_link": "nopmid.html", "variable": "Vagina", "value": 2.52370712127}, {"LDSC Total Observed Scale h2": "0.1742 (0.0104)", "PMID or link": "25673412", "association": 3.445559, "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Intercept": "0.817 (0.0077)", "LDSC Mean Chi-Square": 1.105, "gene": "MRPS21", "index": 17, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88019, "phenotype": "Hip circumference European female", "code": "HIPC01F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Vagina", "value": 2.9521101561000003}, {"LDSC Total Observed Scale h2": "0.1493 (0.0086)", "PMID or link": "25673412", "association": 3.460924, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Mean Chi-Square": 1.1677, "gene": "MRPS21", "index": 18, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145308, "phenotype": "Hip circumference European", "code": "HIPC01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Vagina", "value": 1.9314157737}, {"LDSC Total Observed Scale h2": "0.1759 (0.0106)", "PMID or link": "25673412", "association": 3.55544, "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Intercept": "0.8148 (0.0077)", "LDSC Mean Chi-Square": 1.1055, "gene": "MRPS21", "index": 19, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 88035, "phenotype": "Hip circumference All Ancestries female", "code": "HIPC02F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Vagina", "value": 2.89073900606}, {"LDSC Total Observed Scale h2": "0.1491 (0.0085)", "PMID or link": "25673412", "association": 3.675121, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "MRPS21", "index": 20, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 145414, "phenotype": "Hip circumference All Ancestries", "code": "HIPC02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Vagina", "value": 1.72384672887}, {"LDSC Total Observed Scale h2": "0.1549 (0.0119)", "PMID or link": "25673412", "association": 2.4355, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Mean Chi-Square": 1.1134, "gene": "MRPS21", "index": 21, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86792, "phenotype": "Hip circumference European female BMI-adjusted", "code": "HIPC03F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Vagina", "value": 0.776280577175}, {"LDSC Total Observed Scale h2": "0.1459 (0.009)", "PMID or link": "25673412", "association": 3.2362130000000002, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Mean Chi-Square": 1.2106, "gene": "MRPS21", "index": 22, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143579, "phenotype": "Hip circumference European BMI-adjusted", "code": "HIPC03", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Vagina", "value": 0.263471011808}, {"LDSC Total Observed Scale h2": "0.1546 (0.0116)", "PMID or link": "25673412", "association": 2.534736, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Mean Chi-Square": 1.1104, "gene": "MRPS21", "index": 23, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 86647, "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "code": "HIPC04F", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Vagina", "value": 0.8678415258609999}, {"LDSC Total Observed Scale h2": "0.1457 (0.0089)", "PMID or link": "25673412", "association": 2.946883, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Mean Chi-Square": 1.2088, "gene": "MRPS21", "index": 24, "category": "b: anthropometric", "LDSC Lambda GC": 1.0466, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "ENSEMBL": "ENSG00000187145", "study size": 143575, "phenotype": "Hip circumference All Ancestries BMI-adjusted", "code": "HIPC04", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "variable": "Vagina", "value": 0.191304835936}, {"LDSC Total Observed Scale h2": "0.2904 (0.0116)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "association": 5.65606, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Mean Chi-Square": 3.1445, "gene": "MRPS21", "index": 25, "category": "b: anthropometric", "LDSC Lambda GC": 2.1851, "LDSC Ratio": "0.0513 (0.0128)", "ENSEMBL": "ENSG00000187145", "study size": 353972, "phenotype": "Fat free mass / lean mass", "code": "LEAN06", "phenotype_link": "nopmid.html", "variable": "Vagina", "value": 2.4837744426499997}, {"LDSC Total Observed Scale h2": "0.4467 (0.0157)", "PMID or link": "25056061", "association": 6.782911, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Mean Chi-Square": 1.755, "gene": "MRPS21", "index": 26, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5696, "LDSC Ratio": "0.0437 (0.0138)", "ENSEMBL": "ENSG00000187145", "study size": 82315, "phenotype": "Schizophrenia PGC 2014", "code": "SCHI01", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Vagina", "value": -3.41600471339}, {"LDSC Total Observed Scale h2": "0.465 (0.0168)", "PMID or link": "25056061", "association": 6.266979, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "MRPS21", "index": 27, "category": "c: mental and behavioural", "LDSC Lambda GC": 1.5622, "LDSC Ratio": "0.0515 (0.0139)", "ENSEMBL": "ENSG00000187145", "study size": 77096, "phenotype": "Schizophrenia PGC 2014 European", "code": "SCHI02", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "variable": "Vagina", "value": -3.4646003375800003}]}};
    var embed_opt = {"mode": "vega-lite"};

    function showError(el, error){
        el.innerHTML = ('<div class="error">'
                        + '<p>JavaScript Error: ' + error.message + '</p>'
                        + "<p>This usually means there's a typo in your chart specification. "
                        + "See the javascript console for the full traceback.</p>"
                        + '</div>');
        throw error;
    }
    const el = document.getElementById('vis');
    vegaEmbed("#vis", spec, embed_opt)
      .catch(error => showError(el, error));
  </script>
</body>
</html>
