
<!DOCTYPE html>

<html>
<head><style>.vega-actions a {margin-right: 12px;color: #757575;font-weight: normal;font-size: 13px;}.error {color: red;}</style><script src="https://cdn.jsdelivr.net/npm//vega@4" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-lite@2.6.0" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-embed@3" type="text/javascript"></script><meta content="text/html; charset=utf-8" http-equiv="Content-Type"/><meta content="IE=edge" http-equiv="X-UA-Compatible"/><meta content="width=device-width, initial-scale=1" name="viewport"/><link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/Font-Awesome-4.7/css/font-awesome.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap-theme.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/images/gnomic_icon.png" rel="icon" type="image/png"/><script src="https://phenviz.navigome.com/js/jquery.min.js"></script><script src="https://phenviz.navigome.com/js/bootstrap.js"></script><title>Gene profile: ENSG00000204305</title></head>
<body>
<h4>Gene profile across phenotypes - ENSG00000204305 (AGER)</h4><div id="vis"></div><div class="col-lg-12 col-md-12 content-item content-item-1"><p>This is a visualization of GWAS-derived gene associations  across Navigome phenotypes. Only phenotypes for which the gene has at least one significant association (p-value â‰¤ 0.05/20000) in <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> <b>and/or</b> <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a> analyses are shown. The first panel is a visualization of gene-phenotype associations obtained using <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> (-log10(p-value)). Hover over any bar to highlight the phenotype in the other panels. The other panels are parallel coordinates visualizations for tissue-wise associations (z-scores)computed using <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a>  and eQTL expression data from GTEx and DGN. Significance thresholds are delineated with black lines. The parallel coordinates show which phenotypes have a significant association and in which tissues, based on gene expression data. Clicking on a specific phenotype will lead to the reference of the GWAS study.</p></div>
<script type="text/javascript">
    var spec = {"config": {"view": {"width": 400, "height": 300}}, "vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "bar", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "href": {"type": "nominal", "field": "phenotype_link"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "quantitative", "axis": {"title": "MAGMA -log10(p)"}, "field": "value"}, "y": {"type": "nominal", "axis": {"title": ""}, "field": "phenotype_reference"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-4e93ee1a91df87c61b7d73e211f0f4d1"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "x": {"type": "quantitative", "field": "ThresholdVal"}}}], "title": "MAGMA gene-wise association"}], "center": true, "data": {"name": "data-3d3e4ee6f677d1279a6cf6ee949b036f"}}, {"vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Nervous system", "width": 500}], "center": true, "data": {"name": "data-56b73d974f66597076e101d335695bb3"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Digestive System", "width": 500}], "center": true, "data": {"name": "data-c02a5e5d16d31565c61976b760d88509"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Blood, Arteries and Heart", "width": 500}], "center": true, "data": {"name": "data-6aca3c3f501254c2a07307a754dcb3d9"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Adipose Tissue, Muscle, Skin and Cells", "width": 500}], "center": true, "data": {"name": "data-be276ec5314085486ce5fa431438fef2"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-29.9345490252, 29.9345490252]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Endocrine System and Others", "width": 500}], "center": true, "data": {"name": "data-da4b445591b9c0699a70f658a69716ba"}}]}], "spacing": 40, "$schema": "https://vega.github.io/schema/vega-lite/v2.6.0.json", "datasets": {"data-3d3e4ee6f677d1279a6cf6ee949b036f": [{"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 19.572839000000002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 17.193386}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 18.078314000000002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 32.011463}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 37.446906}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 14.54171}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 26.284874}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 7.174113}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 18.102912}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 20.772062}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 21.737454}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 32.275814000000004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 30.468662}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 37.344852}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 14.605566}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 16.594893}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 12.84333}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 20.847865}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 17.75965}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 26.629894}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 9.405088000000001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 9.968955000000001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 11.247183999999999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 11.073375}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 10.552067}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 27.292446999999996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 42.731445}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 14.09174}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 8.044601}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 12.808912}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 16.988218}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 13.351913}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "magma_log10p", "value": 34.282246}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "magma_log10p", "value": 11.503943}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "magma_log10p", "value": 7.043577}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "magma_log10p", "value": 11.510956}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "magma_log10p", "value": 3.9654519999999995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "magma_log10p", "value": 1.945157}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "magma_log10p", "value": 15.005072}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "magma_log10p", "value": 5.703357}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "magma_log10p", "value": 14.00567}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "magma_log10p", "value": 13.778585999999999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "magma_log10p", "value": 42.950897999999995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "magma_log10p", "value": 5.166923000000001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "magma_log10p", "value": 7.984221000000001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "magma_log10p", "value": 67.22332800000001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "magma_log10p", "value": 93.01243199999999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "magma_log10p", "value": 88.339789}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "magma_log10p", "value": 1.4676790000000002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "magma_log10p", "value": 1.1834209999999998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "magma_log10p", "value": 24.576836}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "magma_log10p", "value": 6.070106}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "magma_log10p", "value": 16.287014000000003}], "data-4e93ee1a91df87c61b7d73e211f0f4d1": [{"Threshold": "MAGMA -log10(p) \u2265 5.60206", "ThresholdVal": 5.60206}], "data-56b73d974f66597076e101d335695bb3": [{"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Brain_Amygdala", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Brain_Caudate", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Brain_Putamen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Brain_Cerebellum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Brain_Cortex", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Brain_Hippocampus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Brain_Hypothalamus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": -3.2021625042000004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": -2.03550577164}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": -2.10010051727}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": -0.9819996953010001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": -6.43769693375}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 1.53006756306}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 1.37091970444}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 1.8012086153}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 4.226266860959999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 3.09820008278}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": -0.41888526081999994}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": -1.7212800979599998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 5.36566829681}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": -5.24216890335}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": -3.4055538177499995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 0.488493978977}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": -4.74249696732}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 2.7940649986299997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": -3.9871916770899998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": -0.835031449795}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Brain_Substantia_nigra", "value": 1.9505550861400003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Brain_Substantia_nigra", "value": 0.848083674908}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Brain_Substantia_nigra", "value": -1.36222577095}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Brain_Substantia_nigra", "value": -1.47264516354}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Brain_Substantia_nigra", "value": 2.1444106102}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Brain_Substantia_nigra", "value": -13.975487709000001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Brain_Substantia_nigra", "value": -13.975487709000001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Brain_Substantia_nigra", "value": -0.6727361679080001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -8.1292491636}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -3.42582136159}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -0.854366203363}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -1.2143886026100001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -1.44537244907}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -1.86777707717}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -2.12522134356}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": 0.14782510996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -2.1839575721099997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -2.16050289911}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": 1.6313879132100002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -1.67090996348}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -2.03722964842}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -2.56103387662}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -0.584561302098}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -1.1573309397799998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": 5.270088940959999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -3.38625741283}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -2.2115326418200003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -3.13894194788}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -1.8735320799200001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -0.0654346903722}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": 4.20207866661}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": 4.87186927309}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -5.07230263063}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": 9.9555264421}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -2.1097151773599996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": 1.28608182206}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": 3.10799835163}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -6.540343335219999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": 3.2384348991400005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -0.583956288451}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Nerve_Tibial", "value": -3.0004429715099996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Nerve_Tibial", "value": 2.72683595852}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Nerve_Tibial", "value": -0.359002497857}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Nerve_Tibial", "value": -1.67691198459}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Nerve_Tibial", "value": 0.854835603214}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Nerve_Tibial", "value": -2.8157078291000004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Nerve_Tibial", "value": 2.99898655532}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Nerve_Tibial", "value": 3.40053529501}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Nerve_Tibial", "value": 0.14815261852199998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Nerve_Tibial", "value": -8.28186017515}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Nerve_Tibial", "value": 4.65382501389}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Nerve_Tibial", "value": -4.25417997986}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Nerve_Tibial", "value": -1.64522665238}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Nerve_Tibial", "value": 10.5614268349}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Nerve_Tibial", "value": -10.172875617799999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Nerve_Tibial", "value": -10.172875617799999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Nerve_Tibial", "value": -0.10256542154300001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Nerve_Tibial", "value": 0.0203200567164}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Nerve_Tibial", "value": -0.46610330163900004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Nerve_Tibial", "value": -0.960578607344}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Nerve_Tibial", "value": -12.4660974389}], "data-c4c9a654d45741ebc60aa567761bff19": [{"Text": "Significance Threshold", "Threshold": "PrediXcan zscore \u2265 4.70813", "ThresholdVal": 4.70813}], "data-4a3aa5bf8c5a41879455008930c25774": [{"Threshold": "PrediXcan zscore \u2264 -4.70813", "ThresholdVal": -4.70813}], "data-c02a5e5d16d31565c61976b760d88509": [{"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 2.0529027667400004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 1.49806565651}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 1.42343150625}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 0.606425854458}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 5.35845806099}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": -3.1170639558}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 0.252820228396}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": -0.316645779579}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": -2.29705667759}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": -0.31772279999499997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 1.0301574913}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 1.2890356068200002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": -5.16661406622}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 3.5729666482}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 2.2333509723400002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": -0.445270372516}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": 3.01828623126}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": -3.6823417334}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": -1.2252864592100001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": -0.402532970247}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Colon_Sigmoid", "value": -0.40226766953800003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Colon_Sigmoid", "value": 2.89494255156}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Colon_Sigmoid", "value": 1.3978768361200002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Colon_Sigmoid", "value": 0.8564615661570001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Colon_Sigmoid", "value": 0.796597050999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Colon_Sigmoid", "value": -1.85965723333}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Colon_Sigmoid", "value": 3.08009383212}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Colon_Sigmoid", "value": 2.41256541858}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Colon_Sigmoid", "value": -0.423768062776}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Colon_Sigmoid", "value": 1.5361949459600002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Colon_Sigmoid", "value": 1.88196908426}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Colon_Sigmoid", "value": -1.39371886166}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Colon_Sigmoid", "value": -1.29222513376}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Colon_Sigmoid", "value": 7.78655339386}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Colon_Sigmoid", "value": 4.66926119941}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Colon_Sigmoid", "value": 4.66926119941}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Colon_Sigmoid", "value": 1.66681268799}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Colon_Sigmoid", "value": 1.8113536277900002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Colon_Sigmoid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Colon_Sigmoid", "value": -0.206364383372}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Colon_Sigmoid", "value": 1.8929994802599999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Colon_Transverse", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 3.3370931952599996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.582487327348}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.00871510906}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.39830905129}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -1.3494052845500002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -7.63032431697}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -3.4962624785600003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.119181899164}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -7.6477276952899995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -7.4407545616399995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -4.5611085762300005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -8.073729928610001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -6.20248576358}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -3.5986291885900004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.445750751062}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.46956200384400004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 3.5232276839499996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -1.34059130537}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.637146987019}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.906315650813}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.0543973022}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.33064703366}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.48994190928}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.711835333563}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -4.5415334202099995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.5744430341499998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -6.12790476907}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.410725550725}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.352485783303}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -7.41859108406}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.457021647871}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.338010110885}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Esophagus_Gastroesophageal_Junction", "value": -1.56292875304}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.65140005255}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.811930288645}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.116258863556}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Esophagus_Gastroesophageal_Junction", "value": 5.21641954003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.3530976316499999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.8753737037}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.36221048764}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Esophagus_Gastroesophageal_Junction", "value": -1.3684832433200003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Esophagus_Gastroesophageal_Junction", "value": 3.77490147379}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 3.83487460491}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.210285797684}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.663793314356}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Esophagus_Gastroesophageal_Junction", "value": 3.94235326688}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Esophagus_Gastroesophageal_Junction", "value": -24.6326093632}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Esophagus_Gastroesophageal_Junction", "value": -24.022384841599997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.591924497525}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.987776628598}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Esophagus_Gastroesophageal_Junction", "value": -4.10794210434}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.9943730268050001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.4704499862399998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 3.39329532923}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 1.5054996301199999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": -0.11682184046800001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 0.7332453093189999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 5.0901174775}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 1.37744576213}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 4.14136637365}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 2.49179391885}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 1.7013704259600002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 2.21107386488}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 4.59383621378}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 3.63371915751}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 3.2639903189}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 0.288593865205}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": -0.611692104048}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": -0.8710889411730001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 3.22721688639}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": -1.62237289856}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": -0.381372430862}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": -0.817174564445}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": -1.53611130818}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 2.1053999349900003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 3.18701476191}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 2.54807233994}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": -2.9775268209099996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 4.44213586387}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 3.5810989761500003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 1.36635753621}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 2.2731490497999998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": -3.5410119256800003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": 1.51314282146}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": -1.0681171514399999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Esophagus_Mucosa", "value": -0.12355508488}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Esophagus_Mucosa", "value": 1.7933199307900003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Esophagus_Mucosa", "value": -0.721905722146}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Esophagus_Mucosa", "value": 4.3505218367300005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Esophagus_Mucosa", "value": 1.06468005407}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Esophagus_Mucosa", "value": -1.33535715862}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Esophagus_Mucosa", "value": 0.800749998947}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Esophagus_Mucosa", "value": 2.78048686626}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Esophagus_Mucosa", "value": 0.4408646801100001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Esophagus_Mucosa", "value": -8.0209072642}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Esophagus_Mucosa", "value": -0.710247254974}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Esophagus_Mucosa", "value": -6.926168962110001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Esophagus_Mucosa", "value": -0.7899901228470001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Esophagus_Mucosa", "value": 1.63295951268}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Esophagus_Mucosa", "value": -8.89336212499}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Esophagus_Mucosa", "value": -8.89336212499}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Esophagus_Mucosa", "value": 1.05219845251}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Esophagus_Mucosa", "value": 1.22284029669}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Esophagus_Mucosa", "value": 0.541591405869}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Esophagus_Mucosa", "value": 0.22068664784400002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Esophagus_Mucosa", "value": -12.3591573498}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": -5.52956560809}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": -6.60094803276}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": -3.39294666839}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": -3.6663324498300005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 5.19826065302}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": -5.7332633804199995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": -3.44918117619}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": -4.39772619056}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": -1.27912529186}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": -1.0674535221700001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 1.24383259521}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 1.82078399374}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": -5.32989383072}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 3.62506996329}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": -5.767649489480001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 3.4866641286}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 3.1654335793700006}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": -6.09441882755}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": -0.881767424728}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": 2.0467980306999998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Esophagus_Muscularis", "value": -5.307395830380001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Esophagus_Muscularis", "value": 3.75154238241}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Esophagus_Muscularis", "value": 0.0798602642138}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Esophagus_Muscularis", "value": -2.6151915054700003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Esophagus_Muscularis", "value": 0.945093115619}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Esophagus_Muscularis", "value": -5.28588496669}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Esophagus_Muscularis", "value": 4.1432839564499995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Esophagus_Muscularis", "value": 4.80292870322}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Esophagus_Muscularis", "value": -0.358439314386}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Esophagus_Muscularis", "value": -5.41614506849}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Esophagus_Muscularis", "value": 5.06782590513}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Esophagus_Muscularis", "value": -2.56886976305}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Esophagus_Muscularis", "value": -1.87149961306}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Esophagus_Muscularis", "value": 18.2603041099}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Esophagus_Muscularis", "value": -6.44397936556}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Esophagus_Muscularis", "value": -6.44397936556}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Esophagus_Muscularis", "value": -1.3347787562}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Esophagus_Muscularis", "value": -1.2057119280700002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Esophagus_Muscularis", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Esophagus_Muscularis", "value": -0.766004246652}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Esophagus_Muscularis", "value": -8.31698402343}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Liver", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Stomach", "value": -2.1045119288599996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Stomach", "value": -6.412867745290001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Stomach", "value": -4.75331854542}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Stomach", "value": -5.10649593789}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Stomach", "value": 4.62050974034}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Stomach", "value": -4.96005586128}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Stomach", "value": -4.00471305774}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Stomach", "value": -4.75726709962}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Stomach", "value": -2.22446987756}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Stomach", "value": -1.1783119746600001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Stomach", "value": 0.0193674319723}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Stomach", "value": 0.556872565367}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Stomach", "value": -4.2694428612}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Stomach", "value": 3.9465779141}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Stomach", "value": -2.4734178667299997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Stomach", "value": 5.02460803917}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Stomach", "value": 4.10712131459}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Stomach", "value": -6.11714563602}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Stomach", "value": -1.11396681485}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Stomach", "value": 2.5566461166099996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Stomach", "value": -5.3184687555000005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Stomach", "value": 4.01299024149}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Stomach", "value": 1.85645724963}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Stomach", "value": 5.44887555108}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Stomach", "value": 3.3486452897600003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Stomach", "value": 4.7253000597}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Stomach", "value": -3.33242193677}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Stomach", "value": 1.3692913426700002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Stomach", "value": -1.78493279745}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Stomach", "value": 15.7058703651}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Stomach", "value": -15.5947805278}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Stomach", "value": -15.5947805278}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Stomach", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Stomach", "value": 0.541591405869}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Stomach", "value": 0.22801809545899998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Stomach", "value": 0.0}], "data-6aca3c3f501254c2a07307a754dcb3d9": [{"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": 1.0379357337951658}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -0.2931598126888275}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": 0.031463373452425}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -3.5081114768981934}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -4.45727014541626}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -8.103023529052733}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -2.939634084701538}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -3.362250328063965}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -8.215603828430176}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -8.280557632446289}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -13.296210289001465}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -12.006993293762207}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -7.712129179596324}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -6.5402090724389605}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -2.518812895487935}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -2.405687684174827}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": 3.9326721775180937}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -1.6013407984290955}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": 1.433549989706537}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": 0.6720517673216377}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -0.10844498932526596}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": 0.8279904422827026}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": 5.192821109434973}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": 5.686890553776909}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -4.606251053619891}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": 10.369431675020452}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -7.931147812898392}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": 2.196847468169628}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": 1.4494248624935142}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -9.062432394445993}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": 4.50211753892481}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": 0.8447755527602099}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "DGN_Whole_Blood", "value": -0.02345522717159516}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "DGN_Whole_Blood", "value": 0.4440293750699575}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "DGN_Whole_Blood", "value": -0.8463364464105788}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "DGN_Whole_Blood", "value": 0.3404883986811641}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "DGN_Whole_Blood", "value": 1.3624340946131834}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "DGN_Whole_Blood", "value": 0.4972517627844586}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "DGN_Whole_Blood", "value": 0.4049103257639928}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "DGN_Whole_Blood", "value": 2.205163827870691}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "DGN_Whole_Blood", "value": 0.4905517900555335}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "DGN_Whole_Blood", "value": -2.4038152095869716}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "DGN_Whole_Blood", "value": 4.4063999156299625}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "DGN_Whole_Blood", "value": 7.142453678254433}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "DGN_Whole_Blood", "value": -3.5865459921463434}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "DGN_Whole_Blood", "value": 15.021395399092178}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "DGN_Whole_Blood", "value": -10.45089968391267}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "DGN_Whole_Blood", "value": -10.45089968391267}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "DGN_Whole_Blood", "value": -5.5979021675187335}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "DGN_Whole_Blood", "value": -5.889263595756877}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "DGN_Whole_Blood", "value": -1.1294690370559692}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "DGN_Whole_Blood", "value": -0.7099389332170615}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "DGN_Whole_Blood", "value": -4.444613460465246}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -4.8724619242900005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -1.1024671104500001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": 1.26902591275}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -2.1457333864700003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -7.74555983876}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -8.48963803029}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -7.80129515528}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -5.9059821062}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -9.13095141538}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -9.51461743972}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -9.40491108845}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -11.5170185528}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -5.790324933330001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -7.251103990630001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -3.6895664056300004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -3.90534489507}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": 1.21976512333}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -5.86836585625}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -5.70625081763}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -6.58740416806}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -1.8488058524900002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -1.3738263492899998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": 0.399123276963}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": 0.809611299198}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -0.6103205788530001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": 4.29227982744}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -6.269534631980001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": 3.8344533964}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": 2.6926661446}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -7.4531878246100005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": 1.6544215471200001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": 3.9942992973400004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Whole_Blood", "value": -7.608900141960001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Whole_Blood", "value": 3.2130038501700002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Whole_Blood", "value": -1.21969450861}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Whole_Blood", "value": -1.23639277879}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Whole_Blood", "value": 0.647203862667}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Whole_Blood", "value": 0.501527428627}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Whole_Blood", "value": 3.7354659176199996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Whole_Blood", "value": 3.62887983643}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Whole_Blood", "value": 1.67926384292}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Whole_Blood", "value": 1.65105962753}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Whole_Blood", "value": 7.16733045125}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Whole_Blood", "value": -0.625163316727}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Whole_Blood", "value": -2.04909932708}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Whole_Blood", "value": 22.864742351599997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Whole_Blood", "value": -10.7260876723}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Whole_Blood", "value": -10.7260876723}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Whole_Blood", "value": -1.12285280228}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Whole_Blood", "value": -1.05461812019}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Whole_Blood", "value": -2.66168936965}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Whole_Blood", "value": 0.20169629842}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Whole_Blood", "value": 1.8482928276099997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 1.48413581491}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 4.9760914575}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 2.0201834518400004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 2.46063657874}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 5.704044172200001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 3.35038912362}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 2.98783218929}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 2.53784674028}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 3.5917422574300004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 4.11854557871}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 3.82661116687}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 4.82251894934}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": -4.75705335001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": -6.84237243171}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": -4.15800237357}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": -4.354064123730001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 3.33901173272}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": -3.31376004027}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": -2.02498186542}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": -2.67455879728}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": -1.48054066498}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": -0.408986072312}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 1.61375519285}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 1.8958372960099998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": -3.08747742536}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 6.261381890700001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": -5.07025604617}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 4.10324557779}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 2.83446457186}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": -5.68549670956}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 1.65728746815}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": 2.10278860545}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Artery_Aorta", "value": -3.8151876421399997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Artery_Aorta", "value": 2.01131633987}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Artery_Aorta", "value": -1.9862706619}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Artery_Aorta", "value": -1.86052420186}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Artery_Aorta", "value": 0.6941578582960001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Artery_Aorta", "value": 0.612132985025}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Artery_Aorta", "value": 3.39625176402}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Artery_Aorta", "value": 3.47827381867}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Artery_Aorta", "value": 4.165645899559999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Artery_Aorta", "value": -2.78957377732}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Artery_Aorta", "value": 7.93523002391}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Artery_Aorta", "value": -1.9531497036400003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Artery_Aorta", "value": -3.13521131032}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Artery_Aorta", "value": 15.0015646148}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Artery_Aorta", "value": -10.3621417845}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Artery_Aorta", "value": -10.3621417845}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Artery_Aorta", "value": -1.0772172149299999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Artery_Aorta", "value": -1.56530007568}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Artery_Aorta", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Artery_Aorta", "value": -2.17330922832}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Artery_Aorta", "value": -4.3440930813800005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Artery_Coronary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": -6.70408161753}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": -7.14959449197}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": -3.20574989149}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": -3.4077849331700003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 2.57456378916}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": -4.22774696113}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": -2.8398629241}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": -3.88411929023}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": -1.74795495562}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 1.38172831116}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 5.31939839712}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 5.320926364719999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": -2.27478081286}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 8.22953156577}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": -6.91268511878}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 3.3979108222900005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 2.72748002046}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": -6.43047799626}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 2.38349570414}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": 0.0421643142426}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Artery_Tibial", "value": -4.38046036684}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Artery_Tibial", "value": 3.07062435449}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Artery_Tibial", "value": -1.83482986592}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Artery_Tibial", "value": -2.84595811052}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Artery_Tibial", "value": 3.78548185729}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Artery_Tibial", "value": 3.61782946487}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Artery_Tibial", "value": 2.15302062567}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Artery_Tibial", "value": 7.39136878883}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Artery_Tibial", "value": -2.4154754185900003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Artery_Tibial", "value": 15.230123441500002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Artery_Tibial", "value": -18.5016282355}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Artery_Tibial", "value": -18.5016282355}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Artery_Tibial", "value": -2.0683722661200004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Artery_Tibial", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 6.050494956930001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 5.09599812625}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 3.0809520598400004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 2.46299914153}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 5.12480675891}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": -1.40718910658}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 2.17655121433}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 1.9071361551099997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": -1.0352775122700002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": -1.40967928932}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 0.0647337012442}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 0.837249738717}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": -5.23178697119}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 1.8867488604900002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 6.44044068712}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": -1.88338539153}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 1.7225290677}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": -2.9185613626399998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": -0.356533557557}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 1.3878775693599998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Heart_Left_Ventricle", "value": 1.76787814061}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Heart_Left_Ventricle", "value": 0.398728623709}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Heart_Left_Ventricle", "value": 3.3363453604}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Heart_Left_Ventricle", "value": 0.723161105092}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Heart_Left_Ventricle", "value": 1.7821427148}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Heart_Left_Ventricle", "value": -1.0217664777}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Heart_Left_Ventricle", "value": 0.119161676429}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Heart_Left_Ventricle", "value": 1.13717899247}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Heart_Left_Ventricle", "value": 1.20110048309}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Heart_Left_Ventricle", "value": 2.5710588029799997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Heart_Left_Ventricle", "value": -1.1289304338200001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Heart_Left_Ventricle", "value": 1.40412640572}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Heart_Left_Ventricle", "value": -1.3774012221}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Heart_Left_Ventricle", "value": 5.26021573857}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Heart_Left_Ventricle", "value": 27.2909979959}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Heart_Left_Ventricle", "value": 27.2909979959}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Heart_Left_Ventricle", "value": 1.3043641266700001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Heart_Left_Ventricle", "value": 1.3590358464200003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Heart_Left_Ventricle", "value": 2.5773818742200003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Heart_Left_Ventricle", "value": 2.54038559452}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Spleen", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Spleen", "value": 0.0}], "data-be276ec5314085486ce5fa431438fef2": [{"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": -6.110015270469999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": -6.4967848648}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": -2.88654181814}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": -3.12082534867}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 2.9714399485}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": -4.44075794777}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": -2.90585221518}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": -3.95905698792}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": -1.50788155088}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 1.2154525177200002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 4.94225341029}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 5.03360158007}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": -2.74753903049}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 8.997711803889999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": -6.31612651675}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 3.1164069451}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 2.5778716410799998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": -6.60932199367}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 2.23282639397}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": 0.00751919279189}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Adipose_Subcutaneous", "value": -4.264062372840001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Adipose_Subcutaneous", "value": 2.96604898995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Adipose_Subcutaneous", "value": -2.19262383409}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Adipose_Subcutaneous", "value": -2.5034655241099997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Adipose_Subcutaneous", "value": 1.65960415312}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Adipose_Subcutaneous", "value": -0.986302128421}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Adipose_Subcutaneous", "value": 3.4763743782999996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Adipose_Subcutaneous", "value": 4.3115395238800005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Adipose_Subcutaneous", "value": 2.21262979671}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Adipose_Subcutaneous", "value": 1.68542369239}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Adipose_Subcutaneous", "value": 6.779805113609999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Adipose_Subcutaneous", "value": -3.6066122088}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Adipose_Subcutaneous", "value": -2.59051361075}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Adipose_Subcutaneous", "value": 12.439571719100002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Adipose_Subcutaneous", "value": -19.451063366099998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Adipose_Subcutaneous", "value": -19.451063366099998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Adipose_Subcutaneous", "value": 0.211185716932}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Adipose_Subcutaneous", "value": 0.120540993632}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Adipose_Subcutaneous", "value": -2.7648939666000003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Adipose_Subcutaneous", "value": 1.42973108699}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": -3.85531624509}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": -2.4383851948900004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.17415630425200002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": -0.189975455602}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 1.8404252076900003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": -0.7661162918209999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.894314683376}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.292171477063}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": -1.08814891591}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 2.74770642268}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 6.04436730354}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 5.74655749987}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": -1.72898250164}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 9.008715773139999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": -3.7072777538599997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.429363717482}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 1.3061639035600001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": -5.29831364574}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 2.8093377439}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": -2.23791854458}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Adipose_Visceral_Omentum", "value": 0.663913124418}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Adipose_Visceral_Omentum", "value": 0.9836983921789999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Adipose_Visceral_Omentum", "value": -1.3644509293599998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Adipose_Visceral_Omentum", "value": -1.42933942088}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Adipose_Visceral_Omentum", "value": 3.9535815715800005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Adipose_Visceral_Omentum", "value": 1.90793204308}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Adipose_Visceral_Omentum", "value": 1.7923061607700002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Adipose_Visceral_Omentum", "value": 1.8003647108}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Adipose_Visceral_Omentum", "value": 1.7073719018900002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Adipose_Visceral_Omentum", "value": -0.21908608079}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Adipose_Visceral_Omentum", "value": 3.86903187245}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Adipose_Visceral_Omentum", "value": -2.26721024513}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Adipose_Visceral_Omentum", "value": -1.73561960342}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Adipose_Visceral_Omentum", "value": 5.98003230701}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Adipose_Visceral_Omentum", "value": -11.383919738800001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Adipose_Visceral_Omentum", "value": -11.383919738800001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Adipose_Visceral_Omentum", "value": -1.1812586784399999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Adipose_Visceral_Omentum", "value": -1.2086430788}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Adipose_Visceral_Omentum", "value": -1.3317041505299998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Adipose_Visceral_Omentum", "value": -3.53214716911}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 2.0876239892}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": -0.371230051272}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 0.9109025598530001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 0.0769783236366}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 1.55690865074}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 0.0264191347654}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 1.7232019658000002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 0.945421802735}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": -0.108398886911}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 0.10955538987400001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": -1.2164645033700001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": -0.256777464622}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": -5.50397576728}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": -8.08470747388}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": -5.4722371264300005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": -5.31331063456}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 2.48457798346}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": -4.7048910611300006}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": -2.99288669879}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": -4.10492885826}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": -0.10398338665}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 0.402008261814}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 4.76115724737}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 5.17703228187}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": -2.71395708473}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 10.2027762054}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": -5.91264934809}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 4.8929771164}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 1.82394723853}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": -5.978663893119999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 1.7409706686000002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": 0.8347182325120001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Breast_Mammary_Tissue", "value": -4.882984531530001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Breast_Mammary_Tissue", "value": 2.11637965257}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Breast_Mammary_Tissue", "value": -1.5855006982}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Breast_Mammary_Tissue", "value": -0.560679735827}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Breast_Mammary_Tissue", "value": 1.54946703759}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Breast_Mammary_Tissue", "value": -0.8771969986839999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Breast_Mammary_Tissue", "value": 2.42908472435}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Breast_Mammary_Tissue", "value": 4.23803521331}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Breast_Mammary_Tissue", "value": 0.46495836551700004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Breast_Mammary_Tissue", "value": 1.3418045960200002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Breast_Mammary_Tissue", "value": 4.59817696074}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Breast_Mammary_Tissue", "value": -1.03310517224}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Breast_Mammary_Tissue", "value": -2.68328755251}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Breast_Mammary_Tissue", "value": 18.5090012122}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Breast_Mammary_Tissue", "value": -18.1167198718}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Breast_Mammary_Tissue", "value": -18.1167198718}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Breast_Mammary_Tissue", "value": -2.73785975893}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Breast_Mammary_Tissue", "value": -2.92015643962}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Breast_Mammary_Tissue", "value": -3.36898888323}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Breast_Mammary_Tissue", "value": -1.9271966319}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Breast_Mammary_Tissue", "value": 0.72508004001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 1.9640008211099997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": -1.2315961122499999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": -0.554699838161}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 0.7050164341930001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 1.34849262238}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 0.0152910286561}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 0.8130314946170001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 1.0877624750100001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 0.21213719248799998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 0.267869204283}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 1.76341807842}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 0.978150308132}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": -3.29903517244}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": -4.1531670398400005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": -2.94299546046}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": -2.7408677400099997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 2.63547777554}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": -1.9191705214800001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": -3.06866923136}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": -3.7707475901699996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 1.8709606523099997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": -0.048782792408800006}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 3.9917910771800007}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 4.65665485832}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": -3.6123866929000004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 7.91578417847}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": -3.5806690934099996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 2.35390385554}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": -0.181674138006}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": -3.9917405820300003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": 2.10624701105}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": -1.02511739881}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Muscle_Skeletal", "value": -3.285320028}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Muscle_Skeletal", "value": 1.2292486735500001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Muscle_Skeletal", "value": -1.21005028597}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Muscle_Skeletal", "value": -2.62868685419}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Muscle_Skeletal", "value": -2.1248395443}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Muscle_Skeletal", "value": 0.0903614386916}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Muscle_Skeletal", "value": 1.8052621701599998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Muscle_Skeletal", "value": 4.127489721530001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Muscle_Skeletal", "value": 1.77982907167}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Muscle_Skeletal", "value": -6.83493709564}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Muscle_Skeletal", "value": 2.70108350178}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Muscle_Skeletal", "value": 1.1207537651100001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Muscle_Skeletal", "value": -2.2172999159}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Muscle_Skeletal", "value": 7.43673254582}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Muscle_Skeletal", "value": -8.01911552976}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Muscle_Skeletal", "value": -7.908252369310001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Muscle_Skeletal", "value": 2.03910040855}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Muscle_Skeletal", "value": 2.0080308914200002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Muscle_Skeletal", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Muscle_Skeletal", "value": -0.494566869685}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Muscle_Skeletal", "value": -8.292599678039998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.87218873996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -3.80347731199}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -5.47277690995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.61013226053}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.2696235156599998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 5.2980656890199995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 7.7108691954}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.01173924486}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 5.83106260929}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 5.856321630219999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 5.17545360998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 6.61154149069}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.45904445114}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 5.1019586717}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.68911860422}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.9784148372099994}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 4.5540980396699995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.4554209281}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -1.75194072882}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.25136938032}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -0.94625172922}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -1.37686438659}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.6733227193469999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.49588305778}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -4.4365331570999995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.7536057310800004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.90804789857}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.89174225283}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.31142292989}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.71934565403}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -1.83084519028}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -0.70406031921}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -1.4581740245200001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.82680294819}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.027006808420099997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.448884194659}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 4.25259456509}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -3.29811939432}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.66928013938}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.90350821008}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -3.1577936902499997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.29551554424}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -0.89464939923}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -13.9669666451}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.160009054572}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.55299843656}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -5.9606730171899995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -6.02468764725}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.31308736231}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.35062809649}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.250399818923}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -6.6556383519599995}], "data-da4b445591b9c0699a70f658a69716ba": [{"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 1.28384701018}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 0.276661859337}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 0.8383274125169999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": -0.024162843451799997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 6.244426124219999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": -5.3601106273}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": -2.7048902680299998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": -3.63033470989}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": -2.6765751657}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": -2.89923452056}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 0.44386844790799995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 1.7485064994400001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": -5.73094893526}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 6.083295538680001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 1.33536949101}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 0.36484295865}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 3.8044166812099998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": -6.4983075961}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 2.13923206585}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": 1.89569529257}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Adrenal_Gland", "value": -3.91628263435}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Adrenal_Gland", "value": 3.9831721358699994}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Adrenal_Gland", "value": 0.5713160549849999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Adrenal_Gland", "value": 0.37540760706799997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Adrenal_Gland", "value": 3.851687969}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Adrenal_Gland", "value": 2.79461808481}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Adrenal_Gland", "value": -0.6468405874050001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Adrenal_Gland", "value": 3.1773421432599998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Adrenal_Gland", "value": -1.18836554567}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Adrenal_Gland", "value": 9.280070802300001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Adrenal_Gland", "value": 14.6766401272}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Adrenal_Gland", "value": 14.6766401272}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Adrenal_Gland", "value": 0.60901945008}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Adrenal_Gland", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Pituitary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Pancreas", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Pancreas", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Pancreas", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Pancreas", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Pancreas", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Pancreas", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Pancreas", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Pancreas", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Pancreas", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Pancreas", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Pancreas", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Pancreas", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Pancreas", "value": -0.9193394435500001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Pancreas", "value": -0.5341831678}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Pancreas", "value": 0.0392029613141}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Pancreas", "value": 0.09611391935739999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Pancreas", "value": 3.5853727783699996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Pancreas", "value": -4.287199964569999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Pancreas", "value": -0.30822545955200004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Pancreas", "value": -1.1002040706}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Pancreas", "value": 1.6347609026899999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Pancreas", "value": -0.58854217748}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Pancreas", "value": 1.62663098124}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Pancreas", "value": 1.94151726949}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Pancreas", "value": -4.36432325019}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Pancreas", "value": 5.06613269583}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Pancreas", "value": -0.868185766626}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Pancreas", "value": 0.0056955177887200005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Pancreas", "value": -0.251512195398}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Pancreas", "value": -5.35823422986}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Pancreas", "value": 2.45625605391}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Pancreas", "value": 2.9998041459500002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Pancreas", "value": -1.5547347291100002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Pancreas", "value": 2.4455650791999997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Pancreas", "value": -0.13670609509099999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Pancreas", "value": 1.04049222718}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Pancreas", "value": 0.898911118507}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Pancreas", "value": -0.142101287842}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Pancreas", "value": 2.58450438799}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Pancreas", "value": 1.3746030035299999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Pancreas", "value": 2.99185427843}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Pancreas", "value": -2.07309961319}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Pancreas", "value": 5.72524349239}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Pancreas", "value": 3.07595562935}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Pancreas", "value": -2.38235601676}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Pancreas", "value": 7.961120135710001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Pancreas", "value": 7.049617959219999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Pancreas", "value": 7.049617959219999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Pancreas", "value": -1.5443646907799997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Pancreas", "value": -1.63380408287}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Pancreas", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Pancreas", "value": -2.04766451163}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Pancreas", "value": -3.7902429103900004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Thyroid", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Lung", "value": 1.7757648992299997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Lung", "value": 1.8528760554599997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Lung", "value": 5.09942611206}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Lung", "value": 0.9028593700409999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Lung", "value": -2.79295276124}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Lung", "value": -8.81824600833}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Lung", "value": -5.50585461038}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Lung", "value": -1.24710985515}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Lung", "value": -9.02662339095}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Lung", "value": -8.91734336859}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Lung", "value": -6.93161403635}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Lung", "value": -10.3158388244}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Lung", "value": -8.4373707751}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Lung", "value": -5.1008302982}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Lung", "value": -2.24645413523}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Lung", "value": -1.62538853225}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Lung", "value": -5.06853539449}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Lung", "value": 2.85712444878}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Lung", "value": 1.0710514431}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Lung", "value": 1.42131991277}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Lung", "value": 1.23871989077}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Lung", "value": 5.2945940485300005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Lung", "value": 2.6788802926299997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Lung", "value": 0.537094390465}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Lung", "value": 4.863384900990001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Lung", "value": -4.24084913426}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Lung", "value": -8.60772070763}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Lung", "value": 0.712221349772}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Lung", "value": -2.25681794909}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Lung", "value": 0.136273168777}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Lung", "value": -3.7414191166800004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Lung", "value": -4.0988040054}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Lung", "value": 0.390910696589}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Lung", "value": -1.01288820033}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Lung", "value": 0.377020892386}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Lung", "value": 2.0838810616799996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Lung", "value": -1.4918455609299999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Lung", "value": -2.5565528783300002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Lung", "value": -1.4649507919100002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Lung", "value": 2.7989453709}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Lung", "value": -0.350076469854}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Lung", "value": -1.46806274283}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Lung", "value": -0.631075505553}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Lung", "value": 0.6024191636199999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Lung", "value": -1.5649533104799997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Lung", "value": 6.07177323707}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Lung", "value": -29.9345490252}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Lung", "value": -29.9345490252}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Lung", "value": 0.112525658108}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Lung", "value": 0.144039083613}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Lung", "value": -6.73335027695}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Lung", "value": 0.201142989245}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Lung", "value": -1.38662883776}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Ovary", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Prostate", "value": -6.98766914643}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Prostate", "value": -4.707160400119999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Prostate", "value": -4.6289120055}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Prostate", "value": -4.50096788801}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Prostate", "value": -6.77409388635}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Prostate", "value": -5.8377589597399995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Prostate", "value": -7.19054617622}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Prostate", "value": -3.26514344083}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Prostate", "value": -6.34301652224}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Prostate", "value": -6.77691580065}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Prostate", "value": -6.26474652386}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Prostate", "value": -8.26447652932}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Prostate", "value": 2.17272091036}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Prostate", "value": 0.34037323153800003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Prostate", "value": -0.0862187237303}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Prostate", "value": -0.686238945009}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Prostate", "value": 6.07507914789}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Prostate", "value": -0.18954447495700003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Prostate", "value": 1.08845266861}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Prostate", "value": 0.957211199902}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Prostate", "value": -0.6306650990339999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Prostate", "value": -0.603012131003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Prostate", "value": 0.214510705262}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Prostate", "value": 0.5821621548870001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Prostate", "value": -6.364930273300001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Prostate", "value": 4.9030103943}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Prostate", "value": 2.19270251955}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Prostate", "value": 0.7740846916390001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Prostate", "value": 2.1908172920299998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Prostate", "value": -2.45249554216}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Prostate", "value": -0.418233849019}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Prostate", "value": 0.8181822303049999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Prostate", "value": 1.13699762951}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Prostate", "value": 1.78116614275}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Prostate", "value": 2.4277396257599997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Prostate", "value": 1.4166890165200001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Prostate", "value": -0.100463040329}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Prostate", "value": -2.0335844133900003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Prostate", "value": 1.43420139715}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Prostate", "value": 2.6720740957299998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Prostate", "value": -3.5461873230300003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Prostate", "value": -3.09704383669}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Prostate", "value": 0.255174383697}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Prostate", "value": -0.580440730931}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Prostate", "value": -2.19381230727}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Prostate", "value": 4.42585507706}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Prostate", "value": -11.1358263344}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Prostate", "value": -8.73568024158}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Prostate", "value": -1.43578724481}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Prostate", "value": -1.40482350178}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Prostate", "value": -2.80847812815}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Prostate", "value": 0.735974129326}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Prostate", "value": -4.057876837519999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Testis", "value": -2.55142197785}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Testis", "value": 0.287223052771}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Testis", "value": -3.3923888464200003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Testis", "value": -1.9552383587400002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Testis", "value": -2.8463768142799997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Testis", "value": 5.72488185138}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Testis", "value": 6.7171360594400005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Testis", "value": 2.27611684556}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Testis", "value": 6.03496711011}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Testis", "value": 5.452754432690001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Testis", "value": 4.26380348597}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Testis", "value": 6.12813516678}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Testis", "value": 0.7896110054469999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Testis", "value": -1.02713949054}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Testis", "value": -0.575101780524}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Testis", "value": -1.42721274474}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Testis", "value": 6.43629013994}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Testis", "value": -1.9996011766400001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Testis", "value": -1.4797181369999999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Testis", "value": -1.8257442526099998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Testis", "value": -3.3005414981299994}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Testis", "value": -0.629362827989}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Testis", "value": 0.0214427802533}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Testis", "value": 0.331450711966}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Testis", "value": -5.845897491390001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Testis", "value": 4.05540514462}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Testis", "value": 0.960812061335}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Testis", "value": 1.40976632322}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Testis", "value": 4.4066165991199995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Testis", "value": -5.8896256265}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Testis", "value": -0.9637734840659999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Testis", "value": 0.5082868991139999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Testis", "value": -1.88916429657}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Testis", "value": 3.85050467627}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Testis", "value": 0.822721339328}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Testis", "value": 2.81849785871}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Testis", "value": 2.5817978197299998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Testis", "value": -0.6818366079350001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Testis", "value": 3.6421286327}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Testis", "value": 3.9478818017900004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Testis", "value": -1.30147881733}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Testis", "value": -2.1296651260900004}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Testis", "value": 2.02632559406}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Testis", "value": -4.71172004819}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Testis", "value": -1.47404279813}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Testis", "value": 5.256608513680001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Testis", "value": -10.4409072599}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Testis", "value": -10.4409072599}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Testis", "value": -1.56375593348}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Testis", "value": -1.3176461413200002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Testis", "value": -0.11669721454399999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Testis", "value": 1.10382393093}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Testis", "value": -5.282395635019999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Uterus", "value": 0.6431912779810001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Uterus", "value": -0.106229834259}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Uterus", "value": -2.2235312461900003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Uterus", "value": -2.4655580520599996}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Uterus", "value": 0.935753703117}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Uterus", "value": 2.49713873863}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Uterus", "value": -0.0961518138647}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Uterus", "value": 0.8078099489210001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Uterus", "value": 2.67080307007}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Uterus", "value": 2.57135987282}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Uterus", "value": 2.2266600132}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Uterus", "value": 2.8319160938300003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Uterus", "value": -2.6115691470099995}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Uterus", "value": -3.3741401661699997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Uterus", "value": -1.91642327145}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Uterus", "value": -2.12131504778}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Uterus", "value": 2.3936117614500003}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Uterus", "value": -4.525330987569999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Uterus", "value": -6.551796605519999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Uterus", "value": -7.31863671607}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Uterus", "value": -1.1486841891600001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Uterus", "value": -1.72334273212}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Uterus", "value": 0.596798120321}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Uterus", "value": 1.59126786187}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Uterus", "value": -2.01179670255}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Uterus", "value": 1.8162766152599998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Uterus", "value": -2.67926127923}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Uterus", "value": 1.9422482120599998}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Uterus", "value": 1.9508003577000002}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Uterus", "value": -3.51002787922}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Uterus", "value": -2.08989892269}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Uterus", "value": 0.900692043561}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Uterus", "value": -7.6298770645800005}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Uterus", "value": 4.63444195093}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Uterus", "value": -2.50641067146}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Uterus", "value": 0.0778055560096}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Uterus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Uterus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Uterus", "value": 4.643327241600001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Uterus", "value": 5.06121077013}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Uterus", "value": 1.32189678877}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Uterus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Uterus", "value": 5.86976021944}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Uterus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Uterus", "value": -1.8492048727700001}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Uterus", "value": 12.917640905999999}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Uterus", "value": -21.3867110387}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Uterus", "value": -21.3029944624}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Uterus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Uterus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Uterus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Uterus", "value": -2.3769617309099997}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Uterus", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "study size": 173480, "association": 19.572839000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet count (BLOO01) ", "index": 0, "LDSC Mean Chi-Square": 1.768, "category": "g: blood", "phenotype": "platelet count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2531, "code": "BLOO01", "LDSC Intercept": "0.9967 (0.0271)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "study size": 173480, "association": 17.193386, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red blood cell count (BLOO02) ", "index": 1, "LDSC Mean Chi-Square": 1.6473, "category": "g: blood", "phenotype": "red blood cell count", "LDSC Ratio": "0.0032 (0.0394)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2464, "code": "BLOO02", "LDSC Intercept": "1.0021 (0.0255)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "study size": 173480, "association": 18.078314000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "index": 2, "LDSC Mean Chi-Square": 1.4634, "category": "g: blood", "phenotype": "hemoglobin concentration", "LDSC Ratio": "0.0199 (0.0502)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO03", "LDSC Intercept": "1.0092 (0.0233)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "study size": 173480, "association": 32.011463, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte count (BLOO04) ", "index": 3, "LDSC Mean Chi-Square": 1.6377, "category": "g: blood", "phenotype": "reticulocyte count", "LDSC Ratio": "0.0243 (0.0638)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.207, "code": "BLOO04", "LDSC Intercept": "1.0155 (0.0407)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "study size": 173480, "association": 37.446906, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte count (BLOO05) ", "index": 4, "LDSC Mean Chi-Square": 1.6776, "category": "g: blood", "phenotype": "monocyte count", "LDSC Ratio": "0.0686 (0.0499)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO05", "LDSC Intercept": "1.0465 (0.0338)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "study size": 173480, "association": 14.54171, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil count (BLOO06) ", "index": 5, "LDSC Mean Chi-Square": 1.516, "category": "g: blood", "phenotype": "neutrophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO06", "LDSC Intercept": "0.9567 (0.0204)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "study size": 173480, "association": 26.284874, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil count (BLOO07) ", "index": 6, "LDSC Mean Chi-Square": 1.6196, "category": "g: blood", "phenotype": "eosinophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO07", "LDSC Intercept": "0.9587 (0.0273)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0706 (0.0093)", "study size": 173480, "association": 7.174113, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Basophil count (BLOO08) ", "index": 7, "LDSC Mean Chi-Square": 1.2466, "category": "g: blood", "phenotype": "basophil count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1144, "code": "BLOO08", "LDSC Intercept": "1. (0.0171)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "study size": 173480, "association": 18.102912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte count (BLOO09) ", "index": 8, "LDSC Mean Chi-Square": 1.5217, "category": "g: blood", "phenotype": "granulocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO09", "LDSC Intercept": "0.9591 (0.0206)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "study size": 173480, "association": 20.772062, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "index": 9, "LDSC Mean Chi-Square": 1.5375, "category": "g: blood", "phenotype": "myeloid white cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2168, "code": "BLOO10", "LDSC Intercept": "0.9623 (0.02)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "study size": 173480, "association": 21.737454, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte count (BLOO11) ", "index": 10, "LDSC Mean Chi-Square": 1.6107, "category": "g: blood", "phenotype": "lymphocyte count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO11", "LDSC Intercept": "0.9683 (0.0197)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "study size": 173480, "association": 32.275814000000004, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "White blood cell count (BLOO12) ", "index": 11, "LDSC Mean Chi-Square": 1.5935, "category": "g: blood", "phenotype": "white blood cell count", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2431, "code": "BLOO12", "LDSC Intercept": "0.9648 (0.0192)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "study size": 173480, "association": 30.468662, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "index": 12, "LDSC Mean Chi-Square": 1.5207, "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO13", "LDSC Intercept": "0.9585 (0.0207)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "study size": 173480, "association": 37.344852, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "index": 13, "LDSC Mean Chi-Square": 1.5912, "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1908, "code": "BLOO16", "LDSC Intercept": "0.9694 (0.0258)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "study size": 173480, "association": 14.605566, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "index": 14, "LDSC Mean Chi-Square": 1.5997, "category": "g: blood", "phenotype": "Eosinophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO17", "LDSC Intercept": "0.9613 (0.0261)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "study size": 173480, "association": 16.594893, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "index": 15, "LDSC Mean Chi-Square": 1.5779, "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1876, "code": "BLOO18", "LDSC Intercept": "0.9588 (0.0243)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1561 (0.018)", "study size": 173480, "association": 12.84333, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Granulocyte percentage of myeloid white cells (BLOO19) ", "index": 16, "LDSC Mean Chi-Square": 1.5446, "category": "g: blood", "phenotype": "Granulocyte percentage of myeloid white cells", "LDSC Ratio": "0.0309 (0.0551)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1683, "code": "BLOO19", "LDSC Intercept": "1.0168 (0.03)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "study size": 173480, "association": 20.847865, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Hematocrit (BLOO20) ", "index": 17, "LDSC Mean Chi-Square": 1.47, "category": "g: blood", "phenotype": "Hematocrit", "LDSC Ratio": "0.0221 (0.0511)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2103, "code": "BLOO20", "LDSC Intercept": "1.0104 (0.024)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "study size": 173480, "association": 17.75965, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "index": 18, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "LDSC Ratio": "0.0417 (0.0682)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO21", "LDSC Intercept": "1.0265 (0.0434)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "study size": 173480, "association": 26.629894, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "index": 19, "LDSC Mean Chi-Square": 1.6418, "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "LDSC Ratio": "0.0416 (0.0672)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2201, "code": "BLOO22", "LDSC Intercept": "1.0267 (0.0431)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "study size": 173480, "association": 9.405088000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "index": 20, "LDSC Mean Chi-Square": 1.4799, "category": "g: blood", "phenotype": "Lymphocyte percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO24", "LDSC Intercept": "0.9664 (0.0153)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "study size": 173480, "association": 9.968955000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "index": 21, "LDSC Mean Chi-Square": 1.2926, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.0988, "code": "BLOO25", "LDSC Intercept": "0.9869 (0.0181)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "study size": 173480, "association": 11.247183999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "index": 22, "LDSC Mean Chi-Square": 1.7671, "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Ratio": "0.012 (0.0532)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.194, "code": "BLOO26", "LDSC Intercept": "1.0092 (0.0408)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "study size": 173480, "association": 11.073375, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "index": 23, "LDSC Mean Chi-Square": 1.8174, "category": "g: blood", "phenotype": "Mean corpuscular volume", "LDSC Ratio": "0.0096 (0.0469)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO27", "LDSC Intercept": "1.0078 (0.0383)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1686 (0.0207)", "study size": 173480, "association": 10.552067, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Monocyte percentage of white cells (BLOO28) ", "index": 24, "LDSC Mean Chi-Square": 1.6022, "category": "g: blood", "phenotype": "Monocyte percentage of white cells", "LDSC Ratio": "0.0601 (0.0581)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1715, "code": "BLOO28", "LDSC Intercept": "1.0362 (0.035)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "study size": 173480, "association": 27.292446999999996, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Mean platelet volume (BLOO29) ", "index": 25, "LDSC Mean Chi-Square": 1.901, "category": "g: blood", "phenotype": "Mean platelet volume", "LDSC Ratio": "0.0057 (0.0508)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2005, "code": "BLOO29", "LDSC Intercept": "1.0051 (0.0458)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "study size": 173480, "association": 42.731445, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "index": 26, "LDSC Mean Chi-Square": 1.5234, "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2136, "code": "BLOO30", "LDSC Intercept": "0.961 (0.0204)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "study size": 173480, "association": 14.09174, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "index": 27, "LDSC Mean Chi-Square": 1.5319, "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1843, "code": "BLOO31", "LDSC Intercept": "0.9571 (0.0217)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1411 (0.0122)", "study size": 173480, "association": 8.044601, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Neutrophil percentage of white cells (BLOO32) ", "index": 28, "LDSC Mean Chi-Square": 1.4507, "category": "g: blood", "phenotype": "Neutrophil percentage of white cells", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1973, "code": "BLOO32", "LDSC Intercept": "0.9561 (0.0162)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "study size": 173480, "association": 12.808912, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Plateletcrit (BLOO33) ", "index": 29, "LDSC Mean Chi-Square": 1.6946, "category": "g: blood", "phenotype": "Plateletcrit", "LDSC Ratio": "0.0253 (0.0354)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2498, "code": "BLOO33", "LDSC Intercept": "1.0176 (0.0246)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "study size": 173480, "association": 16.988218, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Platelet distribution width (BLOO34) ", "index": 30, "LDSC Mean Chi-Square": 1.5949, "category": "g: blood", "phenotype": "Platelet distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1747, "code": "BLOO34", "LDSC Intercept": "0.9902 (0.0289)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "study size": 173480, "association": 13.351913, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Red cell distribution width (BLOO35) ", "index": 31, "LDSC Mean Chi-Square": 1.6273, "category": "g: blood", "phenotype": "Red cell distribution width", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.1651, "code": "BLOO35", "LDSC Intercept": "0.9673 (0.0292)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "study size": 173480, "association": 34.282246, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "index": 32, "LDSC Mean Chi-Square": 1.6369999999999998, "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "LDSC Ratio": "0.0197 (0.0653)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "LDSC Lambda GC": 1.2038, "code": "BLOO36", "LDSC Intercept": "1.0125 (0.0416)", "PMID or link": "27863252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "study size": 353972, "association": 11.503943, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "BMI (BODY08) ", "index": 33, "LDSC Mean Chi-Square": 2.6286, "category": "b: anthropometric", "phenotype": "BMI", "LDSC Ratio": "0.0217 (0.01)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0471, "code": "BODY08", "LDSC Intercept": "1.0353 (0.0162)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1203 (0.0202)", "study size": 100184, "association": 7.043577, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Total cholesterol (CHOL21) ", "index": 34, "LDSC Mean Chi-Square": 1.1399, "category": "a: metabolic", "phenotype": "Total cholesterol", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0016, "code": "CHOL21", "LDSC Intercept": "0.9027 (0.0273)", "PMID or link": "20686565", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "study size": 13239, "association": 11.510956, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "index": 35, "LDSC Mean Chi-Square": 1.1057, "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "LDSC Ratio": "0.0369 (0.0909)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "LDSC Lambda GC": 1.0496, "code": "CIRR01", "LDSC Intercept": "1.0039 (0.0096)", "PMID or link": "26394269", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.5029 (0.0592)", "study size": 20883, "association": 3.9654519999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Crohn's disease (CROH01) ", "index": 36, "LDSC Mean Chi-Square": 1.2302, "category": "e: autoimmune", "phenotype": "Crohn's disease", "LDSC Ratio": "0.0946 (0.0489)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1396, "code": "CROH01", "LDSC Intercept": "1.0218 (0.0113)", "PMID or link": "26192919", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "study size": 766345, "association": 1.945157, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "index": 37, "LDSC Mean Chi-Square": 2.6521, "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "LDSC Lambda GC": 2.094, "code": "EDUC03", "LDSC Intercept": "0.979 (0.016)", "PMID or link": "30038396", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "study size": 353972, "association": 15.005072, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat mass (FATM04) ", "index": 38, "LDSC Mean Chi-Square": 2.5512, "category": "b: anthropometric", "phenotype": "Fat mass", "LDSC Ratio": "0.0277 (0.0105)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.0175, "code": "FATM04", "LDSC Intercept": "1.043 (0.0163)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "study size": 123665, "association": 5.703357, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "HbA1c European (GLYC22) ", "index": 39, "LDSC Mean Chi-Square": 1.0661, "category": "a: metabolic", "phenotype": "HbA1c European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "LDSC Lambda GC": 1.0105, "code": "GLYC22", "LDSC Intercept": "0.9192 (0.0088)", "PMID or link": "28898252", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "study size": 253288, "association": 14.00567, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Height 2014 (HEIG03) ", "index": 40, "LDSC Mean Chi-Square": 2.9423, "category": "b: anthropometric", "phenotype": "Height 2014", "LDSC Ratio": "0.1178 (0.0152)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "LDSC Lambda GC": 2.0007, "code": "HEIG03", "LDSC Intercept": "1.2288 (0.0295)", "PMID or link": "25282103", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.3352 (0.0346)", "study size": 34652, "association": 13.778585999999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Inflammatory bowel diseases (UC + CD) (INFB01) ", "index": 41, "LDSC Mean Chi-Square": 1.2790000000000001, "category": "e: autoimmune", "phenotype": "Inflammatory bowel diseases (UC + CD)", "LDSC Ratio": "0.1827 (0.0362)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.1747, "code": "INFB01", "LDSC Intercept": "1.051 (0.0101)", "PMID or link": "26192919", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "study size": 353972, "association": 42.950897999999995, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "index": 42, "LDSC Mean Chi-Square": 3.1445, "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "LDSC Ratio": "0.0513 (0.0128)", "phenotype_link": "nopmid.html", "LDSC Lambda GC": 2.1851, "code": "LEAN06", "LDSC Intercept": "1.11 (0.0274)", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "study size": 23210, "association": 5.166923000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "index": 43, "LDSC Mean Chi-Square": 1.2181, "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "LDSC Ratio": "0.4868 (0.0447)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "LDSC Lambda GC": 1.1715, "code": "LUPU01", "LDSC Intercept": "1.1062 (0.0098)", "PMID or link": "26502338", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1353 (0.0158)", "study size": 69360, "association": 7.984221000000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Age at Menopause (MENO01F) ", "index": 44, "LDSC Mean Chi-Square": 1.175, "category": "j: other", "phenotype": "Age at Menopause", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26414677", "LDSC Lambda GC": 1.0957, "code": "MENO01F", "LDSC Intercept": "0.9895 (0.0137)", "PMID or link": "26414677", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "study size": 24751, "association": 67.22332800000001, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "index": 45, "LDSC Mean Chi-Square": 1.0872, "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "LDSC Ratio": "0.0664 (0.1142)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "LDSC Lambda GC": 1.0557, "code": "PRSC01", "LDSC Intercept": "1.0058 (0.01)", "PMID or link": "27992413", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "study size": 79799, "association": 93.01243199999999, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "index": 46, "LDSC Mean Chi-Square": 1.1179999999999999, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU01", "LDSC Intercept": "0.9435 (0.0122)", "PMID or link": "24390342", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "study size": 58284, "association": 88.339789, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "index": 47, "LDSC Mean Chi-Square": 1.1164, "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "LDSC Lambda GC": 1.0466, "code": "RHEU02", "LDSC Intercept": "0.9414 (0.0119)", "PMID or link": "24390342", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "study size": 82315, "association": 1.4676790000000002, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "index": 48, "LDSC Mean Chi-Square": 1.755, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "LDSC Ratio": "0.0437 (0.0138)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5696, "code": "SCHI01", "LDSC Intercept": "1.033 (0.0104)", "PMID or link": "25056061", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "study size": 77096, "association": 1.1834209999999998, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "index": 49, "LDSC Mean Chi-Square": 1.7409999999999999, "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Ratio": "0.0515 (0.0139)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "LDSC Lambda GC": 1.5622, "code": "SCHI02", "LDSC Intercept": "1.0382 (0.0103)", "PMID or link": "25056061", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2842 (0.163)", "study size": 3362, "association": 24.576836, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Multiple sclerosis (SCLE01) ", "index": 50, "LDSC Mean Chi-Square": 1.0438, "category": "e: autoimmune", "phenotype": "Multiple sclerosis", "LDSC Ratio": "0.5259 (0.2279)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/17660530", "LDSC Lambda GC": 1.0618, "code": "SCLE01", "LDSC Intercept": "1.023 (0.01)", "PMID or link": "17660530", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.1493 (0.0259)", "study size": 96598, "association": 6.070106, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Triglycerides (TRGL03) ", "index": 51, "LDSC Mean Chi-Square": 1.1502, "category": "a: metabolic", "phenotype": "Triglycerides", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/20686565", "LDSC Lambda GC": 1.0075, "code": "TRGL03", "LDSC Intercept": "0.8638 (0.0143)", "PMID or link": "20686565", "variable": "Vagina", "value": 0.0}, {"gene": "AGER", "LDSC Total Observed Scale h2": "0.2586 (0.0324)", "study size": 27432, "association": 16.287014000000003, "ENSEMBL": "ENSG00000204305", "phenotype_reference": "Ulcerative Colitis (ULCC01) ", "index": 52, "LDSC Mean Chi-Square": 1.1833, "category": "e: autoimmune", "phenotype": "Ulcerative Colitis", "LDSC Ratio": "0.2487 (0.0533)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26192919", "LDSC Lambda GC": 1.127, "code": "ULCC01", "LDSC Intercept": "1.0456 (0.0098)", "PMID or link": "26192919", "variable": "Vagina", "value": 0.0}]}};
    var embed_opt = {"mode": "vega-lite"};

    function showError(el, error){
        el.innerHTML = ('<div class="error">'
                        + '<p>JavaScript Error: ' + error.message + '</p>'
                        + "<p>This usually means there's a typo in your chart specification. "
                        + "See the javascript console for the full traceback.</p>"
                        + '</div>');
        throw error;
    }
    const el = document.getElementById('vis');
    vegaEmbed("#vis", spec, embed_opt)
      .catch(error => showError(el, error));
  </script>
</body>
</html>
