
<!DOCTYPE html>

<html>
<head><style>.vega-actions a {margin-right: 12px;color: #757575;font-weight: normal;font-size: 13px;}.error {color: red;}</style><script src="https://cdn.jsdelivr.net/npm//vega@4" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-lite@2.6.0" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-embed@3" type="text/javascript"></script><meta content="text/html; charset=utf-8" http-equiv="Content-Type"/><meta content="IE=edge" http-equiv="X-UA-Compatible"/><meta content="width=device-width, initial-scale=1" name="viewport"/><link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/Font-Awesome-4.7/css/font-awesome.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap-theme.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/images/gnomic_icon.png" rel="icon" type="image/png"/><script src="https://phenviz.navigome.com/js/jquery.min.js"></script><script src="https://phenviz.navigome.com/js/bootstrap.js"></script><title>Gene profile: ENSG00000196966</title></head>
<body>
<h4>Gene profile across phenotypes - ENSG00000196966 (HIST1H3E)</h4><div id="vis"></div><div class="col-lg-12 col-md-12 content-item content-item-1"><p>This is a visualization of GWAS-derived gene associations  across Navigome phenotypes. Only phenotypes for which the gene has at least one significant association (p-value â‰¤ 0.05/20000) in <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> <b>and/or</b> <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a> analyses are shown. The first panel is a visualization of gene-phenotype associations obtained using <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> (-log10(p-value)). Hover over any bar to highlight the phenotype in the other panels. The other panels are parallel coordinates visualizations for tissue-wise associations (z-scores)computed using <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a>  and eQTL expression data from GTEx and DGN. Significance thresholds are delineated with black lines. The parallel coordinates show which phenotypes have a significant association and in which tissues, based on gene expression data. Clicking on a specific phenotype will lead to the reference of the GWAS study.</p></div>
<script type="text/javascript">
    var spec = {"config": {"view": {"width": 400, "height": 300}}, "vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "bar", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "href": {"type": "nominal", "field": "phenotype_link"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "quantitative", "axis": {"title": "MAGMA -log10(p)"}, "field": "value"}, "y": {"type": "nominal", "axis": {"title": ""}, "field": "phenotype_reference"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-4e93ee1a91df87c61b7d73e211f0f4d1"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "x": {"type": "quantitative", "field": "ThresholdVal"}}}], "title": "MAGMA gene-wise association"}], "center": true, "data": {"name": "data-952685c782c7c464bcfd67db9b1b5f73"}}, {"vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Nervous system", "width": 500}], "center": true, "data": {"name": "data-aab812fa42132b79660fd36490b5061a"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Digestive System", "width": 500}], "center": true, "data": {"name": "data-fda6895a2b3cb3d1c9bf3801652aaa6a"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Blood, Arteries and Heart", "width": 500}], "center": true, "data": {"name": "data-8598cfa8f5616616f25367b6bfb77e4a"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Adipose Tissue, Muscle, Skin and Cells", "width": 500}], "center": true, "data": {"name": "data-65e81b922a9b3b0da2a40371079c3a2a"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3646672021, 26.3646672021]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Endocrine System and Others", "width": 500}], "center": true, "data": {"name": "data-ce3762ec2e4166403b9ed0d98ba8f7a4"}}]}], "spacing": 40, "$schema": "https://vega.github.io/schema/vega-lite/v2.6.0.json", "datasets": {"data-952685c782c7c464bcfd67db9b1b5f73": [{"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "magma_log10p", "value": 18.536361}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "magma_log10p", "value": 5.9573849999999995}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "magma_log10p", "value": 27.282787}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "magma_log10p", "value": 12.58902}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "magma_log10p", "value": 14.472422}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "magma_log10p", "value": 10.947421}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "magma_log10p", "value": 7.488745}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "magma_log10p", "value": 9.489509}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "magma_log10p", "value": 23.59941}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "magma_log10p", "value": 20.767460999999997}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "magma_log10p", "value": 5.859083999999999}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "magma_log10p", "value": 13.207391000000001}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "magma_log10p", "value": 12.811128}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "magma_log10p", "value": 3.4357480000000002}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "magma_log10p", "value": 3.431259}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "magma_log10p", "value": 23.230629999999998}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "magma_log10p", "value": 29.375521999999997}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "magma_log10p", "value": 42.353606}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "magma_log10p", "value": 6.001004}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "magma_log10p", "value": 7.128514999999999}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "magma_log10p", "value": 37.978521}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "magma_log10p", "value": 9.981674}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "magma_log10p", "value": 10.182825}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "magma_log10p", "value": 25.839411}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "magma_log10p", "value": 1.71193}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "magma_log10p", "value": 9.935130000000001}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "magma_log10p", "value": 37.329476}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "magma_log10p", "value": 18.707345}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "magma_log10p", "value": 8.835617}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "magma_log10p", "value": 9.861066000000001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "magma_log10p", "value": 13.480684}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "magma_log10p", "value": 8.413931}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "magma_log10p", "value": 12.813948}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "magma_log10p", "value": 13.004027}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "magma_log10p", "value": 12.738856}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "magma_log10p", "value": 17.339713}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "magma_log10p", "value": 12.501510000000001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "magma_log10p", "value": 9.697258}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "magma_log10p", "value": 24.440535999999998}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "magma_log10p", "value": 4.276905999999999}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "magma_log10p", "value": 11.611721000000001}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "magma_log10p", "value": 15.580010999999999}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "magma_log10p", "value": 14.643649}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "magma_log10p", "value": 37.536525}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "magma_log10p", "value": 8.757334}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "magma_log10p", "value": 14.960745000000001}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "magma_log10p", "value": 16.370407}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "magma_log10p", "value": 7.116089}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "magma_log10p", "value": 6.507729}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "magma_log10p", "value": 5.846063}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "magma_log10p", "value": 6.914781}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "magma_log10p", "value": 12.600967}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "magma_log10p", "value": 6.442168}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "magma_log10p", "value": 7.077752}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "magma_log10p", "value": 11.746444}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "magma_log10p", "value": 9.551618}], "data-4e93ee1a91df87c61b7d73e211f0f4d1": [{"Threshold": "MAGMA -log10(p) \u2265 5.60206", "ThresholdVal": 5.60206}], "data-aab812fa42132b79660fd36490b5061a": [{"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 2.18311738293}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": -4.72638461482}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 4.26259894396}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 2.33669779494}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 3.40709666333}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.8553268810599997}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.95137066232}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 2.13039443156}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 2.67897111268}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 2.8408511216400005}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 2.22140612995}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.98250891714}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 3.28501126417}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.7222315237000001}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.585482407428}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 4.451682595059999}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 7.80495644153}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 6.8331386491199995}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 2.3492727282900003}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": -4.23384200482}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": -4.5355083480000005}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 2.44541382715}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.4218408196}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 3.0259997248799997}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": -1.54588471974}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": -0.544560067543}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.42691282426}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 2.1758764260900003}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": -0.06422632043769999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.3693449994100002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.48908116565}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.3502598146200002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.3812806195700003}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 2.1640894477200003}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.34427054386699996}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.87117295126}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 2.1128635362}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.274771377246}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.72864183311}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": -1.30252063603}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 5.48532974793}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 3.40909198931}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": -0.542708631241}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": -3.04214148943}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.5411704644449999}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": -0.6304812156090001}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": -0.597667882707}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.6577632977399999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.56233960358}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.55762510921}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.02294163856}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.60876595215}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.5341559267299998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.982312119917}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.54822924743}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 1.7332685484400001}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Brain_Caudate", "value": 4.33455287451}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Brain_Caudate", "value": -4.793744165390001}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Brain_Caudate", "value": 10.4277110426}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Brain_Caudate", "value": 4.96268225384}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Brain_Caudate", "value": 2.26357667358}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Brain_Caudate", "value": 0.36428367927199995}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Brain_Caudate", "value": 0.680810146029}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Brain_Caudate", "value": 0.895771471266}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Brain_Caudate", "value": 0.790149242757}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Brain_Caudate", "value": 1.06311618823}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Brain_Caudate", "value": 0.658052615972}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Brain_Caudate", "value": 0.64488765092}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Brain_Caudate", "value": 6.27209558015}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Brain_Caudate", "value": 2.8853989029400005}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Brain_Caudate", "value": 2.31792352907}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Brain_Caudate", "value": 12.7104034197}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Brain_Caudate", "value": 19.9613687021}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Brain_Caudate", "value": 16.126147788900003}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Brain_Caudate", "value": 0.756768138306}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Brain_Caudate", "value": -4.24665032763}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Brain_Caudate", "value": -12.8469120916}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Brain_Caudate", "value": 5.90655754485}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Brain_Caudate", "value": 1.40417710014}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Brain_Caudate", "value": 5.569737398480001}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Brain_Caudate", "value": -3.1956632812700003}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Brain_Caudate", "value": 1.25572263131}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Brain_Caudate", "value": 3.7176604102399997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Brain_Caudate", "value": 3.75242033856}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Brain_Caudate", "value": 1.8409603604799998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Brain_Caudate", "value": 2.78910405093}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Brain_Caudate", "value": 2.56969398196}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Brain_Caudate", "value": 2.8334765932200003}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Brain_Caudate", "value": 2.45134678537}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Brain_Caudate", "value": 2.9193208956}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Brain_Caudate", "value": 2.21861246937}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Brain_Caudate", "value": 3.58785580681}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Brain_Caudate", "value": 2.79809906116}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Brain_Caudate", "value": 2.1976240856400002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Brain_Caudate", "value": 3.4076968802599996}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Brain_Caudate", "value": -5.92572900508}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Brain_Caudate", "value": 10.4936053434}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Brain_Caudate", "value": 5.5332165713}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Brain_Caudate", "value": -0.5190234971159999}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Brain_Caudate", "value": -2.49847355977}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Brain_Caudate", "value": -0.591029869497}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Brain_Caudate", "value": -1.50941374098}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Brain_Caudate", "value": -1.2567940832100002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Brain_Caudate", "value": 2.3630915545200004}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Brain_Caudate", "value": 2.22809186168}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Brain_Caudate", "value": 2.48348135159}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Brain_Caudate", "value": 1.90338217313}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Brain_Caudate", "value": 2.97582137926}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Brain_Caudate", "value": 2.49314965966}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Brain_Caudate", "value": 1.83917910357}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Brain_Caudate", "value": 2.98346512568}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Brain_Caudate", "value": 2.64344755287}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Brain_Nucleus_accumbens", "value": 2.99061733033}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Brain_Nucleus_accumbens", "value": -4.43108008306}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Brain_Nucleus_accumbens", "value": 6.11911365652}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Brain_Nucleus_accumbens", "value": 4.18432414844}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Brain_Nucleus_accumbens", "value": 2.1175147204}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Brain_Nucleus_accumbens", "value": 1.04659954945}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Brain_Nucleus_accumbens", "value": 0.317703174643}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Brain_Nucleus_accumbens", "value": 0.448661237691}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Brain_Nucleus_accumbens", "value": 2.7521284677099995}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Brain_Nucleus_accumbens", "value": 1.39641404602}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Brain_Nucleus_accumbens", "value": 0.22492645957}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Brain_Nucleus_accumbens", "value": 1.3607046185799998}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Brain_Nucleus_accumbens", "value": 3.87980588333}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Brain_Nucleus_accumbens", "value": 2.5803858638499997}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Brain_Nucleus_accumbens", "value": 2.35537975882}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Brain_Nucleus_accumbens", "value": 7.01916362584}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Brain_Nucleus_accumbens", "value": 10.4716214123}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Brain_Nucleus_accumbens", "value": 8.59198299014}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Brain_Nucleus_accumbens", "value": 0.32969602936299996}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Brain_Nucleus_accumbens", "value": -4.46053357206}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Brain_Nucleus_accumbens", "value": -6.60795647784}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Brain_Nucleus_accumbens", "value": 4.569893688630001}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Brain_Nucleus_accumbens", "value": 2.10928231771}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Brain_Nucleus_accumbens", "value": 4.47713903985}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Brain_Nucleus_accumbens", "value": -1.7745768731599998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Brain_Nucleus_accumbens", "value": 0.34051350588}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Brain_Nucleus_accumbens", "value": 2.7546415075599997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Brain_Nucleus_accumbens", "value": 3.52019195565}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Brain_Nucleus_accumbens", "value": 1.0641001430500001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Brain_Nucleus_accumbens", "value": 2.11151883295}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Brain_Nucleus_accumbens", "value": 2.4177637902}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Brain_Nucleus_accumbens", "value": 1.98536553283}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Brain_Nucleus_accumbens", "value": 2.275500917}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Brain_Nucleus_accumbens", "value": 2.7426397506200004}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Brain_Nucleus_accumbens", "value": 1.2255846438299998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Brain_Nucleus_accumbens", "value": 2.8730592003099997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Brain_Nucleus_accumbens", "value": 2.60349640425}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Brain_Nucleus_accumbens", "value": 1.0723472064700001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Brain_Nucleus_accumbens", "value": 2.62320115558}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Brain_Nucleus_accumbens", "value": -1.7846166809000001}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Brain_Nucleus_accumbens", "value": 4.843894605309999}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Brain_Nucleus_accumbens", "value": 5.4031030613499995}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Brain_Nucleus_accumbens", "value": -0.581266394953}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Brain_Nucleus_accumbens", "value": -2.0575714642}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Brain_Nucleus_accumbens", "value": 0.21592735085}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Brain_Nucleus_accumbens", "value": -0.254102275456}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Brain_Nucleus_accumbens", "value": -0.202916009399}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Brain_Nucleus_accumbens", "value": 2.64348233735}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Brain_Nucleus_accumbens", "value": 2.6371953920299998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Brain_Nucleus_accumbens", "value": 2.65343603784}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Brain_Nucleus_accumbens", "value": 1.57190461562}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Brain_Nucleus_accumbens", "value": 2.8117173995200004}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Brain_Nucleus_accumbens", "value": 2.61811040324}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Brain_Nucleus_accumbens", "value": 1.51231661637}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Brain_Nucleus_accumbens", "value": 2.75979354079}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Brain_Nucleus_accumbens", "value": 2.2091374252000002}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Brain_Putamen", "value": 0.7691731284379999}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Brain_Putamen", "value": -3.79086469309}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Brain_Putamen", "value": 8.80396036859}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Brain_Putamen", "value": 5.00672050537}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Brain_Putamen", "value": 2.77200310164}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Brain_Putamen", "value": 1.8579788354900002}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Brain_Putamen", "value": 2.39954875176}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Brain_Putamen", "value": 2.48268812108}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Brain_Putamen", "value": 3.8764458234900006}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Brain_Putamen", "value": 3.61191926851}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Brain_Putamen", "value": 2.26691074211}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Brain_Putamen", "value": 2.17528436764}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Brain_Putamen", "value": 5.650235363609999}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Brain_Putamen", "value": 2.46442283802}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Brain_Putamen", "value": 2.2264901845400003}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Brain_Putamen", "value": 9.75843936025}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Brain_Putamen", "value": 14.0022714974}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Brain_Putamen", "value": 11.319804107000001}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Brain_Putamen", "value": 2.39370719317}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Brain_Putamen", "value": -4.09912525069}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Brain_Putamen", "value": -9.50493879089}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Brain_Putamen", "value": 5.50269596724}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Brain_Putamen", "value": 2.3950421769799997}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Brain_Putamen", "value": 1.24434237379}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Brain_Putamen", "value": -2.63635606091}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Brain_Putamen", "value": -0.439737045604}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Brain_Putamen", "value": 1.47129870958}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Brain_Putamen", "value": 1.94073515185}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Brain_Putamen", "value": 0.390384458637}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Brain_Putamen", "value": 1.5043446123}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Brain_Putamen", "value": 1.16178612093}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Brain_Putamen", "value": 1.4410887777599999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Brain_Putamen", "value": 1.12910719131}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Brain_Putamen", "value": 1.6419510743}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Brain_Putamen", "value": 0.545902832933}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Brain_Putamen", "value": 1.5283102794999999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Brain_Putamen", "value": 1.68416379319}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Brain_Putamen", "value": 0.503918470733}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Brain_Putamen", "value": 1.51180439238}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Brain_Putamen", "value": -3.5345066909500003}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Brain_Putamen", "value": 7.983455799430001}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Brain_Putamen", "value": 1.6766936758700002}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Brain_Putamen", "value": -2.25063844209}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Brain_Putamen", "value": -4.50113646316}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Brain_Putamen", "value": -0.741895417059}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Brain_Putamen", "value": 1.20851400346}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Brain_Putamen", "value": 1.29705630296}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Brain_Putamen", "value": 1.75215071692}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Brain_Putamen", "value": 1.72170152158}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Brain_Putamen", "value": 1.9154697901700002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Brain_Putamen", "value": 1.0749932468999999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Brain_Putamen", "value": 1.9219712077700002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Brain_Putamen", "value": 1.99983438012}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Brain_Putamen", "value": 1.06249220626}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Brain_Putamen", "value": 1.97120047527}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Brain_Putamen", "value": 1.57996615147}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Brain_Cortex", "value": 1.12689337968}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Brain_Cortex", "value": -3.5303137257}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Brain_Cortex", "value": 3.09761930435}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Brain_Cortex", "value": 0.658626076699}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Brain_Cortex", "value": 2.39407619474}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Brain_Cortex", "value": 1.6472308300999998}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Brain_Cortex", "value": 2.0213146835}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Brain_Cortex", "value": 2.10084730754}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Brain_Cortex", "value": 3.2603516773}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Brain_Cortex", "value": 3.1195833688200003}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Brain_Cortex", "value": 1.8798952523900003}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Brain_Cortex", "value": 1.9125656706299998}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Brain_Cortex", "value": 2.47211333405}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Brain_Cortex", "value": -0.662226767916}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Brain_Cortex", "value": -0.550818788856}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Brain_Cortex", "value": 2.1806826096200003}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Brain_Cortex", "value": 2.82790811209}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Brain_Cortex", "value": 2.50316790604}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Brain_Cortex", "value": 2.0493605110599997}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Brain_Cortex", "value": -3.37863075793}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Brain_Cortex", "value": -1.55302452914}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Brain_Cortex", "value": 0.500835956869}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Brain_Cortex", "value": 2.66312212367}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Brain_Cortex", "value": 1.7125206174399998}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Brain_Cortex", "value": -0.24119651566}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Brain_Cortex", "value": -0.8917048711560001}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Brain_Cortex", "value": -0.396770706232}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Brain_Cortex", "value": 2.01921197121}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Brain_Cortex", "value": -1.37260617039}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Brain_Cortex", "value": 0.795304133239}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Brain_Cortex", "value": 0.922256759263}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Brain_Cortex", "value": 0.702515418175}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Brain_Cortex", "value": 0.69540502803}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Brain_Cortex", "value": 1.12946050569}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Brain_Cortex", "value": -0.975611401966}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Brain_Cortex", "value": 0.264837852684}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Brain_Cortex", "value": 0.963012509075}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Brain_Cortex", "value": -1.11913241759}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Brain_Cortex", "value": 0.0180335255013}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Brain_Cortex", "value": 1.8829551667500002}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Brain_Cortex", "value": 1.97748810825}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Brain_Cortex", "value": 1.8554490975799998}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Brain_Cortex", "value": -1.39547349903}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Brain_Cortex", "value": -3.11286812122}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Brain_Cortex", "value": 0.418203799165}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Brain_Cortex", "value": 0.9257699636229999}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Brain_Cortex", "value": 1.18904828825}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Brain_Cortex", "value": 1.33073416296}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Brain_Cortex", "value": 1.1550961535}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Brain_Cortex", "value": 0.7559606960959999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Brain_Cortex", "value": -0.128892712755}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Brain_Cortex", "value": 0.27053419651}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Brain_Cortex", "value": 0.7322240035590001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Brain_Cortex", "value": -0.275214765719}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Brain_Cortex", "value": 0.183113851125}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Brain_Cortex", "value": 1.5288759917299999}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Brain_Hippocampus", "value": 2.32934467646}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Brain_Hippocampus", "value": -4.587818479}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Brain_Hippocampus", "value": 5.50398435615}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Brain_Hippocampus", "value": 3.11634745949}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Brain_Hippocampus", "value": 2.2150595312}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Brain_Hippocampus", "value": 1.55985330747}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Brain_Hippocampus", "value": 0.9850938252439999}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Brain_Hippocampus", "value": 1.09725783854}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Brain_Hippocampus", "value": 2.53831139595}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Brain_Hippocampus", "value": 1.90712993223}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Brain_Hippocampus", "value": 0.8789717772299999}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Brain_Hippocampus", "value": 1.97386830596}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Brain_Hippocampus", "value": 3.5108188501999997}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Brain_Hippocampus", "value": 1.42955399947}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Brain_Hippocampus", "value": 1.24379123096}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Brain_Hippocampus", "value": 6.35272651314}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Brain_Hippocampus", "value": 9.26381326119}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Brain_Hippocampus", "value": 7.55578481457}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Brain_Hippocampus", "value": 1.00892355301}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Brain_Hippocampus", "value": -4.77753620868}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Brain_Hippocampus", "value": -5.40427139865}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Brain_Hippocampus", "value": 3.41089823786}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Brain_Hippocampus", "value": 2.15261220725}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Brain_Hippocampus", "value": 3.5780902054300006}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Brain_Hippocampus", "value": -1.17802252979}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Brain_Hippocampus", "value": -0.257610210929}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Brain_Hippocampus", "value": 1.9022791814700002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Brain_Hippocampus", "value": 2.83405527573}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Brain_Hippocampus", "value": 0.714969703178}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Brain_Hippocampus", "value": 2.04855704704}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Brain_Hippocampus", "value": 1.56064252739}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Brain_Hippocampus", "value": 1.99609416561}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Brain_Hippocampus", "value": 1.4946776012399998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Brain_Hippocampus", "value": 1.71034093965}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Brain_Hippocampus", "value": 0.7883112645379999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Brain_Hippocampus", "value": 1.7969972221700001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Brain_Hippocampus", "value": 1.7162622298399999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Brain_Hippocampus", "value": 0.670616345723}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Brain_Hippocampus", "value": 1.66334751933}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Brain_Hippocampus", "value": -1.6491154220299997}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Brain_Hippocampus", "value": 4.92908713532}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Brain_Hippocampus", "value": 4.41026311184}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Brain_Hippocampus", "value": -0.7789168210609999}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Brain_Hippocampus", "value": -2.05550178955}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Brain_Hippocampus", "value": 0.30287175620900003}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Brain_Hippocampus", "value": -0.800106546301}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Brain_Hippocampus", "value": -0.643187675238}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Brain_Hippocampus", "value": 1.9726321473900001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Brain_Hippocampus", "value": 1.9213714326599998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Brain_Hippocampus", "value": 1.88342966755}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Brain_Hippocampus", "value": 1.13593436515}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Brain_Hippocampus", "value": 1.9527491392700003}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Brain_Hippocampus", "value": 1.88367186983}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Brain_Hippocampus", "value": 1.07815460576}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Brain_Hippocampus", "value": 1.8834529511099998}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Brain_Hippocampus", "value": 1.8427834174200002}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Nerve_Tibial", "value": 2.21559987853}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Nerve_Tibial", "value": -4.68459995328}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Nerve_Tibial", "value": 6.28303164865}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Nerve_Tibial", "value": 3.6008419165}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Nerve_Tibial", "value": 2.2386904366400002}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Nerve_Tibial", "value": 1.59327156058}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Nerve_Tibial", "value": 1.0036261306899998}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Nerve_Tibial", "value": 1.11060556562}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Nerve_Tibial", "value": 2.5446463284200003}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Nerve_Tibial", "value": 1.9372429761200003}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Nerve_Tibial", "value": 0.878253077}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Nerve_Tibial", "value": 1.96938935533}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Nerve_Tibial", "value": 4.0897705186}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Nerve_Tibial", "value": 1.8541645441699999}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Nerve_Tibial", "value": 1.65414451752}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Nerve_Tibial", "value": 6.95478113813}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Nerve_Tibial", "value": 10.438961512999999}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Nerve_Tibial", "value": 8.578213163180001}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Nerve_Tibial", "value": 1.03174018061}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Nerve_Tibial", "value": -4.64487466551}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Nerve_Tibial", "value": -6.29975209841}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Nerve_Tibial", "value": 3.9369280274699996}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Nerve_Tibial", "value": 2.78420842051}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Nerve_Tibial", "value": 3.59211183884}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Nerve_Tibial", "value": -1.48958104216}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Nerve_Tibial", "value": -0.216388993957}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Nerve_Tibial", "value": 1.9862142764200001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Nerve_Tibial", "value": 2.8831012765099997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Nerve_Tibial", "value": 0.733731484119}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Nerve_Tibial", "value": 2.2866686276299997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Nerve_Tibial", "value": 1.8386727248199999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Nerve_Tibial", "value": 2.22511729176}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Nerve_Tibial", "value": 1.7626398656599997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Nerve_Tibial", "value": 1.81295287256}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Nerve_Tibial", "value": 1.07489127583}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Nerve_Tibial", "value": 2.05617579947}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Nerve_Tibial", "value": 1.79806024663}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Nerve_Tibial", "value": 0.9784618331739999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Nerve_Tibial", "value": 1.93109662786}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Nerve_Tibial", "value": -1.7307765680900002}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Nerve_Tibial", "value": 5.22459358558}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Nerve_Tibial", "value": 4.54005979041}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Nerve_Tibial", "value": -0.852115105604}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Nerve_Tibial", "value": -2.3365879706599997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Nerve_Tibial", "value": -0.21860617637899998}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Nerve_Tibial", "value": -0.79007984068}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Nerve_Tibial", "value": -0.622756049423}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Nerve_Tibial", "value": 2.1019746469}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Nerve_Tibial", "value": 2.0519253171700003}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Nerve_Tibial", "value": 1.88384382194}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Nerve_Tibial", "value": 1.22006406962}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Nerve_Tibial", "value": 2.0246949520700004}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Nerve_Tibial", "value": 1.8896800874400002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Nerve_Tibial", "value": 1.20597961821}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Nerve_Tibial", "value": 1.9968213545900002}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Nerve_Tibial", "value": 1.8112576734499999}], "data-c4c9a654d45741ebc60aa567761bff19": [{"Text": "Significance Threshold", "Threshold": "PrediXcan zscore \u2265 4.70813", "ThresholdVal": 4.70813}], "data-4a3aa5bf8c5a41879455008930c25774": [{"Threshold": "PrediXcan zscore \u2264 -4.70813", "ThresholdVal": -4.70813}], "data-fda6895a2b3cb3d1c9bf3801652aaa6a": [{"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Colon_Sigmoid", "value": -0.767231255976}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Colon_Sigmoid", "value": -7.10757239452}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Colon_Sigmoid", "value": 7.55126304538}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Colon_Sigmoid", "value": 5.08607550001}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Colon_Sigmoid", "value": 2.38436581712}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Colon_Sigmoid", "value": 1.35226161481}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Colon_Sigmoid", "value": 0.8970499719920001}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Colon_Sigmoid", "value": 1.1713687795899999}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Colon_Sigmoid", "value": 2.2919926014}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Colon_Sigmoid", "value": 1.86037992063}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Colon_Sigmoid", "value": 0.6873274271359999}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Colon_Sigmoid", "value": 0.963524372403}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Colon_Sigmoid", "value": 5.7288623007}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Colon_Sigmoid", "value": 3.2112278146}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Colon_Sigmoid", "value": 3.09158079334}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Colon_Sigmoid", "value": 7.91374497276}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Colon_Sigmoid", "value": 12.3999222947}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Colon_Sigmoid", "value": 10.2755507059}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Colon_Sigmoid", "value": 0.821700791463}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Colon_Sigmoid", "value": -5.78003130071}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Colon_Sigmoid", "value": -9.00341523023}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Colon_Sigmoid", "value": 6.278812470859999}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Colon_Sigmoid", "value": -0.753342281704}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Colon_Sigmoid", "value": -0.491830094322}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Colon_Sigmoid", "value": -2.6600300167}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Colon_Sigmoid", "value": -1.2037266258200001}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Colon_Sigmoid", "value": 0.432888211376}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Colon_Sigmoid", "value": 1.45803675622}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Colon_Sigmoid", "value": 0.43290853473800006}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Colon_Sigmoid", "value": -0.9799792911859999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Colon_Sigmoid", "value": 0.0526553757718}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Colon_Sigmoid", "value": -1.14054417466}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Colon_Sigmoid", "value": 0.032351216951799995}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Colon_Sigmoid", "value": 0.607665707459}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Colon_Sigmoid", "value": -1.1102342187799998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Colon_Sigmoid", "value": -0.30979575353}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Colon_Sigmoid", "value": 0.786506823252}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Colon_Sigmoid", "value": -1.07224762072}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Colon_Sigmoid", "value": -0.204524041963}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Colon_Sigmoid", "value": -3.19741619293}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Colon_Sigmoid", "value": 7.100769736469999}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Colon_Sigmoid", "value": 1.0619144868500001}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Colon_Sigmoid", "value": -0.48906902133999997}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Colon_Sigmoid", "value": -3.6715815190300005}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Colon_Sigmoid", "value": -0.296949703332}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Colon_Sigmoid", "value": 0.755806315316}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Colon_Sigmoid", "value": 0.813522582879}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Colon_Sigmoid", "value": -0.07949335880760001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Colon_Sigmoid", "value": -0.11109340249800001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Colon_Sigmoid", "value": 0.649941870151}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Colon_Sigmoid", "value": -1.59098909099}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Colon_Sigmoid", "value": -0.583618123362}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Colon_Sigmoid", "value": 0.7779732273989999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Colon_Sigmoid", "value": -1.57680531646}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Colon_Sigmoid", "value": -0.43268305842}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Colon_Sigmoid", "value": 0.42152671916}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Colon_Transverse", "value": 1.50546795221}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Colon_Transverse", "value": -3.85836327579}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Colon_Transverse", "value": 9.07099343652}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Colon_Transverse", "value": 5.0397088100400005}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Colon_Transverse", "value": 2.72356428321}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Colon_Transverse", "value": 1.48438404366}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Colon_Transverse", "value": 1.60656737461}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Colon_Transverse", "value": 1.7072820236500001}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Colon_Transverse", "value": 2.71451705371}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Colon_Transverse", "value": 2.4099910768}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Colon_Transverse", "value": 1.5053645221899998}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Colon_Transverse", "value": 1.76111647002}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Colon_Transverse", "value": 5.54811076983}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Colon_Transverse", "value": 2.8280619848799997}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Colon_Transverse", "value": 2.48424486158}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Colon_Transverse", "value": 10.6621274509}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Colon_Transverse", "value": 15.493225790799999}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Colon_Transverse", "value": 12.462845765399999}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Colon_Transverse", "value": 1.6413247851600001}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Colon_Transverse", "value": -4.5779160998699995}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Colon_Transverse", "value": -9.76644756102}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Colon_Transverse", "value": 5.674894257819999}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Colon_Transverse", "value": 3.0713751338099997}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Colon_Transverse", "value": 2.570481677}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Colon_Transverse", "value": -2.78742170515}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Colon_Transverse", "value": -0.811906988402}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Colon_Transverse", "value": 1.3528018169999998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Colon_Transverse", "value": 2.7342865561400003}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Colon_Transverse", "value": 0.300375967315}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Colon_Transverse", "value": 1.2761401209}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Colon_Transverse", "value": 1.13903617535}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Colon_Transverse", "value": 1.26703375419}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Colon_Transverse", "value": 1.10377605217}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Colon_Transverse", "value": 1.30690864429}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Colon_Transverse", "value": 0.06716440691759999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Colon_Transverse", "value": 0.998601800659}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Colon_Transverse", "value": 1.28675399454}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Colon_Transverse", "value": 0.031088652848400004}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Colon_Transverse", "value": 0.9059098160079999}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Colon_Transverse", "value": -4.36557296135}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Colon_Transverse", "value": 8.87840954265}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Colon_Transverse", "value": 3.2960662241400005}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Colon_Transverse", "value": -0.842525498687}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Colon_Transverse", "value": -2.56927441727}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Colon_Transverse", "value": -0.541443437263}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Colon_Transverse", "value": -0.454479164062}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Colon_Transverse", "value": -0.394684026052}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Colon_Transverse", "value": 1.77403008444}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Colon_Transverse", "value": 1.7454831785}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Colon_Transverse", "value": 1.80270946822}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Colon_Transverse", "value": 0.747446547323}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Colon_Transverse", "value": 1.61453322572}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Colon_Transverse", "value": 1.7882158777700001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Colon_Transverse", "value": 0.717883354892}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Colon_Transverse", "value": 1.58100177102}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Colon_Transverse", "value": 2.04090469854}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.9651206914400003}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": -4.3992762798500005}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.9508008738}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 3.24448367242}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.63575100137}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.56447771085}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.899527395448}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.0492697488}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.73108581883}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.9735270426499998}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.778964487626}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.01269757254}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 3.1917403728099996}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.7734627488499999}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.6794640677000001}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 5.46132954185}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 7.471517103310001}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 5.995070989249999}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.8880372897409999}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": -4.418063964730001}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": -4.9083054722099995}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 3.3953775174299996}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.6860568867900003}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 3.3057891770200003}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": -1.4487312033000002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.352737392057}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.4329307539}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 3.29798324278}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.9175881897689999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.2894617085799998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.18892545136}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.23146916255}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.1130942710799996}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.2121941487}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.9852000751629999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.28854585867}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.2156585820299997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.8776117944410001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.1530189895099996}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": -1.20073953365}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.046577711319999}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 4.46129926609}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.539173845215}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": -2.43822217738}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.137566511591}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.18464448876200001}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": -0.0213830722597}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.40171123729}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.3493296848500003}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.91087154345}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.3420779830299998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.12940225662}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.93627777961}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.2811498042799998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 2.07041264159}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Esophagus_Gastroesophageal_Junction", "value": 1.985628001}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Esophagus_Mucosa", "value": -0.48778721006700004}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Esophagus_Mucosa", "value": -3.81987339563}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Esophagus_Mucosa", "value": 2.9010814625099997}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Esophagus_Mucosa", "value": 0.9524505817550001}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Esophagus_Mucosa", "value": 1.07483255549}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Esophagus_Mucosa", "value": 1.3884819121}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Esophagus_Mucosa", "value": 0.8729070216500001}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Esophagus_Mucosa", "value": 0.804906595619}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Esophagus_Mucosa", "value": 1.71239982083}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Esophagus_Mucosa", "value": 1.4260038316299999}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Esophagus_Mucosa", "value": 0.753581235367}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Esophagus_Mucosa", "value": 1.69330285945}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Esophagus_Mucosa", "value": 2.02885651215}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Esophagus_Mucosa", "value": -0.19843488113699997}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Esophagus_Mucosa", "value": -0.236056881633}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Esophagus_Mucosa", "value": 2.95198945605}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Esophagus_Mucosa", "value": 3.82086840131}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Esophagus_Mucosa", "value": 3.18559958073}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Esophagus_Mucosa", "value": 0.92180778096}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Esophagus_Mucosa", "value": -4.37357783421}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Esophagus_Mucosa", "value": -2.03320532692}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Esophagus_Mucosa", "value": 0.9913141196899999}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Esophagus_Mucosa", "value": 4.59435750129}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Esophagus_Mucosa", "value": 0.244952884803}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Esophagus_Mucosa", "value": -0.146559380423}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Esophagus_Mucosa", "value": -2.4776757854400002}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Esophagus_Mucosa", "value": -2.49926010377}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Esophagus_Mucosa", "value": 0.200947384659}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Esophagus_Mucosa", "value": -1.59033367162}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Esophagus_Mucosa", "value": 0.23576980046799997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Esophagus_Mucosa", "value": -0.7039973224860001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Esophagus_Mucosa", "value": 0.271798059661}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Esophagus_Mucosa", "value": -0.637271678857}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Esophagus_Mucosa", "value": -0.992959912795}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Esophagus_Mucosa", "value": -1.56817043372}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Esophagus_Mucosa", "value": -1.7162520508}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Esophagus_Mucosa", "value": -0.853043207837}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Esophagus_Mucosa", "value": -1.4527402498}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Esophagus_Mucosa", "value": -1.5554211931}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Esophagus_Mucosa", "value": 0.8191231372300001}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Esophagus_Mucosa", "value": 2.03966667438}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Esophagus_Mucosa", "value": 1.06076790052}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Esophagus_Mucosa", "value": -0.407093201165}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Esophagus_Mucosa", "value": -1.8396552254499998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Esophagus_Mucosa", "value": 0.191437918996}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Esophagus_Mucosa", "value": 0.00564346084426}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Esophagus_Mucosa", "value": 0.223664983001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Esophagus_Mucosa", "value": 0.0740695421964}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Esophagus_Mucosa", "value": 0.11639249175899999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Esophagus_Mucosa", "value": 0.229748524555}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Esophagus_Mucosa", "value": -1.4029522735}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Esophagus_Mucosa", "value": -0.8684737461740001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Esophagus_Mucosa", "value": 0.369043227564}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Esophagus_Mucosa", "value": -1.30505851535}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Esophagus_Mucosa", "value": -0.7277776613439999}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Esophagus_Mucosa", "value": -0.14214216009400002}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Esophagus_Muscularis", "value": 1.85582261652}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Esophagus_Muscularis", "value": -5.01333723851}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Esophagus_Muscularis", "value": 8.196197917989998}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Esophagus_Muscularis", "value": 4.88848007049}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Esophagus_Muscularis", "value": 1.8825985991599998}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Esophagus_Muscularis", "value": 0.9947655816650001}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Esophagus_Muscularis", "value": 0.7794367373309999}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Esophagus_Muscularis", "value": 0.8667146033040001}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Esophagus_Muscularis", "value": 1.93563241503}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Esophagus_Muscularis", "value": 1.54231174581}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Esophagus_Muscularis", "value": 0.624863997226}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Esophagus_Muscularis", "value": 1.3861761616600001}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Esophagus_Muscularis", "value": 5.25548984298}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Esophagus_Muscularis", "value": 3.0240972732899998}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Esophagus_Muscularis", "value": 2.73824979232}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Esophagus_Muscularis", "value": 8.51301943185}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Esophagus_Muscularis", "value": 13.4399035806}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Esophagus_Muscularis", "value": 11.143378808800001}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Esophagus_Muscularis", "value": 0.811398658107}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Esophagus_Muscularis", "value": -4.42977902377}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Esophagus_Muscularis", "value": -8.49251118875}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Esophagus_Muscularis", "value": 5.26439614726}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Esophagus_Muscularis", "value": 3.22493143934}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Esophagus_Muscularis", "value": 3.1177858597799997}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Esophagus_Muscularis", "value": -2.32979510821}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Esophagus_Muscularis", "value": -0.48055232929700004}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Esophagus_Muscularis", "value": 1.4019263204299999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Esophagus_Muscularis", "value": 2.62150200911}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Esophagus_Muscularis", "value": 0.0106497487778}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Esophagus_Muscularis", "value": 2.390485151}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Esophagus_Muscularis", "value": 1.8175652419599997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Esophagus_Muscularis", "value": 2.3609476578599997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Esophagus_Muscularis", "value": 1.7509488075400002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Esophagus_Muscularis", "value": 1.6732163619100002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Esophagus_Muscularis", "value": 1.10280358784}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Esophagus_Muscularis", "value": 1.9325324439400002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Esophagus_Muscularis", "value": 1.68015180787}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Esophagus_Muscularis", "value": 0.978103325174}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Esophagus_Muscularis", "value": 1.77156696241}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Esophagus_Muscularis", "value": -1.7087881896200001}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Esophagus_Muscularis", "value": 5.667855305780001}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Esophagus_Muscularis", "value": 3.97624175617}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Esophagus_Muscularis", "value": -1.3152931646500001}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Esophagus_Muscularis", "value": -2.81728443158}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Esophagus_Muscularis", "value": -1.2343414127399999}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Esophagus_Muscularis", "value": -0.847041354271}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Esophagus_Muscularis", "value": -0.7838293707840001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Esophagus_Muscularis", "value": 1.8879830422}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Esophagus_Muscularis", "value": 1.88923866268}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Esophagus_Muscularis", "value": 1.88316073833}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Esophagus_Muscularis", "value": 0.896674193641}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Esophagus_Muscularis", "value": 1.81292435328}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Esophagus_Muscularis", "value": 1.9643283114099996}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Esophagus_Muscularis", "value": 0.871570746014}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Esophagus_Muscularis", "value": 1.8099989394599998}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Esophagus_Muscularis", "value": 1.69302105521}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Liver", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 2.3663030693200002}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Small_Intestine_Terminal_Ileum", "value": -3.0491419987299997}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 15.8007125879}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 10.0934714827}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 3.9968361716400005}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 1.19293923146}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 2.59835628648}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 2.8540860956}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 2.46778522789}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 3.42750938535}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 3.00489219853}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 2.30466579676}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 9.7123870922}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 6.1972077066799995}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 5.600123667640001}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 17.1478476975}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 26.3646672021}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 21.7545971247}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 3.07869016667}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Small_Intestine_Terminal_Ileum", "value": -2.7353911761500003}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Small_Intestine_Terminal_Ileum", "value": -18.0563206438}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 10.8086126557}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 2.3093331273599995}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 3.7476530021599994}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Small_Intestine_Terminal_Ileum", "value": -6.590989595980001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.578296407334}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 3.2353841605299998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 3.2115811425599996}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 1.28501177469}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 1.7662057244599998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 2.1115840119400002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 1.7843769866699999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 2.0759088118}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 2.0699808817200003}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 1.3896014870500002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 2.31011516449}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 1.8696572003999998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 1.40623674856}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 2.15766279013}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Small_Intestine_Terminal_Ileum", "value": -9.25670125126}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 14.0042486371}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 4.852402649069999}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Small_Intestine_Terminal_Ileum", "value": -1.9739288738900003}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Small_Intestine_Terminal_Ileum", "value": -3.4869262814499997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Small_Intestine_Terminal_Ileum", "value": -3.17786754378}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.469379287025}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.550613530802}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 3.4037392660299997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 3.4586243622}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 3.83138233988}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 2.5719644492700002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 4.33440839648}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 3.83900865648}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 2.60186975594}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 4.35452579327}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Small_Intestine_Terminal_Ileum", "value": 1.9170011119200001}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Stomach", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Stomach", "value": 0.0}], "data-8598cfa8f5616616f25367b6bfb77e4a": [{"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "DGN_Whole_Blood", "value": 1.9781021254364235}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "DGN_Whole_Blood", "value": -4.1552040156175085}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "DGN_Whole_Blood", "value": 7.533548379642452}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "DGN_Whole_Blood", "value": 4.121552373552056}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "DGN_Whole_Blood", "value": 1.7056736846472735}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "DGN_Whole_Blood", "value": 0.8337986966173716}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "DGN_Whole_Blood", "value": 1.6643559456918524}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "DGN_Whole_Blood", "value": 1.6808754425162444}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "DGN_Whole_Blood", "value": 2.1053280084038968}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "DGN_Whole_Blood", "value": 2.243805227958842}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "DGN_Whole_Blood", "value": 1.600845907015461}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "DGN_Whole_Blood", "value": 1.2254992252693488}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "DGN_Whole_Blood", "value": 4.690388024955567}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "DGN_Whole_Blood", "value": 2.0527974168888634}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "DGN_Whole_Blood", "value": 1.7671085515436662}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "DGN_Whole_Blood", "value": 8.05707644813276}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "DGN_Whole_Blood", "value": 12.696859871383765}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "DGN_Whole_Blood", "value": 10.554042644805419}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "DGN_Whole_Blood", "value": 1.704473047832153}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "DGN_Whole_Blood", "value": -4.032554836824123}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "DGN_Whole_Blood", "value": -6.559695574681827}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "DGN_Whole_Blood", "value": 4.513760990108485}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "DGN_Whole_Blood", "value": 3.159156109603693}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "DGN_Whole_Blood", "value": 3.360557471620901}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "DGN_Whole_Blood", "value": -1.5037915043090984}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "DGN_Whole_Blood", "value": -0.747822177587951}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "DGN_Whole_Blood", "value": 0.8613602472569507}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "DGN_Whole_Blood", "value": 1.768246110993809}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "DGN_Whole_Blood", "value": 0.01530049634992212}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "DGN_Whole_Blood", "value": 1.959582249541321}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "DGN_Whole_Blood", "value": 0.9970595629056388}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "DGN_Whole_Blood", "value": 1.9500666350997768}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "DGN_Whole_Blood", "value": 0.9695772609870248}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "DGN_Whole_Blood", "value": 0.44316739407277184}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "DGN_Whole_Blood", "value": 0.5437977744947768}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "DGN_Whole_Blood", "value": 0.6305072769934145}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "DGN_Whole_Blood", "value": 0.4690089897894027}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "DGN_Whole_Blood", "value": 0.4876582710392761}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "DGN_Whole_Blood", "value": 0.5781309498975057}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "DGN_Whole_Blood", "value": -2.6597642822277208}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "DGN_Whole_Blood", "value": 6.621226231813762}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "DGN_Whole_Blood", "value": 3.9968371479060543}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "DGN_Whole_Blood", "value": -1.2357344606756908}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "DGN_Whole_Blood", "value": -1.231824914489679}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "DGN_Whole_Blood", "value": -1.4828257900431654}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "DGN_Whole_Blood", "value": -1.4029242381419091}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "DGN_Whole_Blood", "value": -1.2288640020519226}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "DGN_Whole_Blood", "value": 1.8719557108557656}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "DGN_Whole_Blood", "value": 1.8058372636995672}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "DGN_Whole_Blood", "value": 1.535970201024741}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "DGN_Whole_Blood", "value": 0.9390022047732768}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "DGN_Whole_Blood", "value": 1.569853423525398}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "DGN_Whole_Blood", "value": 1.541517491144741}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "DGN_Whole_Blood", "value": 0.9001270432111219}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "DGN_Whole_Blood", "value": 1.5192722158249456}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "DGN_Whole_Blood", "value": 1.5786088885849634}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Whole_Blood", "value": 1.5923112429799997}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Whole_Blood", "value": -4.52914195072}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Whole_Blood", "value": 7.442617848489999}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Whole_Blood", "value": 4.22188497743}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Whole_Blood", "value": 1.39435253114}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Whole_Blood", "value": 1.1555039464899999}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Whole_Blood", "value": 0.778949671498}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Whole_Blood", "value": 0.814170234276}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Whole_Blood", "value": 1.5775574366}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Whole_Blood", "value": 1.34182880808}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Whole_Blood", "value": 0.646627342375}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Whole_Blood", "value": 1.3894007928799998}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Whole_Blood", "value": 4.74873410969}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Whole_Blood", "value": 2.387352926}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Whole_Blood", "value": 2.1429578260900004}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Whole_Blood", "value": 8.39578102105}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Whole_Blood", "value": 12.5802238972}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Whole_Blood", "value": 10.2720231578}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Whole_Blood", "value": 0.8321050407780001}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Whole_Blood", "value": -4.3845184615}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Whole_Blood", "value": -8.12743725398}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Whole_Blood", "value": 4.6848269480699996}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Whole_Blood", "value": 3.31307807051}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Whole_Blood", "value": 2.85700653082}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Whole_Blood", "value": -2.26936365249}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Whole_Blood", "value": -0.201046486576}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Whole_Blood", "value": 1.9956112856200001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Whole_Blood", "value": 2.60985289972}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Whole_Blood", "value": 0.667084951486}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Whole_Blood", "value": 2.43628073821}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Whole_Blood", "value": 1.7695349513799998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Whole_Blood", "value": 2.44284325544}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Whole_Blood", "value": 1.70699591835}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Whole_Blood", "value": 2.0145830892200003}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Whole_Blood", "value": 1.05223997704}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Whole_Blood", "value": 1.5475695368}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Whole_Blood", "value": 1.9406977241999999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Whole_Blood", "value": 1.01491608226}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Whole_Blood", "value": 1.4211662726199998}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Whole_Blood", "value": -1.94800981871}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Whole_Blood", "value": 5.67440883039}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Whole_Blood", "value": 3.8100798251}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Whole_Blood", "value": -1.2711709500700001}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Whole_Blood", "value": -2.3982407547099998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Whole_Blood", "value": -1.28156700396}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Whole_Blood", "value": -0.73146201428}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Whole_Blood", "value": -0.5784048780650001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Whole_Blood", "value": 2.4059115269999998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Whole_Blood", "value": 2.4271434366599998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Whole_Blood", "value": 1.89324192213}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Whole_Blood", "value": 1.3952095708399999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Whole_Blood", "value": 2.18104967049}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Whole_Blood", "value": 2.0086592709}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Whole_Blood", "value": 1.3925577438499999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Whole_Blood", "value": 2.26953949118}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Whole_Blood", "value": 1.76593202409}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Artery_Aorta", "value": 0.521554599829}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Artery_Aorta", "value": -4.97943866748}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Artery_Aorta", "value": 3.73686961012}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Artery_Aorta", "value": 1.6843488069900001}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Artery_Aorta", "value": 2.1424822469400002}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Artery_Aorta", "value": 2.35329440791}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Artery_Aorta", "value": 0.9706068237909999}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Artery_Aorta", "value": 1.0618023962899998}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Artery_Aorta", "value": 1.31572082301}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Artery_Aorta", "value": 1.4486103567500002}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Artery_Aorta", "value": 0.884749060227}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Artery_Aorta", "value": 2.80130758968}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Artery_Aorta", "value": 2.2015285484400002}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Artery_Aorta", "value": 0.06983751686160002}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Artery_Aorta", "value": -0.102871083368}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Artery_Aorta", "value": 4.4932718977}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Artery_Aorta", "value": 6.704478915369999}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Artery_Aorta", "value": 5.47028354546}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Artery_Aorta", "value": 1.16789468585}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Artery_Aorta", "value": -5.1292414842300005}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Artery_Aorta", "value": -4.46645839507}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Artery_Aorta", "value": 1.9340509116099998}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Artery_Aorta", "value": 2.86550367318}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Artery_Aorta", "value": 1.49163240945}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Artery_Aorta", "value": -1.6113321955200002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Artery_Aorta", "value": -0.63232673227}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Artery_Aorta", "value": 0.9690097985539999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Artery_Aorta", "value": 2.6409457604599997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Artery_Aorta", "value": 0.00705373058129}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Artery_Aorta", "value": 0.9514109195379999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Artery_Aorta", "value": 0.100370471615}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Artery_Aorta", "value": 0.91351690426}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Artery_Aorta", "value": 0.1257697544}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Artery_Aorta", "value": 0.918022557559}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Artery_Aorta", "value": -0.0780409092209}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Artery_Aorta", "value": 0.65054102985}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Artery_Aorta", "value": 0.925170939068}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Artery_Aorta", "value": -0.14496396645}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Artery_Aorta", "value": 0.610252965036}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Artery_Aorta", "value": -0.43293205741999996}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Artery_Aorta", "value": 4.08512695704}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Artery_Aorta", "value": 2.66421360218}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Artery_Aorta", "value": 0.20311725604400002}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Artery_Aorta", "value": -1.48742339567}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Artery_Aorta", "value": 1.0009268228200001}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Artery_Aorta", "value": -1.55398337297}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Artery_Aorta", "value": -1.37643844058}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Artery_Aorta", "value": 0.522651239726}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Artery_Aorta", "value": 0.573808434612}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Artery_Aorta", "value": 1.03305429254}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Artery_Aorta", "value": 0.25564909733}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Artery_Aorta", "value": 0.707671062403}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Artery_Aorta", "value": 1.09628687582}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Artery_Aorta", "value": 0.227674073629}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Artery_Aorta", "value": 0.69971370206}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Artery_Aorta", "value": 1.15773214408}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Artery_Tibial", "value": 2.42694194882}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Artery_Tibial", "value": -3.0245467501200003}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Artery_Tibial", "value": 3.63069110635}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Artery_Tibial", "value": 2.91341840207}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Artery_Tibial", "value": 1.7153474855500002}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Artery_Tibial", "value": 1.36202277028}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Artery_Tibial", "value": 0.21105089169500002}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Artery_Tibial", "value": 0.286608860046}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Artery_Tibial", "value": 2.0416598429499997}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Artery_Tibial", "value": 1.11729747161}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Artery_Tibial", "value": 0.137820817085}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Artery_Tibial", "value": 1.51219350776}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Artery_Tibial", "value": 2.1613978875599997}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Artery_Tibial", "value": 1.86303842945}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Artery_Tibial", "value": 1.72341761064}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Artery_Tibial", "value": 4.668367147280001}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Artery_Tibial", "value": 5.97159526574}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Artery_Tibial", "value": 4.77407021057}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Artery_Tibial", "value": 0.262482197865}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Artery_Tibial", "value": -3.4712639362099997}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Artery_Tibial", "value": -4.3327662268500005}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Artery_Tibial", "value": 3.1326931514300003}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Artery_Tibial", "value": 2.21537028532}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Artery_Tibial", "value": 3.74881144795}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Artery_Tibial", "value": -1.06890588915}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Artery_Tibial", "value": 0.626052387456}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Artery_Tibial", "value": 2.88586597209}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Artery_Tibial", "value": 3.03829129838}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Artery_Tibial", "value": 1.0771155945}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Artery_Tibial", "value": 2.3978960693299998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Artery_Tibial", "value": 2.63263251525}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Artery_Tibial", "value": 2.3016137244299997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Artery_Tibial", "value": 2.53614142594}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Artery_Tibial", "value": 2.18811462157}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Artery_Tibial", "value": 0.873673190042}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Artery_Tibial", "value": 2.24177760634}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Artery_Tibial", "value": 2.13709168863}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Artery_Tibial", "value": 0.8332269076450001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Artery_Tibial", "value": 2.1412499193400003}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Artery_Tibial", "value": -0.838332332827}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Artery_Tibial", "value": 3.96248692799}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Artery_Tibial", "value": 4.3596904390099995}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Artery_Tibial", "value": 0.5339511808199999}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Artery_Tibial", "value": -1.21828635733}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Artery_Tibial", "value": 2.0996531480300002}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Artery_Tibial", "value": 0.27721555286}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Artery_Tibial", "value": 0.398125262405}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Artery_Tibial", "value": 2.85567670559}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Artery_Tibial", "value": 2.73705336663}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Artery_Tibial", "value": 2.1238361154599996}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Artery_Tibial", "value": 0.990164166934}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Artery_Tibial", "value": 2.10296379908}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Artery_Tibial", "value": 2.16668113723}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Artery_Tibial", "value": 0.9559286059759999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Artery_Tibial", "value": 2.10299274339}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Artery_Tibial", "value": 2.1185731655400004}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Heart_Atrial_Appendage", "value": 1.8930010074}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Heart_Atrial_Appendage", "value": -3.9447574520199997}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Heart_Atrial_Appendage", "value": 8.81833146712}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Heart_Atrial_Appendage", "value": 4.8541553162}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Heart_Atrial_Appendage", "value": 0.946404710679}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Heart_Atrial_Appendage", "value": -0.09622877699889999}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Heart_Atrial_Appendage", "value": 0.26792054889899997}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Heart_Atrial_Appendage", "value": 0.35495765981399996}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Heart_Atrial_Appendage", "value": 0.190446607739}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Heart_Atrial_Appendage", "value": 0.40059297402500005}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Heart_Atrial_Appendage", "value": -0.0992784571482}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Heart_Atrial_Appendage", "value": -0.757347799668}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Heart_Atrial_Appendage", "value": 5.2719132696}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Heart_Atrial_Appendage", "value": 3.6357791486699997}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Heart_Atrial_Appendage", "value": 3.0003783907700003}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Heart_Atrial_Appendage", "value": 9.79070204919}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Heart_Atrial_Appendage", "value": 16.8876279792}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Heart_Atrial_Appendage", "value": 14.4618675017}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Heart_Atrial_Appendage", "value": -0.0180926788545}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Heart_Atrial_Appendage", "value": -2.07342313231}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Heart_Atrial_Appendage", "value": -9.62302783908}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Heart_Atrial_Appendage", "value": 5.63962823794}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Heart_Atrial_Appendage", "value": 2.8339437024799996}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Heart_Atrial_Appendage", "value": 2.3549091072599997}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Heart_Atrial_Appendage", "value": -2.3951394269799997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Heart_Atrial_Appendage", "value": -0.011482954666200001}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Heart_Atrial_Appendage", "value": 1.5509814934900001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Heart_Atrial_Appendage", "value": 1.00901811071}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Heart_Atrial_Appendage", "value": 0.8077367470029999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Heart_Atrial_Appendage", "value": 2.60833026763}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Heart_Atrial_Appendage", "value": 1.7147221963400001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Heart_Atrial_Appendage", "value": 2.65010130378}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Heart_Atrial_Appendage", "value": 1.6064066762399998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Heart_Atrial_Appendage", "value": 1.11787864924}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Heart_Atrial_Appendage", "value": 2.14613081062}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Heart_Atrial_Appendage", "value": 2.1061822778}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Heart_Atrial_Appendage", "value": 1.13398023835}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Heart_Atrial_Appendage", "value": 2.2134838833400003}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Heart_Atrial_Appendage", "value": 2.12626162962}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Heart_Atrial_Appendage", "value": -3.5933573615699994}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Heart_Atrial_Appendage", "value": 6.9921601855499995}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Heart_Atrial_Appendage", "value": 1.7353626436200003}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Heart_Atrial_Appendage", "value": -1.6967331357700002}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Heart_Atrial_Appendage", "value": -3.2641832666000004}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Heart_Atrial_Appendage", "value": -3.10762615989}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Heart_Atrial_Appendage", "value": 0.42459840780599994}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Heart_Atrial_Appendage", "value": 0.40004482289899995}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Heart_Atrial_Appendage", "value": 1.9824584860099999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Heart_Atrial_Appendage", "value": 1.98615601275}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Heart_Atrial_Appendage", "value": 2.12091047153}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Heart_Atrial_Appendage", "value": 1.49223326831}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Heart_Atrial_Appendage", "value": 2.3594499370400004}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Heart_Atrial_Appendage", "value": 2.2542697451700002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Heart_Atrial_Appendage", "value": 1.5923889103400002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Heart_Atrial_Appendage", "value": 2.56785233968}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Heart_Atrial_Appendage", "value": 0.0458803923439}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Heart_Left_Ventricle", "value": 2.40780976467}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Heart_Left_Ventricle", "value": -4.60260461753}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Heart_Left_Ventricle", "value": 5.753430760950001}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Heart_Left_Ventricle", "value": 3.3424963048}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Heart_Left_Ventricle", "value": 2.26119429378}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Heart_Left_Ventricle", "value": 1.2758942813}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Heart_Left_Ventricle", "value": 1.12196125445}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Heart_Left_Ventricle", "value": 1.22801198241}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Heart_Left_Ventricle", "value": 2.5825509664200004}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Heart_Left_Ventricle", "value": 2.0386069018}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Heart_Left_Ventricle", "value": 1.02826672263}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Heart_Left_Ventricle", "value": 1.6792359249899997}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Heart_Left_Ventricle", "value": 3.70807461856}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Heart_Left_Ventricle", "value": 1.69042251979}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Heart_Left_Ventricle", "value": 1.50689399419}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Heart_Left_Ventricle", "value": 6.49966928245}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Heart_Left_Ventricle", "value": 9.54800460206}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Heart_Left_Ventricle", "value": 7.80171199875}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Heart_Left_Ventricle", "value": 1.14509460219}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Heart_Left_Ventricle", "value": -4.6126371764}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Heart_Left_Ventricle", "value": -5.4488567437599995}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Heart_Left_Ventricle", "value": 3.6535969678199995}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Heart_Left_Ventricle", "value": 2.36288110615}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Heart_Left_Ventricle", "value": 3.7141515088999997}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Heart_Left_Ventricle", "value": -1.32484068649}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Heart_Left_Ventricle", "value": -0.42558595026}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Heart_Left_Ventricle", "value": 1.97416127585}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Heart_Left_Ventricle", "value": 2.8393643307599996}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Heart_Left_Ventricle", "value": 0.6752492046199999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Heart_Left_Ventricle", "value": 2.1892717284}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Heart_Left_Ventricle", "value": 1.71790045623}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Heart_Left_Ventricle", "value": 2.13447254188}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Heart_Left_Ventricle", "value": 1.63867720314}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Heart_Left_Ventricle", "value": 1.6773244638}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Heart_Left_Ventricle", "value": 0.922677971081}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Heart_Left_Ventricle", "value": 1.8450621799400002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Heart_Left_Ventricle", "value": 1.6466354007600001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Heart_Left_Ventricle", "value": 0.838928427499}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Heart_Left_Ventricle", "value": 1.72133051466}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Heart_Left_Ventricle", "value": -1.77028409461}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Heart_Left_Ventricle", "value": 5.117985114780001}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Heart_Left_Ventricle", "value": 4.50111663736}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Heart_Left_Ventricle", "value": -0.910358316092}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Heart_Left_Ventricle", "value": -2.17650533182}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Heart_Left_Ventricle", "value": -0.22422711725299999}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Heart_Left_Ventricle", "value": -0.6936266745190001}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Heart_Left_Ventricle", "value": -0.525841821345}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Heart_Left_Ventricle", "value": 2.11418801548}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Heart_Left_Ventricle", "value": 2.0524844450599997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Heart_Left_Ventricle", "value": 1.8308425872299998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Heart_Left_Ventricle", "value": 1.18973438751}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Heart_Left_Ventricle", "value": 1.9611386695500002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Heart_Left_Ventricle", "value": 1.8135016788299998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Heart_Left_Ventricle", "value": 1.16978968875}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Heart_Left_Ventricle", "value": 1.9120643720500001}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Heart_Left_Ventricle", "value": 1.97954552648}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Spleen", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Spleen", "value": 0.0}], "data-65e81b922a9b3b0da2a40371079c3a2a": [{"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Adipose_Subcutaneous", "value": 2.33515414712}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Adipose_Subcutaneous", "value": -5.71668605688}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Adipose_Subcutaneous", "value": 5.25056676797}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Adipose_Subcutaneous", "value": 3.2437248716000004}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Adipose_Subcutaneous", "value": 2.28764048026}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Adipose_Subcutaneous", "value": 1.32911475578}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Adipose_Subcutaneous", "value": 1.15923119143}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Adipose_Subcutaneous", "value": 1.27850455129}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Adipose_Subcutaneous", "value": 3.37156519788}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Adipose_Subcutaneous", "value": 2.3611495723400004}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Adipose_Subcutaneous", "value": 1.0389160311799999}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Adipose_Subcutaneous", "value": 1.7282267542200003}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Adipose_Subcutaneous", "value": 3.4312878935400004}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Adipose_Subcutaneous", "value": 1.76198195085}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Adipose_Subcutaneous", "value": 1.62187024253}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Adipose_Subcutaneous", "value": 5.84132707975}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Adipose_Subcutaneous", "value": 8.43169635723}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Adipose_Subcutaneous", "value": 6.81061900243}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Adipose_Subcutaneous", "value": 1.17911087713}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Adipose_Subcutaneous", "value": -5.43937891615}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Adipose_Subcutaneous", "value": -4.87984739209}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Adipose_Subcutaneous", "value": 3.4927567506800004}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Adipose_Subcutaneous", "value": 1.8657084919599998}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Adipose_Subcutaneous", "value": 3.4767680175}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Adipose_Subcutaneous", "value": -1.09925206387}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Adipose_Subcutaneous", "value": -0.558997733145}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Adipose_Subcutaneous", "value": 2.08724324007}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Adipose_Subcutaneous", "value": 2.92331721941}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Adipose_Subcutaneous", "value": 0.663066424089}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Adipose_Subcutaneous", "value": 1.77232229453}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Adipose_Subcutaneous", "value": 1.3015282343}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Adipose_Subcutaneous", "value": 1.6463473576899998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Adipose_Subcutaneous", "value": 1.2289164613899999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Adipose_Subcutaneous", "value": 1.8870357025}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Adipose_Subcutaneous", "value": 0.47804846939499995}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Adipose_Subcutaneous", "value": 1.7372136060099999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Adipose_Subcutaneous", "value": 1.82660884537}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Adipose_Subcutaneous", "value": 0.43275876700000004}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Adipose_Subcutaneous", "value": 1.63767565321}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Adipose_Subcutaneous", "value": -1.1344703775799998}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Adipose_Subcutaneous", "value": 4.50006920461}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Adipose_Subcutaneous", "value": 4.177946035280001}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Adipose_Subcutaneous", "value": -1.05531986144}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Adipose_Subcutaneous", "value": -3.24199815088}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Adipose_Subcutaneous", "value": 0.252372577355}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Adipose_Subcutaneous", "value": -0.208436866204}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Adipose_Subcutaneous", "value": -0.0705111866387}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Adipose_Subcutaneous", "value": 1.64197223155}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Adipose_Subcutaneous", "value": 1.5502324341299998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Adipose_Subcutaneous", "value": 1.73002554142}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Adipose_Subcutaneous", "value": 0.745734575338}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Adipose_Subcutaneous", "value": 1.56345995687}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Adipose_Subcutaneous", "value": 1.7049275140200002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Adipose_Subcutaneous", "value": 0.6876320861570001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Adipose_Subcutaneous", "value": 1.48049233}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Adipose_Subcutaneous", "value": 1.88224564716}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Adipose_Visceral_Omentum", "value": 2.8337433238700003}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Adipose_Visceral_Omentum", "value": -6.0073700502900005}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Adipose_Visceral_Omentum", "value": 4.18890566121}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Adipose_Visceral_Omentum", "value": 2.7677467527}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Adipose_Visceral_Omentum", "value": 1.4211273516799998}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Adipose_Visceral_Omentum", "value": 1.1906347359799998}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Adipose_Visceral_Omentum", "value": -0.09272837252520001}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Adipose_Visceral_Omentum", "value": 0.00982625876426}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Adipose_Visceral_Omentum", "value": 1.6301441595800001}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Adipose_Visceral_Omentum", "value": 0.554033949617}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Adipose_Visceral_Omentum", "value": -0.231590717547}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Adipose_Visceral_Omentum", "value": 1.5307208889100001}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Adipose_Visceral_Omentum", "value": 2.7414025038800003}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Adipose_Visceral_Omentum", "value": 1.57879680113}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Adipose_Visceral_Omentum", "value": 1.37940151171}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Adipose_Visceral_Omentum", "value": 4.73072317642}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Adipose_Visceral_Omentum", "value": 7.7236285772399995}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Adipose_Visceral_Omentum", "value": 6.45836512242}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Adipose_Visceral_Omentum", "value": -0.0757574668955}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Adipose_Visceral_Omentum", "value": -5.4149735521}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Adipose_Visceral_Omentum", "value": -4.18013298179}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Adipose_Visceral_Omentum", "value": 3.0647425736000002}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Adipose_Visceral_Omentum", "value": 1.3391672537000001}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Adipose_Visceral_Omentum", "value": 4.25991456556}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Adipose_Visceral_Omentum", "value": -0.819091667104}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Adipose_Visceral_Omentum", "value": -0.0687479550035}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Adipose_Visceral_Omentum", "value": 2.6809134216999997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Adipose_Visceral_Omentum", "value": 2.64461287593}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Adipose_Visceral_Omentum", "value": 1.21996544483}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Adipose_Visceral_Omentum", "value": 1.76214479146}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Adipose_Visceral_Omentum", "value": 1.6722319045299998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Adipose_Visceral_Omentum", "value": 1.64332106538}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Adipose_Visceral_Omentum", "value": 1.54103402694}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Adipose_Visceral_Omentum", "value": 2.15363018921}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Adipose_Visceral_Omentum", "value": 1.0058931145799999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Adipose_Visceral_Omentum", "value": 2.27424260537}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Adipose_Visceral_Omentum", "value": 2.1089449653}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Adipose_Visceral_Omentum", "value": 0.8961851833650001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Adipose_Visceral_Omentum", "value": 2.17169491203}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Adipose_Visceral_Omentum", "value": -0.8171908691490001}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Adipose_Visceral_Omentum", "value": 3.19544335171}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Adipose_Visceral_Omentum", "value": 5.14110578919}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Adipose_Visceral_Omentum", "value": 0.243653533146}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Adipose_Visceral_Omentum", "value": -1.15435529991}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Adipose_Visceral_Omentum", "value": -0.469745185799}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Adipose_Visceral_Omentum", "value": -2.0208835975799997}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Adipose_Visceral_Omentum", "value": -1.9532193622599998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Adipose_Visceral_Omentum", "value": 1.84036612705}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Adipose_Visceral_Omentum", "value": 1.8622948979799998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Adipose_Visceral_Omentum", "value": 1.58063769848}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Adipose_Visceral_Omentum", "value": 1.0516446393}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Adipose_Visceral_Omentum", "value": 1.6639577844100002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Adipose_Visceral_Omentum", "value": 1.6393907661700002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Adipose_Visceral_Omentum", "value": 0.9521082211570001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Adipose_Visceral_Omentum", "value": 1.76422204292}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Adipose_Visceral_Omentum", "value": 2.0485343264}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Muscle_Skeletal", "value": 3.7769887019999997}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Muscle_Skeletal", "value": -4.55885676471}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Muscle_Skeletal", "value": 7.749388870619999}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Muscle_Skeletal", "value": 4.39251555306}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Muscle_Skeletal", "value": 1.92836474192}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Muscle_Skeletal", "value": 0.87004510506}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Muscle_Skeletal", "value": 0.43956333298699996}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Muscle_Skeletal", "value": 0.600698882958}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Muscle_Skeletal", "value": 1.21516645849}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Muscle_Skeletal", "value": 1.02638526207}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Muscle_Skeletal", "value": 0.371904662997}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Muscle_Skeletal", "value": 1.20330086135}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Muscle_Skeletal", "value": 4.6752866415}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Muscle_Skeletal", "value": 2.4484785466799996}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Muscle_Skeletal", "value": 2.0272010744}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Muscle_Skeletal", "value": 9.56253732222}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Muscle_Skeletal", "value": 15.0154635952}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Muscle_Skeletal", "value": 12.2922068581}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Muscle_Skeletal", "value": 0.47526899886999996}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Muscle_Skeletal", "value": -4.10464821559}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Muscle_Skeletal", "value": -8.96722988632}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Muscle_Skeletal", "value": 5.08460830195}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Muscle_Skeletal", "value": 1.26363068642}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Muscle_Skeletal", "value": 5.446355881640001}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Muscle_Skeletal", "value": -2.3476714995099996}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Muscle_Skeletal", "value": 1.8659514411200002}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Muscle_Skeletal", "value": 5.47846810284}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Muscle_Skeletal", "value": 4.7859233263}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Muscle_Skeletal", "value": 2.3311878779499997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Muscle_Skeletal", "value": 3.2835980023}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Muscle_Skeletal", "value": 2.8161257610900003}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Muscle_Skeletal", "value": 3.1963352601599997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Muscle_Skeletal", "value": 2.6912563658899997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Muscle_Skeletal", "value": 3.3642599690900004}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Muscle_Skeletal", "value": 2.85652515217}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Muscle_Skeletal", "value": 4.331593723359999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Muscle_Skeletal", "value": 3.30717838977}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Muscle_Skeletal", "value": 2.6605503132900004}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Muscle_Skeletal", "value": 4.08546965459}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Muscle_Skeletal", "value": -3.43384893636}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Muscle_Skeletal", "value": 7.578249498189999}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Muscle_Skeletal", "value": 6.4923112859800005}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Muscle_Skeletal", "value": -0.6753447589120001}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Muscle_Skeletal", "value": -2.2762188765099998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Muscle_Skeletal", "value": 1.1499204880299998}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Muscle_Skeletal", "value": -1.6273968999500001}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Muscle_Skeletal", "value": -1.50117342069}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Muscle_Skeletal", "value": 2.8577176350999998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Muscle_Skeletal", "value": 2.75285765821}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Muscle_Skeletal", "value": 3.3073166001599996}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Muscle_Skeletal", "value": 2.7114482889800002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Muscle_Skeletal", "value": 3.9655379193699996}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Muscle_Skeletal", "value": 3.2301895906099998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Muscle_Skeletal", "value": 2.5707223889099997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Muscle_Skeletal", "value": 3.73722228681}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Muscle_Skeletal", "value": 3.22508113331}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Cells_Transformed_fibroblasts", "value": 2.2114514193200003}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Cells_Transformed_fibroblasts", "value": -4.666098277840001}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Cells_Transformed_fibroblasts", "value": 5.0739985585500005}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Cells_Transformed_fibroblasts", "value": 2.8502909431}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Cells_Transformed_fibroblasts", "value": 2.21425072412}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.9219649346599998}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Cells_Transformed_fibroblasts", "value": 0.8017793105159999}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Cells_Transformed_fibroblasts", "value": 0.9205855454269999}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Cells_Transformed_fibroblasts", "value": 2.49280066068}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.7581104966999999}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Cells_Transformed_fibroblasts", "value": 0.671162255043}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Cells_Transformed_fibroblasts", "value": 2.35981758241}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Cells_Transformed_fibroblasts", "value": 3.2234236507200005}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.2486827334000001}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.0834311486}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Cells_Transformed_fibroblasts", "value": 5.97246164575}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Cells_Transformed_fibroblasts", "value": 8.46884608323}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Cells_Transformed_fibroblasts", "value": 6.84734179316}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Cells_Transformed_fibroblasts", "value": 0.830893758206}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Cells_Transformed_fibroblasts", "value": -5.01469958056}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Cells_Transformed_fibroblasts", "value": -5.4438677251}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Cells_Transformed_fibroblasts", "value": 3.09357474643}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.9894880955599998}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Cells_Transformed_fibroblasts", "value": 3.25146493679}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Cells_Transformed_fibroblasts", "value": -1.22245898724}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Cells_Transformed_fibroblasts", "value": -0.0936139661925}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.9669425457599998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Cells_Transformed_fibroblasts", "value": 2.92378597982}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Cells_Transformed_fibroblasts", "value": 0.789852485413}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.7614298195599998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.4286601948700002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.7192180232099998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.37527157095}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.97580950562}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Cells_Transformed_fibroblasts", "value": 0.54202629729}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.8735288369099998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.98509481205}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Cells_Transformed_fibroblasts", "value": 0.44938016818800003}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.7645402606}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Cells_Transformed_fibroblasts", "value": -1.2586640542}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Cells_Transformed_fibroblasts", "value": 4.3980584581599995}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Cells_Transformed_fibroblasts", "value": 3.96828266318}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Cells_Transformed_fibroblasts", "value": -0.454371255634}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Cells_Transformed_fibroblasts", "value": -2.2335644398900003}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.00968985802}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Cells_Transformed_fibroblasts", "value": -0.745570554543}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Cells_Transformed_fibroblasts", "value": -0.597632841176}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.769142989}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.72400521158}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.9615595237299999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Cells_Transformed_fibroblasts", "value": 0.9759989593680001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.9029206004400003}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.9649581087}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Cells_Transformed_fibroblasts", "value": 0.920240745045}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.8392894369599997}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Cells_Transformed_fibroblasts", "value": 1.62595258616}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.23805709193}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -4.400143919}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 5.50218906855}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.49466635195}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.61197314592}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.21546537667}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.1646280543799998}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.29656965646}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.75330315193}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.1752685299400003}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.08441446465}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.622716135}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.6129901722800004}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.9327234242700002}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.8124173867900002}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 5.99389212842}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 8.52954975697}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 6.93375016921}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.17264414148}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -4.254789184040001}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -5.012231796569999}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.72814446025}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.7644277492100002}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.7258916548400003}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -1.55542584296}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -0.08835555609449999}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.3457935255}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.07096901365}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.8190636204880001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.44420433855}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.15803774317}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.37135469566}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.0541025634}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.9691683950999999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.16244643535}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.20702600477}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.9033449375}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.07285156696}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.05432195288}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -1.64158900044}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 4.73665801956}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 4.843473057190001}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -0.8559539832170001}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.2779777676}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -0.56958401368}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -0.300615975421}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -0.124390070122}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.45328603961}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.37768779532}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.76024796505}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.4964162001899999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.13132134961}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.7269326639900002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.50058918159}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.08812867957}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.13797531086}], "data-ce3762ec2e4166403b9ed0d98ba8f7a4": [{"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Pituitary", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Pancreas", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Thyroid", "value": -1.61603802283}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Thyroid", "value": 0.8285762057449999}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Thyroid", "value": 6.638821140219999}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Thyroid", "value": 3.0339362938999996}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Thyroid", "value": 1.59113081782}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Thyroid", "value": 0.942420252347}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Thyroid", "value": 1.73682235685}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Thyroid", "value": 1.79067341074}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Thyroid", "value": 2.0601226939400004}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Thyroid", "value": 2.15210634326}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Thyroid", "value": 1.64249722816}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Thyroid", "value": 0.8372957583089999}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Thyroid", "value": 4.0864217158}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Thyroid", "value": 1.76494467495}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Thyroid", "value": 1.63524403787}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Thyroid", "value": 7.00266777349}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Thyroid", "value": 9.3597954068}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Thyroid", "value": 7.184209500489999}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Thyroid", "value": 1.70042962551}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Thyroid", "value": -0.718565996083}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Thyroid", "value": -8.090548279470001}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Thyroid", "value": 3.3486427893799995}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Thyroid", "value": 1.81849825498}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Thyroid", "value": -2.3281973979700004}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Thyroid", "value": -3.8037927327800007}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Thyroid", "value": -0.734747137102}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Thyroid", "value": -0.730677898158}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Thyroid", "value": 0.34271550004199997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Thyroid", "value": -1.57282927177}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Thyroid", "value": 0.360996631853}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Thyroid", "value": 1.14865504844}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Thyroid", "value": 0.7024018155670001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Thyroid", "value": 1.5722068543799999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Thyroid", "value": 0.180608467943}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Thyroid", "value": -0.384782267963}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Thyroid", "value": -0.631551338491}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Thyroid", "value": 0.0926544434549}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Thyroid", "value": -0.165646649599}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Thyroid", "value": -0.712009389886}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Thyroid", "value": -2.22866926741}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Thyroid", "value": 7.98740902249}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Thyroid", "value": -2.4272915042400003}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Thyroid", "value": -0.736572205555}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Thyroid", "value": -2.69097078043}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Thyroid", "value": 0.584084935581}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Thyroid", "value": 1.3046493266399999}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Thyroid", "value": 1.36857387004}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Thyroid", "value": 1.19749303404}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Thyroid", "value": 1.8647535644599997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Thyroid", "value": 0.133900084443}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Thyroid", "value": 0.343247241485}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Thyroid", "value": -0.23560669609700002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Thyroid", "value": 0.515698837773}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Thyroid", "value": -0.6465021478140001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Thyroid", "value": 0.131610194845}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Thyroid", "value": 0.8901674799889999}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Lung", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Ovary", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Prostate", "value": 1.7375939542900003}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Prostate", "value": -5.116265555219999}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Prostate", "value": 7.540054398410001}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Prostate", "value": 4.14621093075}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Prostate", "value": 2.0554437967099997}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Prostate", "value": 2.16234217876}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Prostate", "value": 0.9015406719399999}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Prostate", "value": 0.996358004276}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Prostate", "value": 2.0931107337}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Prostate", "value": 1.7507525695500001}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Prostate", "value": 0.986180589896}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Prostate", "value": 2.5805683599}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Prostate", "value": 5.29851116744}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Prostate", "value": 2.61696772867}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Prostate", "value": 2.3264760161}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Prostate", "value": 8.24342691993}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Prostate", "value": 13.3155451635}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Prostate", "value": 11.2987747099}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Prostate", "value": 1.1990317448}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Prostate", "value": -4.55598093563}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Prostate", "value": -9.561519568789999}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Prostate", "value": 5.0283032755}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Prostate", "value": 2.8820125526599996}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Prostate", "value": 2.61450819421}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Prostate", "value": -1.67904150605}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Prostate", "value": -0.274950293165}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Prostate", "value": 1.89593690879}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Prostate", "value": 2.68997036079}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Prostate", "value": 0.897899688699}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Prostate", "value": 1.62896119241}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Prostate", "value": 1.65995916949}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Prostate", "value": 1.58900073498}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Prostate", "value": 1.5964507145200002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Prostate", "value": 1.6610058860799999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Prostate", "value": 0.587542817317}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Prostate", "value": 1.6827965900200001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Prostate", "value": 1.64724660679}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Prostate", "value": 0.5104418021079999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Prostate", "value": 1.58051822667}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Prostate", "value": -1.72055105324}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Prostate", "value": 5.51329483753}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Prostate", "value": 3.26206038682}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Prostate", "value": -1.71352004806}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Prostate", "value": -3.74312630764}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Prostate", "value": -0.315187030391}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Prostate", "value": -0.0105145064451}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Prostate", "value": 0.139976825684}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Prostate", "value": 1.9458458084799999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Prostate", "value": 1.8940373626400002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Prostate", "value": 1.8764578319999998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Prostate", "value": 1.0145128619}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Prostate", "value": 1.9391823503}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Prostate", "value": 1.87933000715}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Prostate", "value": 0.9780501610439999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Prostate", "value": 1.892398437}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Prostate", "value": 0.9003146259229999}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Testis", "value": 1.57405028667}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Testis", "value": -4.67070805632}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Testis", "value": 5.91106949434}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Testis", "value": 2.6374003417700003}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Testis", "value": 2.86417770491}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Testis", "value": 2.1482913521099998}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Testis", "value": 2.2487160571200002}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Testis", "value": 2.33146900146}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Testis", "value": 2.5975841676}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Testis", "value": 3.04144978641}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Testis", "value": 2.05670765834}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Testis", "value": 2.4440387582099996}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Testis", "value": 3.85020609041}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Testis", "value": 0.39583291091300005}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Testis", "value": 0.19426491668200002}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Testis", "value": 6.597146186180001}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Testis", "value": 9.85231141157}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Testis", "value": 8.30141517228}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Testis", "value": 2.24451269112}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Testis", "value": -4.23048841785}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Testis", "value": -5.9962297283599995}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Testis", "value": 2.98703178852}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Testis", "value": 2.39671529072}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Testis", "value": 2.55966994021}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Testis", "value": -2.77493407482}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Testis", "value": -0.161166676462}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Testis", "value": 1.3774116013799997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Testis", "value": 2.07316372754}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Testis", "value": 0.07966637637910001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Testis", "value": 1.4737972063999998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Testis", "value": 1.48923227405}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Testis", "value": 1.4181786710100002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Testis", "value": 1.4339404493999999}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Testis", "value": 2.4316719379400005}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Testis", "value": 0.887927415203}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Testis", "value": 2.22279237993}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Testis", "value": 2.28100359083}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Testis", "value": 0.954129770981}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Testis", "value": 2.16927018792}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Testis", "value": -0.740371055597}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Testis", "value": 4.69032589181}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Testis", "value": 3.6487370115499997}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Testis", "value": -1.0957659925499998}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Testis", "value": -4.008739701830001}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Testis", "value": -0.395252391704}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Testis", "value": -0.5074863713319999}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Testis", "value": -0.401164774587}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Testis", "value": 1.78671632285}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Testis", "value": 1.6494893779299997}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Testis", "value": 1.51011707259}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Testis", "value": 1.25178035843}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Testis", "value": 1.74402024338}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Testis", "value": 1.4131755228999998}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Testis", "value": 1.2396324466500002}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Testis", "value": 1.7138182646399998}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Testis", "value": 1.79870162313}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Uterus", "value": 0.0}, {"LDSC Lambda GC": 2.0217, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_link": "nopmid.html", "phenotype": "Body fat percentage complete", "association": 18.536361, "category": "b: anthropometric", "code": "BFPC04", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "LDSC Mean Chi-Square": 2.5273, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "index": 0, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.2531, "PMID or link": "27863252", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "platelet count", "association": 5.9573849999999995, "category": "g: blood", "code": "BLOO01", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "LDSC Mean Chi-Square": 1.768, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 1, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Platelet count (BLOO01) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "hemoglobin concentration", "association": 27.282787, "category": "g: blood", "code": "BLOO03", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "LDSC Mean Chi-Square": 1.4634, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "index": 2, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.207, "PMID or link": "27863252", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "reticulocyte count", "association": 12.58902, "category": "g: blood", "code": "BLOO04", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "LDSC Mean Chi-Square": 1.6377, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "index": 3, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte count (BLOO04) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "monocyte count", "association": 14.472422, "category": "g: blood", "code": "BLOO05", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "LDSC Mean Chi-Square": 1.6776, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "index": 4, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Monocyte count (BLOO05) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.1876, "PMID or link": "27863252", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "eosinophil count", "association": 10.947421, "category": "g: blood", "code": "BLOO07", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "LDSC Mean Chi-Square": 1.6196, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 5, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Eosinophil count (BLOO07) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "granulocyte count", "association": 7.488745, "category": "g: blood", "code": "BLOO09", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "LDSC Mean Chi-Square": 1.5217, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 6, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Granulocyte count (BLOO09) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.2168, "PMID or link": "27863252", "LDSC Intercept": "0.9623 (0.02)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "myeloid white cell count", "association": 9.489509, "category": "g: blood", "code": "BLOO10", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "LDSC Mean Chi-Square": 1.5375, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 7, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "lymphocyte count", "association": 23.59941, "category": "g: blood", "code": "BLOO11", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "LDSC Mean Chi-Square": 1.6107, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 8, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Lymphocyte count (BLOO11) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.2431, "PMID or link": "27863252", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "white blood cell count", "association": 20.767460999999997, "category": "g: blood", "code": "BLOO12", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "LDSC Mean Chi-Square": 1.5935, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 9, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "White blood cell count (BLOO12) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.9585 (0.0207)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum basophil neutrophil counts", "association": 5.859083999999999, "category": "g: blood", "code": "BLOO13", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "LDSC Mean Chi-Square": 1.5207, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 10, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.1908, "PMID or link": "27863252", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum eosinophil basophil counts", "association": 13.207391000000001, "category": "g: blood", "code": "BLOO16", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "LDSC Mean Chi-Square": 1.5912, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 11, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.2103, "PMID or link": "27863252", "LDSC Intercept": "1.0104 (0.024)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Hematocrit", "association": 12.811128, "category": "g: blood", "code": "BLOO20", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "LDSC Mean Chi-Square": 1.47, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "index": 12, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hematocrit (BLOO20) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter percentage of red cells", "association": 3.4357480000000002, "category": "g: blood", "code": "BLOO21", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "index": 13, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.2201, "PMID or link": "27863252", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "High light scatter reticulocyte count", "association": 3.431259, "category": "g: blood", "code": "BLOO22", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "LDSC Mean Chi-Square": 1.6418, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "index": 14, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0988, "PMID or link": "27863252", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "association": 23.230629999999998, "category": "g: blood", "code": "BLOO25", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "LDSC Mean Chi-Square": 1.2926, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 15, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.194, "PMID or link": "27863252", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular hemoglobin MCH", "association": 29.375521999999997, "category": "g: blood", "code": "BLOO26", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "LDSC Mean Chi-Square": 1.7671, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "index": 16, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Mean corpuscular volume", "association": 42.353606, "category": "g: blood", "code": "BLOO27", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "LDSC Mean Chi-Square": 1.8174, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "index": 17, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.2136, "PMID or link": "27863252", "LDSC Intercept": "0.961 (0.0204)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Sum neutrophil eosinophil counts", "association": 6.001004, "category": "g: blood", "code": "BLOO30", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "LDSC Mean Chi-Square": 1.5234, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 18, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.2498, "PMID or link": "27863252", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Plateletcrit", "association": 7.128514999999999, "category": "g: blood", "code": "BLOO33", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "LDSC Mean Chi-Square": 1.6946, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "index": 19, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Plateletcrit (BLOO33) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.1651, "PMID or link": "27863252", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Red cell distribution width", "association": 37.978521, "category": "g: blood", "code": "BLOO35", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "LDSC Mean Chi-Square": 1.6273, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 20, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Red cell distribution width (BLOO35) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.2038, "PMID or link": "27863252", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "phenotype": "Reticulocyte fraction of red cells", "association": 9.981674, "category": "g: blood", "code": "BLOO36", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "LDSC Mean Chi-Square": 1.6369999999999998, "gene": "HIST1H3E", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "index": 21, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 2.094, "PMID or link": "30038396", "LDSC Intercept": "0.979 (0.016)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "phenotype": "Educational Attainment: EduYears", "association": 10.182825, "category": "c: mental and behavioural", "code": "EDUC03", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "LDSC Mean Chi-Square": 2.6521, "gene": "HIST1H3E", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 22, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 2.0175, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.043 (0.0163)", "phenotype_link": "nopmid.html", "phenotype": "Fat mass", "association": 25.839411, "category": "b: anthropometric", "code": "FATM04", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "LDSC Mean Chi-Square": 2.5512, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "index": 23, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat mass (FATM04) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0105, "PMID or link": "28898252", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "phenotype": "HbA1c European", "association": 1.71193, "category": "a: metabolic", "code": "GLYC22", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "LDSC Mean Chi-Square": 1.0661, "gene": "HIST1H3E", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 24, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "HbA1c European (GLYC22) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23563607", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "phenotype": "Extreme Height", "association": 9.935130000000001, "category": "b: anthropometric", "code": "HEIG02", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "LDSC Mean Chi-Square": 1.2268, "gene": "HIST1H3E", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 25, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Extreme Height (HEIG02) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 2.0007, "PMID or link": "25282103", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "phenotype": "Height 2014", "association": 37.329476, "category": "b: anthropometric", "code": "HEIG03", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "LDSC Mean Chi-Square": 2.9423, "gene": "HIST1H3E", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "index": 26, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height 2014 (HEIG03) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height female 2013", "association": 18.707345, "category": "b: anthropometric", "code": "HEIG04F", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "LDSC Mean Chi-Square": 1.1891, "gene": "HIST1H3E", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 27, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height female 2013 (HEIG04F) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "23754948", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Height male 2013", "association": 8.835617, "category": "b: anthropometric", "code": "HEIG04M", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "LDSC Mean Chi-Square": 1.1464, "gene": "HIST1H3E", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 28, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Height male 2013 (HEIG04M) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8715 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male", "association": 9.861066000000001, "category": "b: anthropometric", "code": "HIPC01M", "LDSC Total Observed Scale h2": "0.1898 (0.0154)", "LDSC Mean Chi-Square": 1.0853, "gene": "HIST1H3E", "study size": 57568, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 29, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male (HIPC01M) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European", "association": 13.480684, "category": "b: anthropometric", "code": "HIPC01", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "LDSC Mean Chi-Square": 1.1677, "gene": "HIST1H3E", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 30, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European (HIPC01) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8692 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male", "association": 8.413931, "category": "b: anthropometric", "code": "HIPC02M", "LDSC Total Observed Scale h2": "0.1898 (0.0155)", "LDSC Mean Chi-Square": 1.0839, "gene": "HIST1H3E", "study size": 57598, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 31, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male (HIPC02M) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries", "association": 12.813948, "category": "b: anthropometric", "code": "HIPC02", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "LDSC Mean Chi-Square": 1.1656, "gene": "HIST1H3E", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 32, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European female BMI-adjusted", "association": 13.004027, "category": "b: anthropometric", "code": "HIPC03F", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "LDSC Mean Chi-Square": 1.1134, "gene": "HIST1H3E", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 33, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European male BMI-adjusted", "association": 12.738856, "category": "b: anthropometric", "code": "HIPC03M", "LDSC Total Observed Scale h2": "0.204 (0.016)", "LDSC Mean Chi-Square": 1.1064, "gene": "HIST1H3E", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 34, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference European BMI-adjusted", "association": 17.339713, "category": "b: anthropometric", "code": "HIPC03", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "LDSC Mean Chi-Square": 1.2106, "gene": "HIST1H3E", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 35, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "association": 12.501510000000001, "category": "b: anthropometric", "code": "HIPC04F", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "LDSC Mean Chi-Square": 1.1104, "gene": "HIST1H3E", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 36, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.866 (0.0095)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "association": 9.697258, "category": "b: anthropometric", "code": "HIPC04M", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "LDSC Mean Chi-Square": 1.1051, "gene": "HIST1H3E", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 37, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8066 (0.009)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "association": 24.440535999999998, "category": "b: anthropometric", "code": "HIPC04", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "LDSC Mean Chi-Square": 1.2088, "gene": "HIST1H3E", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 38, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0864, "PMID or link": "25352340", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin", "association": 4.276905999999999, "category": "a: metabolic", "code": "IRON02", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "LDSC Mean Chi-Square": 1.1241, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "index": 39, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin (IRON02) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0436, "PMID or link": "25352340", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "phenotype": "Transferrin saturation", "association": 11.611721000000001, "category": "a: metabolic", "code": "IRON03", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "LDSC Mean Chi-Square": 1.0484, "gene": "HIST1H3E", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "index": 40, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Transferrin saturation (IRON03) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 2.1851, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Intercept": "1.11 (0.0274)", "phenotype_link": "nopmid.html", "phenotype": "Fat free mass / lean mass", "association": 15.580010999999999, "category": "b: anthropometric", "code": "LEAN06", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "LDSC Mean Chi-Square": 3.1445, "gene": "HIST1H3E", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "index": 41, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.1715, "PMID or link": "26502338", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "phenotype": "Systemic Lupus Erythematosus", "association": 14.643649, "category": "e: autoimmune", "code": "LUPU01", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "LDSC Mean Chi-Square": 1.2181, "gene": "HIST1H3E", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "index": 42, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0557, "PMID or link": "27992413", "LDSC Intercept": "1.0058 (0.01)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "phenotype": "Primary sclerosing cholangitis", "association": 37.536525, "category": "e: autoimmune", "code": "PRSC01", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "LDSC Mean Chi-Square": 1.0872, "gene": "HIST1H3E", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "index": 43, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "24390342", "LDSC Intercept": "0.9414 (0.0119)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "phenotype": "Rheumatoid Arthritis 2014 European", "association": 8.757334, "category": "h: musculoskeletal", "code": "RHEU02", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "LDSC Mean Chi-Square": 1.1164, "gene": "HIST1H3E", "study size": 58284, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 44, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.5696, "PMID or link": "25056061", "LDSC Intercept": "1.033 (0.0104)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014", "association": 14.960745000000001, "category": "c: mental and behavioural", "code": "SCHI01", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "LDSC Mean Chi-Square": 1.755, "gene": "HIST1H3E", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "index": 45, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.5622, "PMID or link": "25056061", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "phenotype": "Schizophrenia PGC 2014 European", "association": 16.370407, "category": "c: mental and behavioural", "code": "SCHI02", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "LDSC Mean Chi-Square": 1.7409999999999999, "gene": "HIST1H3E", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "index": 46, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7423 (0.0086)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU", "association": 7.116089, "category": "b: anthropometric", "code": "WAIS01", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "LDSC Mean Chi-Square": 1.1487, "gene": "HIST1H3E", "study size": 231927, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 47, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU (WAIS01) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.7406 (0.0087)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries", "association": 6.507729, "category": "b: anthropometric", "code": "WAIS02", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "LDSC Mean Chi-Square": 1.1516, "gene": "HIST1H3E", "study size": 244261, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 48, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8926 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU female BMI-adjusted", "association": 5.846063, "category": "b: anthropometric", "code": "WAIS03F", "LDSC Total Observed Scale h2": "0.1121 (0.0093)", "LDSC Mean Chi-Square": 1.0848, "gene": "HIST1H3E", "study size": 91952, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 49, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU female BMI-adjusted (WAIS03F) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8744 (0.008)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU male BMI-adjusted", "association": 6.914781, "category": "b: anthropometric", "code": "WAIS03M", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "LDSC Mean Chi-Square": 1.0873, "gene": "HIST1H3E", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 50, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference EU BMI-adjusted", "association": 12.600967, "category": "b: anthropometric", "code": "WAIS03", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "LDSC Mean Chi-Square": 1.1406, "gene": "HIST1H3E", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 51, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8907 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries female BMI-adjusted", "association": 6.442168, "category": "b: anthropometric", "code": "WAIS04F", "LDSC Total Observed Scale h2": "0.1136 (0.0094)", "LDSC Mean Chi-Square": 1.0851, "gene": "HIST1H3E", "study size": 132715, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 52, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries female BMI-adjusted (WAIS04F) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "association": 7.077752, "category": "b: anthropometric", "code": "WAIS04M", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "LDSC Mean Chi-Square": 1.084, "gene": "HIST1H3E", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 53, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 1.0466, "PMID or link": "25673412", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "phenotype": "Waist circumference all ancestries BMI-adjusted", "association": 11.746444, "category": "b: anthropometric", "code": "WAIS04", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "LDSC Mean Chi-Square": 1.1420000000000001, "gene": "HIST1H3E", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 54, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "variable": "Vagina", "value": 0.0}, {"LDSC Lambda GC": 0.9986, "PMID or link": "23754948", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "phenotype": "Weight European female", "association": 9.551618, "category": "b: anthropometric", "code": "WEIG04F", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "LDSC Mean Chi-Square": 1.0711, "gene": "HIST1H3E", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "index": 55, "ENSEMBL": "ENSG00000196966", "phenotype_reference": "Weight European female (WEIG04F) ", "variable": "Vagina", "value": 0.0}]}};
    var embed_opt = {"mode": "vega-lite"};

    function showError(el, error){
        el.innerHTML = ('<div class="error">'
                        + '<p>JavaScript Error: ' + error.message + '</p>'
                        + "<p>This usually means there's a typo in your chart specification. "
                        + "See the javascript console for the full traceback.</p>"
                        + '</div>');
        throw error;
    }
    const el = document.getElementById('vis');
    vegaEmbed("#vis", spec, embed_opt)
      .catch(error => showError(el, error));
  </script>
</body>
</html>
